<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />


<title>Total v Nuclear</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">apaQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https/github.com/brimittleman/apaQTL">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Total v Nuclear</h1>
<h4 class="author"><em>Briana Mittleman</em></h4>
<h4 class="date"><em>5/2/2019</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-09-06
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>apaQTL/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.4.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongobjectspresent"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>Environment:</strong> objects present </a>
</p>
</div>
<div id="strongEnvironmentstrongobjectspresent" class="panel-collapse collapse">
<div class="panel-body">
<p>
The global environment had objects present when the code in the R Markdown file was run. These objects can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment. Use <code>wflow_publish</code> or <code>wflow_build</code> to ensure that the code is always run in an empty environment.
</p>
<p>
The following objects were defined in the global environment when these results were created:
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Class
</th>
<th style="text-align:left;">
Size
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
data
</td>
<td style="text-align:left;">
environment
</td>
<td style="text-align:left;">
56 bytes
</td>
</tr>
<tr>
<td style="text-align:left;">
env
</td>
<td style="text-align:left;">
environment
</td>
<td style="text-align:left;">
56 bytes
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190411code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190411)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190411code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190411)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanapaQTLtree5976ed7e0423551d9136c548a0a8acd0b8583231targetblank5976ed7a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/apaQTL/tree/5976ed7e0423551d9136c548a0a8acd0b8583231" target="_blank">5976ed7</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanapaQTLtree5976ed7e0423551d9136c548a0a8acd0b8583231targetblank5976ed7a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    data/.DS_Store
    Ignored:    docs/.DS_Store
    Ignored:    docs/figure/.DS_Store
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  .Rprofile
    Untracked:  ._.DS_Store
    Untracked:  .gitignore
    Untracked:  @
    Untracked:  _workflowr.yml
    Untracked:  analysis/._PASdescriptiveplots.Rmd
    Untracked:  analysis/._cuttoffPercUsage.Rmd
    Untracked:  analysis/APApeak_Phenotype_GeneLocAnno.Nuclear.5perc.fc.gz.qqnorm.allChrom
    Untracked:  analysis/APApeak_Phenotype_GeneLocAnno.Total.5perc.fc.gz.qqnorm.allChrom
    Untracked:  analysis/QTLexampleplots.Rmd
    Untracked:  analysis/cuttoffPercUsage.Rmd
    Untracked:  analysis/eQTLoverlap.Rmd
    Untracked:  analysis/interpret verify bam.Rmd
    Untracked:  analysis/interpret_verifybam.Rmd
    Untracked:  analysis/mergeRNA.Rmd
    Untracked:  analysis/oldstuffNotNeeded.Rmd
    Untracked:  analysis/remove_badlines.Rmd
    Untracked:  analysis/totalspec.Rmd
    Untracked:  apaQTL.Rproj
    Untracked:  code/.NascentRNAdtPlotFirstintronicPAS.sh.swp
    Untracked:  code/._ApaQTL_nominalNonnorm.sh
    Untracked:  code/._BothFracDTPlotGeneRegions.sh
    Untracked:  code/._BothFracDTPlotGeneRegions_normalized.sh
    Untracked:  code/._EandPqtl_perm.sh
    Untracked:  code/._EandPqtls.sh
    Untracked:  code/._FC_NucintornUpandDown.sh
    Untracked:  code/._FC_UTR.sh
    Untracked:  code/._FC_intornUpandDownsteamPAS.sh
    Untracked:  code/._FC_nascentseq.sh
    Untracked:  code/._FC_newPeaks_olddata.sh
    Untracked:  code/._HMMpermuteTotal.py
    Untracked:  code/._HmmPermute.py
    Untracked:  code/._IntronicPASDT.sh
    Untracked:  code/._LC_samplegroups.py
    Untracked:  code/._LD_qtl.sh
    Untracked:  code/._LD_snpsproxy.sh
    Untracked:  code/._NascentRNAdtPlot.sh
    Untracked:  code/._NascentRNAdtPlot3UTRPAS.sh
    Untracked:  code/._NascentRNAdtPlotExcludeFirstintronicPAS.sh
    Untracked:  code/._NascentRNAdtPlotNucPAS.sh
    Untracked:  code/._NascentRNAdtPlotTotPAS.sh
    Untracked:  code/._NascentRNAdtPlotintronicPAS.sh
    Untracked:  code/._NascnetRNAdtPlotPAS.sh
    Untracked:  code/._NetSeq_fourthintronDT.sh
    Untracked:  code/._NomResfromPASSNP.py
    Untracked:  code/._NuclearPAS_5per.bed.py
    Untracked:  code/._PTTfacetboxplots.R
    Untracked:  code/._PrematureQTLNominal.sh
    Untracked:  code/._PrematureQTLPermuted.sh
    Untracked:  code/._QTL2bed.py
    Untracked:  code/._QTL2bed_withstrand.py
    Untracked:  code/._RNAbam2bw.sh
    Untracked:  code/._RNAseqDTplot.sh
    Untracked:  code/._RunRes2PAS.sh
    Untracked:  code/._SAF215upbed.py
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilefiltPAS
    Untracked:  code/._TESplots100bp.sh
    Untracked:  code/._TESplots150bp.sh
    Untracked:  code/._TESplots200bp.sh
    Untracked:  code/._TotalPAS_5perc.bed.py
    Untracked:  code/._Untitled
    Untracked:  code/._ZipandTabPheno.sh
    Untracked:  code/._aAPAqtl_nominal39ind.sh
    Untracked:  code/._annotatePacBioPASregion.sh
    Untracked:  code/._annotatedPAS2bed.py
    Untracked:  code/._apaInPandE.py
    Untracked:  code/._apaQTLCorrectPvalMakeQQ.R
    Untracked:  code/._apaQTLCorrectpval_6or7a.R
    Untracked:  code/._apaQTL_Nominal.sh
    Untracked:  code/._apaQTL_nominalv67.sh
    Untracked:  code/._apaQTL_permuted.sh
    Untracked:  code/._apaQTL_permuted_test6A7A.sh
    Untracked:  code/._apainRibo.py
    Untracked:  code/._assignNucIntonpeak2intronlocs.sh
    Untracked:  code/._assignTotIntronpeak2intronlocs.sh
    Untracked:  code/._bam2BW_5primemost.sh
    Untracked:  code/._bed2saf.py
    Untracked:  code/._bothFracDTplot1stintron.sh
    Untracked:  code/._bothFracDTplot4thintron.sh
    Untracked:  code/._bothFrac_FC.sh
    Untracked:  code/._callPeaksYL.py
    Untracked:  code/._changeRibonomQTLres2genename.py
    Untracked:  code/._changenomQTLres2geneName.py
    Untracked:  code/._chooseAnno2PAS_pacbio.py
    Untracked:  code/._chooseAnno2SAF.py
    Untracked:  code/._chooseSignalSite
    Untracked:  code/._chooseSignalSite.py
    Untracked:  code/._closestannotated.sh
    Untracked:  code/._closestannotated_byfrac.sh
    Untracked:  code/._cluster.json
    Untracked:  code/._clusterPAS.json
    Untracked:  code/._clusterfiltPAS.json
    Untracked:  code/._codingdms2bed.py
    Untracked:  code/._config.yaml
    Untracked:  code/._config2.yaml
    Untracked:  code/._configOLD.yaml
    Untracked:  code/._convertNominal2SNPLOC.py
    Untracked:  code/._convertNominal2SNPloc2Versions.py
    Untracked:  code/._convertNumeric.py
    Untracked:  code/._correctNomeqtl.R
    Untracked:  code/._createPlinkSampfile.py
    Untracked:  code/._dag.pdf
    Untracked:  code/._eQTL_switch2snploc.py
    Untracked:  code/._eQTLgenestestedapa.py
    Untracked:  code/._encodeRNADTplots.sh
    Untracked:  code/._extractGenotypes.py
    Untracked:  code/._extractseqfromqtlfastq.py
    Untracked:  code/._fc2leafphen.py
    Untracked:  code/._fc_filteredPAS6and7As.sh
    Untracked:  code/._fifteenBPupstreamPAS.py
    Untracked:  code/._fiftyBPupstreamPAS.py
    Untracked:  code/._filter5perc.R
    Untracked:  code/._filter5percPheno.py
    Untracked:  code/._filterLDsnps.py
    Untracked:  code/._filterMPPAS.py
    Untracked:  code/._filterMPPAS_15.py
    Untracked:  code/._filterMPPAS_15_7As.py
    Untracked:  code/._filterMPPAS_50.py
    Untracked:  code/._filterSAFforMP.py
    Untracked:  code/._filterpeaks.py
    Untracked:  code/._finalPASbed2SAF.py
    Untracked:  code/._fix4su304corr.py
    Untracked:  code/._fix4su604corr.py
    Untracked:  code/._fix4sukalisto.py
    Untracked:  code/._fixExandUnexeQTL
    Untracked:  code/._fixExandUnexeQTL.py
    Untracked:  code/._fixFChead.py
    Untracked:  code/._fixFChead_bothfrac.py
    Untracked:  code/._fixFChead_short.py
    Untracked:  code/._fixH3k12ac.py
    Untracked:  code/._fixPASregionSNPs.py
    Untracked:  code/._fixRNAhead4corr.py
    Untracked:  code/._fixRNAkalisto.py
    Untracked:  code/._fixgroupedtranscript.py
    Untracked:  code/._fixhead_netseqfc.py
    Untracked:  code/._getAPAfromanyeQTL.py
    Untracked:  code/._getApapval4eqtl.py
    Untracked:  code/._getApapval4eqtl_unexp.py
    Untracked:  code/._getApapval4eqtl_version67.py
    Untracked:  code/._getDownstreamIntronNuclear.py
    Untracked:  code/._getIntronDownstreamPAS.py
    Untracked:  code/._getIntronUpstreamPAS.py
    Untracked:  code/._getQTLalleles.py
    Untracked:  code/._getQTLfastq.sh
    Untracked:  code/._getUpstreamIntronNuclear.py
    Untracked:  code/._grouptranscripts.py
    Untracked:  code/._intersectVCFandupPAS.sh
    Untracked:  code/._keep5perMAF.py
    Untracked:  code/._keepSNP_vcf.sh
    Untracked:  code/._make5percPeakbed.py
    Untracked:  code/._makeFileID.py
    Untracked:  code/._makePheno.py
    Untracked:  code/._makeSAFbothfrac5perc.py
    Untracked:  code/._makeSNP2rsidfile.py
    Untracked:  code/._makeeQTLempirical_unexp.py
    Untracked:  code/._makeeQTLempiricaldist.py
    Untracked:  code/._makegencondeTSSfile.py
    Untracked:  code/._mapSSsnps2PAS.sh
    Untracked:  code/._mergRNABam.sh
    Untracked:  code/._mergeAllBam.sh
    Untracked:  code/._mergeBW_norm.sh
    Untracked:  code/._mergeBamNascent.sh
    Untracked:  code/._mergeByFracBam.sh
    Untracked:  code/._mergePeaks.sh
    Untracked:  code/._mnase1stintron.sh
    Untracked:  code/._mnaseDT_fourthintron.sh
    Untracked:  code/._namePeaks.py
    Untracked:  code/._netseqDTplot1stIntron.sh
    Untracked:  code/._netseqFC.sh
    Untracked:  code/._nucQTLGWAS.py
    Untracked:  code/._nucSpecQTLineData.py
    Untracked:  code/._nucSpeceffectsize.py
    Untracked:  code/._pQTLsotherdata.py
    Untracked:  code/._pacbioDT.sh
    Untracked:  code/._pacbioIntronicDT.sh
    Untracked:  code/._parseBestbamid.py
    Untracked:  code/._peak2PAS.py
    Untracked:  code/._peakFC.sh
    Untracked:  code/._pheno2countonly.R
    Untracked:  code/._phenoQTLfromlist.py
    Untracked:  code/._processYRIgen.py
    Untracked:  code/._pttQTLsinapaQTL.py
    Untracked:  code/._qtlRegionseq.sh
    Untracked:  code/._qtlsPvalOppFrac.py
    Untracked:  code/._quantassign2parsedpeak.py
    Untracked:  code/._removeXfromHmm.py
    Untracked:  code/._removeloc_pheno.py
    Untracked:  code/._riboQTL.sh
    Untracked:  code/._runCorrectNomEqtl.sh
    Untracked:  code/._runHMMpermuteAPAqtls.sh
    Untracked:  code/._runHMMpermuteeQTLS.sh
    Untracked:  code/._runMakeEmpiricaleQTL_unexp.sh
    Untracked:  code/._runMakeeQTLempirical.sh
    Untracked:  code/._run_bam2bw_all3prime.sh
    Untracked:  code/._run_bam2bw_extra3.sh
    Untracked:  code/._run_bestbamid.sj
    Untracked:  code/._run_filtersnpLD.sh
    Untracked:  code/._run_getAPAfromeQTL_version6.7.sh
    Untracked:  code/._run_getApaPval4eqtl.sh
    Untracked:  code/._run_getapafromeQTL.py
    Untracked:  code/._run_getapafromeQTL.sh
    Untracked:  code/._run_getapapval4eqtl_unexp.sh
    Untracked:  code/._run_leafcutterDiffIso.sh
    Untracked:  code/._run_prxySNP.sh
    Untracked:  code/._run_pttfacetboxplot.sh
    Untracked:  code/._run_sepUsagephen.sh
    Untracked:  code/._run_sepgenobychrom.sh
    Untracked:  code/._run_verifybam.sh
    Untracked:  code/._selectNominalPvalues.py
    Untracked:  code/._sepUsagePhen.py
    Untracked:  code/._sepgenobychrom.py
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._sortindexRNAbam.sh
    Untracked:  code/._submit-snakemakePAS.sh
    Untracked:  code/._submit-snakemakefiltPAS.sh
    Untracked:  code/._subsetAPAnotEorPgene.py
    Untracked:  code/._subsetAPAnotEorPgene_2versions.py
    Untracked:  code/._subsetApanoteGene.py
    Untracked:  code/._subsetApanoteGene_2versions.py
    Untracked:  code/._subsetUnexplainedeQTLs.py
    Untracked:  code/._subsetVCF_SS.sh
    Untracked:  code/._subsetVCF_noSSregions.sh
    Untracked:  code/._subsetVCF_upstreamPAS.sh
    Untracked:  code/._subset_diffisopheno.py
    Untracked:  code/._subsetpermAPAwithGenelist.py
    Untracked:  code/._subsetpermAPAwithGenelist_2versions.py
    Untracked:  code/._subsetvcf_otherreg.sh
    Untracked:  code/._subsetvcf_permSS.sh
    Untracked:  code/._subtrachfiveprimeUTR.sh
    Untracked:  code/._subtractExons.sh
    Untracked:  code/._subtractfiveprimeUTR.sh
    Untracked:  code/._tabixSNPS.sh
    Untracked:  code/._tenBPupstreamPAS.py
    Untracked:  code/._testVerifyBam.sh
    Untracked:  code/._totSeceffectsize.py
    Untracked:  code/._twentyBPupstreamPAS.py
    Untracked:  code/._utrdms2saf.py
    Untracked:  code/._vcf2bed.py
    Untracked:  code/._verifyBam18517N.sh
    Untracked:  code/._verifyBam18517T.sh
    Untracked:  code/._verifyBam19128N.sh
    Untracked:  code/._verifyBam19128T.sh
    Untracked:  code/._wrap_verifybam.sh
    Untracked:  code/._writePTTexamplecode.py
    Untracked:  code/._writePTTexamplecode.sh
    Untracked:  code/.pversion
    Untracked:  code/.snakemake/
    Untracked:  code/1
    Untracked:  code/APAqtl_nominal.err
    Untracked:  code/APAqtl_nominal.out
    Untracked:  code/APAqtl_nominal_39.err
    Untracked:  code/APAqtl_nominal_39.out
    Untracked:  code/APAqtl_nominal_nonNorm.err
    Untracked:  code/APAqtl_nominal_nonNorm.out
    Untracked:  code/APAqtl_nominal_versions67.err
    Untracked:  code/APAqtl_nominal_versions67.out
    Untracked:  code/APAqtl_permuted.err
    Untracked:  code/APAqtl_permuted.out
    Untracked:  code/APAqtl_permuted_versions67.err
    Untracked:  code/APAqtl_permuted_versions67.out
    Untracked:  code/ApaQTL_nominalNonnorm.sh
    Untracked:  code/BothFracDTPlot1stintron.err
    Untracked:  code/BothFracDTPlot1stintron.out
    Untracked:  code/BothFracDTPlot4stintron.err
    Untracked:  code/BothFracDTPlot4stintron.out
    Untracked:  code/BothFracDTPlotGeneRegions.err
    Untracked:  code/BothFracDTPlotGeneRegions.out
    Untracked:  code/BothFracDTPlotGeneRegions_norm.err
    Untracked:  code/BothFracDTPlotGeneRegions_norm.out
    Untracked:  code/BothFracDTPlotGeneRegions_normalized.sh
    Untracked:  code/DistPAS2Sig.py
    Untracked:  code/EandPqtl.err
    Untracked:  code/EandPqtl.out
    Untracked:  code/EandPqtl_perm.sh
    Untracked:  code/EandPqtls.sh
    Untracked:  code/EncodeRNADTPlotGeneRegions.err
    Untracked:  code/EncodeRNADTPlotGeneRegions.out
    Untracked:  code/FC_NucintornUpandDown.sh
    Untracked:  code/FC_NucintronPASupandDown.err
    Untracked:  code/FC_NucintronPASupandDown.out
    Untracked:  code/FC_UTR.err
    Untracked:  code/FC_UTR.out
    Untracked:  code/FC_UTR.sh
    Untracked:  code/FC_intornUpandDownsteamPAS.sh
    Untracked:  code/FC_intronPASupandDown.err
    Untracked:  code/FC_intronPASupandDown.out
    Untracked:  code/FC_nascent.err
    Untracked:  code/FC_nascentout
    Untracked:  code/FC_nascentseq.sh
    Untracked:  code/FC_newPAS_olddata.err
    Untracked:  code/FC_newPAS_olddata.out
    Untracked:  code/FC_newPeaks_olddata.sh
    Untracked:  code/HMMpermuteTotal.py
    Untracked:  code/HmmPermute.p
    Untracked:  code/HmmPermute.py
    Untracked:  code/IntronicPASDT.err
    Untracked:  code/IntronicPASDT.out
    Untracked:  code/IntronicPASDT.sh
    Untracked:  code/LC_samplegroups.py
    Untracked:  code/LD_qtl.sh
    Untracked:  code/LD_snpsproxy.sh
    Untracked:  code/LD_vcftools.hap.out
    Untracked:  code/NascentDTPlotGeneRegions.err
    Untracked:  code/NascentDTPlotGeneRegions.out
    Untracked:  code/NascentDTPlotPAS.err
    Untracked:  code/NascentDTPlotPAS.out
    Untracked:  code/NascentDTPlotPAS_3utr.err
    Untracked:  code/NascentDTPlotPAS_3utr.out
    Untracked:  code/NascentDTPlotPAS_firstintron.err
    Untracked:  code/NascentDTPlotPAS_firstintron.out
    Untracked:  code/NascentDTPlotPAS_intron.err
    Untracked:  code/NascentDTPlotPAS_intron.out
    Untracked:  code/NascentDTPlotPAS_nuc.err
    Untracked:  code/NascentDTPlotPAS_nuc.out
    Untracked:  code/NascentDTPlotPAS_tot.err
    Untracked:  code/NascentDTPlotPAS_tot.out
    Untracked:  code/NascentRNAdtPlot.sh
    Untracked:  code/NascentRNAdtPlot3UTRPAS.sh
    Untracked:  code/NascentRNAdtPlotExcludeFirstintronicPAS.sh
    Untracked:  code/NascentRNAdtPlotFirstintronicPAS.sh
    Untracked:  code/NascentRNAdtPlotNucPAS.sh
    Untracked:  code/NascentRNAdtPlotTotPAS.sh
    Untracked:  code/NascentRNAdtPlotintronicPAS.sh
    Untracked:  code/NascnetRNAdtPlotPAS.sh
    Untracked:  code/NetSeq_fourthintronDT.sh
    Untracked:  code/NomResfromPASSNP.py
    Untracked:  code/NuclearPAS_5per.bed.py
    Untracked:  code/Nuclear_example.err
    Untracked:  code/Nuclear_example.out
    Untracked:  code/PACbioDT.err
    Untracked:  code/PACbioDT.out
    Untracked:  code/PACbioDTitronic.err
    Untracked:  code/PACbioDTitronic.out
    Untracked:  code/PTTfacetboxplots.R
    Untracked:  code/PrematureQTLNominal.sh
    Untracked:  code/PrematureQTLPermuted.sh
    Untracked:  code/Prematureqtl_nominal.err
    Untracked:  code/Prematureqtl_nominal.out
    Untracked:  code/Prematureqtl_permuted.err
    Untracked:  code/Prematureqtl_permuted.out
    Untracked:  code/QTL2bed.py
    Untracked:  code/QTL2bed_withstrand.py
    Untracked:  code/README.md
    Untracked:  code/RNABam2BW.err
    Untracked:  code/RNABam2BW.out
    Untracked:  code/RNAbam2bw.sh
    Untracked:  code/RNAseqDTPlotGeneRegions.err
    Untracked:  code/RNAseqDTPlotGeneRegions.out
    Untracked:  code/RNAseqDTplot.sh
    Untracked:  code/Rplots.pdf
    Untracked:  code/RunRes2PAS.sh
    Untracked:  code/SAF215upbed.py
    Untracked:  code/SAF215upbed_gen.py
    Untracked:  code/Script4NuclearPTTqtlexamples.sh
    Untracked:  code/Script4NuclearQTLexamples.sh
    Untracked:  code/Script4TotalPTTqtlexamples.sh
    Untracked:  code/Script4TotalQTLexamples.sh
    Untracked:  code/TESplots100bp.err
    Untracked:  code/TESplots100bp.out
    Untracked:  code/TESplots100bp.sh
    Untracked:  code/TESplots150bp.err
    Untracked:  code/TESplots150bp.out
    Untracked:  code/TESplots150bp.sh
    Untracked:  code/TESplots200bp.err
    Untracked:  code/TESplots200bp.out
    Untracked:  code/TESplots200bp.sh
    Untracked:  code/TotalPAS_5perc.bed.py
    Untracked:  code/Total_example.err
    Untracked:  code/Total_example.out
    Untracked:  code/Untitled
    Untracked:  code/Upstream100Bases_general.py
    Untracked:  code/YRI_LCL.vcf.gz
    Untracked:  code/YRI_LCL_chr1.vcf.gz.log
    Untracked:  code/YRI_LCL_chr1.vcf.gz.recode.vcf
    Untracked:  code/ZipandTabPheno.sh
    Untracked:  code/aAPAqtl_nominal39ind.sh
    Untracked:  code/annotatePacBioPASregion.sh
    Untracked:  code/annotatedPAS2bed.py
    Untracked:  code/annotatedPASregion.err
    Untracked:  code/annotatedPASregion.out
    Untracked:  code/apaInPandE.py
    Untracked:  code/apaQTLCorrectPvalMakeQQ_4pc.R
    Untracked:  code/apaQTLCorrectpval_6or7a.R
    Untracked:  code/apaQTL_Nominal_4pc.sh
    Untracked:  code/apaQTL_nominalv67.sh
    Untracked:  code/apaQTL_permuted.4pc.sh
    Untracked:  code/apaQTL_permuted_test6A7A.sh
    Untracked:  code/apafacetboxplots.R
    Untracked:  code/apainRibo.py
    Untracked:  code/apaqtlfacetboxplots.R
    Untracked:  code/assignNucIntonpeak2intronlocs.sh
    Untracked:  code/assignPeak2Intronicregion.err
    Untracked:  code/assignPeak2Intronicregion.out
    Untracked:  code/assignTotIntronpeak2intronlocs.sh
    Untracked:  code/assigntotPeak2Intronicregion.err
    Untracked:  code/assigntotPeak2Intronicregion.out
    Untracked:  code/bam2BW_5primemost.sh
    Untracked:  code/bam2bw.err
    Untracked:  code/bam2bw.out
    Untracked:  code/bam2bw_5primemost.err
    Untracked:  code/bam2bw_5primemost.out
    Untracked:  code/binary_fileset.log
    Untracked:  code/bothFracDTplot1stintron.sh
    Untracked:  code/bothFracDTplot4thintron.sh
    Untracked:  code/bothFrac_FC.err
    Untracked:  code/bothFrac_FC.out
    Untracked:  code/bothFrac_FC.sh
    Untracked:  code/callSHscripts.txt
    Untracked:  code/changePermQTLres2geneName.py
    Untracked:  code/changeRibonomQTLres2genename.py
    Untracked:  code/changenomQTLres2geneName.py
    Untracked:  code/chooseAnno2PAS_pacbio.py
    Untracked:  code/closestannotated.err
    Untracked:  code/closestannotated.out
    Untracked:  code/closestannotated.sh
    Untracked:  code/closestannotated_byfrac.sh
    Untracked:  code/closestannotatedbyfrac.err
    Untracked:  code/closestannotatedbyfrac.out
    Untracked:  code/codingdms2bed.py
    Untracked:  code/convertNominal2SNPLOC.py
    Untracked:  code/convertNominal2SNPloc2Versions.py
    Untracked:  code/correctNomeqtl.R
    Untracked:  code/createPlinkSampfile.py
    Untracked:  code/dag.pdf
    Untracked:  code/dagPAS.pdf
    Untracked:  code/dagfiltPAS.pdf
    Untracked:  code/eQTL_switch2snploc.py
    Untracked:  code/eQTLgenestestedapa.py
    Untracked:  code/encodeRNADTplots.sh
    Untracked:  code/environmentLeaf.yaml
    Untracked:  code/extractGenotypes.py
    Untracked:  code/extractseqfromqtlfastq.py
    Untracked:  code/fc2leafphen.py
    Untracked:  code/fc_filteredPAS6and7As.sh
    Untracked:  code/fifteenBPupstreamPAS.py
    Untracked:  code/fiftyBPupstreamPAS.py
    Untracked:  code/filterLDsnps.py
    Untracked:  code/filterMPPAS.py
    Untracked:  code/filterMPPAS_15.py
    Untracked:  code/filterMPPAS_15_7As.py
    Untracked:  code/filterMPPAS_50.py
    Untracked:  code/filterSAFforMP.py
    Untracked:  code/filterSAFforMP_gen.py
    Untracked:  code/finalPASbed2SAF.py
    Untracked:  code/findbuginpeaks.R
    Untracked:  code/fix4su304corr.py
    Untracked:  code/fix4su604corr.py
    Untracked:  code/fix4sukalisto.py
    Untracked:  code/fixExandUnexeQTL
    Untracked:  code/fixExandUnexeQTL.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixFChead_short.py
    Untracked:  code/fixFChead_summary.py
    Untracked:  code/fixH3k12ac.py
    Untracked:  code/fixPASregionSNPs.py
    Untracked:  code/fixRNAhead4corr.py
    Untracked:  code/fixRNAkalisto.py
    Untracked:  code/fixgroupedtranscript.py
    Untracked:  code/fixhead_netseqfc.py
    Untracked:  code/genotypesYRI.gen.proc.keep.vcf.log
    Untracked:  code/genotypesYRI.gen.proc.keep.vcf.recode.vcf
    Untracked:  code/get100upPAS.py
    Untracked:  code/getAPAfromanyeQTL.py
    Untracked:  code/getApapval4eqtl.py
    Untracked:  code/getApapval4eqtl_unexp.py
    Untracked:  code/getApapval4eqtl_version67.py
    Untracked:  code/getDownstreamIntronNuclear.py
    Untracked:  code/getIntronDownstreamPAS.py
    Untracked:  code/getIntronUpstreamPAS.py
    Untracked:  code/getQTLalleles.py
    Untracked:  code/getQTLfastq.sh
    Untracked:  code/getSeq100up.sh
    Untracked:  code/getUpstreamIntronNuclear.py
    Untracked:  code/getseq100up.err
    Untracked:  code/getseq100up.out
    Untracked:  code/grouptranscripts.err
    Untracked:  code/grouptranscripts.out
    Untracked:  code/grouptranscripts.py
    Untracked:  code/intersectPAS_ssSNPS.err
    Untracked:  code/intersectPAS_ssSNPS.out
    Untracked:  code/intersectVCFPAS.err
    Untracked:  code/intersectVCFPAS.out
    Untracked:  code/intersectVCFandupPAS.sh
    Untracked:  code/keep5perMAF.py
    Untracked:  code/keepSNP_vcf.sh
    Untracked:  code/log/
    Untracked:  code/makeSAFbothfrac5perc.py
    Untracked:  code/makeSNP2rsidfile.py
    Untracked:  code/makeeQTLempirical_unexp.py
    Untracked:  code/makeeQTLempiricaldist.py
    Untracked:  code/makegencondeTSSfile.py
    Untracked:  code/mapSSsnps2PAS.sh
    Untracked:  code/mergRNABam.sh
    Untracked:  code/mergeBW_norm.sh
    Untracked:  code/mergeBWnorm.err
    Untracked:  code/mergeBWnorm.out
    Untracked:  code/mergeBamNacent.err
    Untracked:  code/mergeBamNacent.out
    Untracked:  code/mergeBamNascent.sh
    Untracked:  code/mergeRNAbam.err
    Untracked:  code/mergeRNAbam.out
    Untracked:  code/mnase1stintron.sh
    Untracked:  code/mnaseDTPlot1stintron.err
    Untracked:  code/mnaseDTPlot1stintron.out
    Untracked:  code/mnaseDTPlot4thintron.err
    Untracked:  code/mnaseDTPlot4thintron.out
    Untracked:  code/mnaseDT_fourthintron.sh
    Untracked:  code/netDTPlot4thintron.out
    Untracked:  code/netseqDTplot1stIntron.sh
    Untracked:  code/netseqFC.err
    Untracked:  code/netseqFC.out
    Untracked:  code/netseqFC.sh
    Untracked:  code/neyDTPlot4thintron.err
    Untracked:  code/nucQTLGWAS.py
    Untracked:  code/nucQTLGWAS_withLD.py
    Untracked:  code/nucSpecQTLineData.py
    Untracked:  code/nucSpeceffectsize.py
    Untracked:  code/pQTLsotherdata.py
    Untracked:  code/pacbioDT.sh
    Untracked:  code/pacbioIntronicDT.sh
    Untracked:  code/parseBestbamid.py
    Untracked:  code/phenoQTLfromlist.py
    Untracked:  code/plink.log
    Untracked:  code/processYRIgen.py
    Untracked:  code/prxySNP.err
    Untracked:  code/prxySNP.out
    Untracked:  code/pttFacetBoxplots.err
    Untracked:  code/pttFacetBoxplots.out
    Untracked:  code/pttQTLsinapaQTL.py
    Untracked:  code/pullTwoMechData.py
    Untracked:  code/qtlFacetBoxplots.err
    Untracked:  code/qtlFacetBoxplots.out
    Untracked:  code/qtlRegionseq.sh
    Untracked:  code/qtlsPvalOppFrac.py
    Untracked:  code/rLD_vcftools.hap.err
    Untracked:  code/removeXfromHmm.py
    Untracked:  code/removeloc_pheno.py
    Untracked:  code/riboQTL.sh
    Untracked:  code/riboqtl.err
    Untracked:  code/riboqtl.out
    Untracked:  code/runBestBamID.err
    Untracked:  code/runCorrectNomEqtl.sh
    Untracked:  code/runCorrectNomeqtl.err
    Untracked:  code/runCorrectNomeqtl.out
    Untracked:  code/runFilterLD.err
    Untracked:  code/runFilterLD.out
    Untracked:  code/runHMMpermute.err
    Untracked:  code/runHMMpermute.out
    Untracked:  code/runHMMpermuteAPAqtls.sh
    Untracked:  code/runHMMpermuteeQTLS.sh
    Untracked:  code/runHMMpermuteeQTLs.err
    Untracked:  code/runHMMpermuteeQTLs.out
    Untracked:  code/runMakeEmpiricaleQTL_unexp.sh
    Untracked:  code/runMakeEmpiricaleQTLs.err
    Untracked:  code/runMakeEmpiricaleQTLs.out
    Untracked:  code/runMakeEmpiricaleQTLsunex.err
    Untracked:  code/runMakeEmpiricaleQTLsunex.out
    Untracked:  code/runMakeeQTLempirical.sh
    Untracked:  code/run_DistPAS2Sig.err
    Untracked:  code/run_DistPAS2Sig.out
    Untracked:  code/run_bam2bw.err
    Untracked:  code/run_bam2bw.out
    Untracked:  code/run_bam2bw_all3prime.sh
    Untracked:  code/run_bam2bw_extra3.sh
    Untracked:  code/run_bam2bwexta.err
    Untracked:  code/run_bam2bwexta.out
    Untracked:  code/run_bestbamid.sh
    Untracked:  code/run_distPAS2Sig.sh
    Untracked:  code/run_filtersnpLD.sh
    Untracked:  code/run_getAPAfromanyeQTL.err
    Untracked:  code/run_getAPAfromanyeQTL.out
    Untracked:  code/run_getAPAfromeQTL_version6.7.sh
    Untracked:  code/run_getApaPval4eQTLs.err
    Untracked:  code/run_getApaPval4eQTLs.out
    Untracked:  code/run_getApaPval4eQTLsunexplained.err
    Untracked:  code/run_getApaPval4eQTLsunexplained.out
    Untracked:  code/run_getApaPval4eqtl.sh
    Untracked:  code/run_getapafromeQTL.sh
    Untracked:  code/run_getapapval4eqtl_unexp.sh
    Untracked:  code/run_leafcutterDiffIso.sh
    Untracked:  code/run_leafcutter_ds.err
    Untracked:  code/run_leafcutter_ds.out
    Untracked:  code/run_prxySNP.sh
    Untracked:  code/run_pttfacetboxplot.sh
    Untracked:  code/run_qtlFacetBoxplots.sh
    Untracked:  code/run_sepUsagephen.sh
    Untracked:  code/run_sepgenobychrom.err
    Untracked:  code/run_sepgenobychrom.out
    Untracked:  code/run_sepgenobychrom.sh
    Untracked:  code/run_sepusage.err
    Untracked:  code/run_sepusage.out
    Untracked:  code/run_verifybam.err
    Untracked:  code/run_verifybam.out
    Untracked:  code/run_verifybam.sh
    Untracked:  code/run_verifybam128N.err
    Untracked:  code/run_verifybam128N.out
    Untracked:  code/run_verifybam128T.err
    Untracked:  code/run_verifybam128T.out
    Untracked:  code/run_verifybam517N.err
    Untracked:  code/run_verifybam517N.out
    Untracked:  code/run_verifybam517T.err
    Untracked:  code/run_verifybam517T.out
    Untracked:  code/run_verifybam_fullVCF.sh
    Untracked:  code/runprxySNP.err
    Untracked:  code/runprxySNP.out
    Untracked:  code/runres2pas.err
    Untracked:  code/runres2pas.out
    Untracked:  code/selectNominalPvalues.py
    Untracked:  code/sepUsagePhen.py
    Untracked:  code/sepgenobychrom.py
    Untracked:  code/seqQTLfastq.err
    Untracked:  code/seqQTLfastq.out
    Untracked:  code/seqQTLregion.err
    Untracked:  code/seqQTLregion.out
    Untracked:  code/snakePASlog.out
    Untracked:  code/snakefiltPASlog.out
    Untracked:  code/sortindexRNABam.err
    Untracked:  code/sortindexRNABam.out
    Untracked:  code/sortindexRNAbam.sh
    Untracked:  code/subsetAPAnotEorPgene.py
    Untracked:  code/subsetAPAnotEorPgene_2versions.py
    Untracked:  code/subsetApanoteGene.py
    Untracked:  code/subsetApanoteGene_2versions.py
    Untracked:  code/subsetUnexplainedeQTLs.py
    Untracked:  code/subsetVCF_SS.sh
    Untracked:  code/subsetVCF_noSSregions.sh
    Untracked:  code/subsetVCF_upstreamPAS.sh
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subsetpermAPAwithGenelist.py
    Untracked:  code/subsetpermAPAwithGenelist_2versions.py
    Untracked:  code/subsetvcf_SS.err
    Untracked:  code/subsetvcf_SS.out
    Untracked:  code/subsetvcf_noSS.err
    Untracked:  code/subsetvcf_noSS.out
    Untracked:  code/subsetvcf_otherreg.sh
    Untracked:  code/subsetvcf_pas.err
    Untracked:  code/subsetvcf_pas.out
    Untracked:  code/subsetvcf_perm.err
    Untracked:  code/subsetvcf_perm.out
    Untracked:  code/subsetvcf_permSS.sh
    Untracked:  code/subsetvcf_rand.err
    Untracked:  code/subsetvcf_rand.out
    Untracked:  code/subtract5UTR.err
    Untracked:  code/subtract5UTR.out
    Untracked:  code/subtractExons.err
    Untracked:  code/subtractExons.out
    Untracked:  code/subtractExons.sh
    Untracked:  code/subtractfiveprimeUTR.sh
    Untracked:  code/tabixSNPS.sh
    Untracked:  code/tabixSNPs.err
    Untracked:  code/tabixSNPs.out
    Untracked:  code/tenBPupstreamPAS.py
    Untracked:  code/testVerifyBam.sh
    Untracked:  code/test_verifybam.err
    Untracked:  code/test_verifybam.out
    Untracked:  code/totSeceffectsize.py
    Untracked:  code/transcriptdm2bed.py
    Untracked:  code/twentyBPupstreamPAS.py
    Untracked:  code/utrdms2saf.py
    Untracked:  code/vcf2bed.py
    Untracked:  code/vcf_keepsnps.err
    Untracked:  code/vcf_keepsnps.out
    Untracked:  code/verifyBam18517N.sh
    Untracked:  code/verifyBam18517T.sh
    Untracked:  code/verifyBam19128N.sh
    Untracked:  code/verifyBam19128T.sh
    Untracked:  code/wrap_verifybam.err
    Untracked:  code/wrap_verifybam.out
    Untracked:  code/wrap_verifybam.sh
    Untracked:  code/wrap_verifybam_full.sh
    Untracked:  code/writeExampleQTLcode.py
    Untracked:  code/writePTTexamplecode.py
    Untracked:  code/zipandtabPhen.err
    Untracked:  code/zipandtabPhen.out
    Untracked:  data/._.DS_Store
    Untracked:  data/._MetaDataSequencing.txt
    Untracked:  data/AnnotatedPAS/
    Untracked:  data/ApaByEgene/
    Untracked:  data/ApaByPgene/
    Untracked:  data/BadLines/
    Untracked:  data/Battle_pQTL/
    Untracked:  data/CheckSums/
    Untracked:  data/CompareOldandNew/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffIso/
    Untracked:  data/EncodeRNA/
    Untracked:  data/ExampleQTLPlots/
    Untracked:  data/FlaggedPAS/
    Untracked:  data/GWAS_overlap/
    Untracked:  data/GeuvadisRNA/
    Untracked:  data/HMMqtls/
    Untracked:  data/Li_eQTLs/
    Untracked:  data/NascentRNA/
    Untracked:  data/NucSpeceQTLeffect/
    Untracked:  data/PAS/
    Untracked:  data/PAS_postFlag/
    Untracked:  data/PolyA_DB/
    Untracked:  data/PreTerm_pheno/
    Untracked:  data/PrematureQTLNominal/
    Untracked:  data/PrematureQTLPermuted/
    Untracked:  data/QTLGenotypes/
    Untracked:  data/QTLoverlap/
    Untracked:  data/QTLoverlap_nonNorm/
    Untracked:  data/README.md
    Untracked:  data/RNAseq/
    Untracked:  data/Reads2UTR/
    Untracked:  data/SNPinSS/
    Untracked:  data/SignalSiteFiles/
    Untracked:  data/TF_motifdisruption/
    Untracked:  data/ThirtyNineIndQtl_nominal/
    Untracked:  data/Version15bp6As/
    Untracked:  data/Version15bp7As/
    Untracked:  data/apaQTLNominal/
    Untracked:  data/apaQTLNominal_4pc/
    Untracked:  data/apaQTLPermuted/
    Untracked:  data/apaQTLPermuted_4pc/
    Untracked:  data/apaQTLs/
    Untracked:  data/assignedPeaks/
    Untracked:  data/assignedPeaks_15Up/
    Untracked:  data/bam/
    Untracked:  data/bam_clean/
    Untracked:  data/bam_waspfilt/
    Untracked:  data/bed_10up/
    Untracked:  data/bed_clean/
    Untracked:  data/bed_clean_sort/
    Untracked:  data/bed_waspfilter/
    Untracked:  data/bedsort_waspfilter/
    Untracked:  data/bothFrac_FC/
    Untracked:  data/bw/
    Untracked:  data/bw_norm/
    Untracked:  data/eQTLs/
    Untracked:  data/exampleQTLs/
    Untracked:  data/fastq/
    Untracked:  data/filterPeaks/
    Untracked:  data/fourSU/
    Untracked:  data/h3k27ac/
    Untracked:  data/highdiffsiggenes.txt
    Untracked:  data/inclusivePeaks/
    Untracked:  data/inclusivePeaks_FC/
    Untracked:  data/intronRNAratio/
    Untracked:  data/intron_analysis/
    Untracked:  data/locusZoom/
    Untracked:  data/mergedBG/
    Untracked:  data/mergedBW_byfrac/
    Untracked:  data/mergedBW_norm/
    Untracked:  data/mergedBam/
    Untracked:  data/mergedbyFracBam/
    Untracked:  data/molPhenos/
    Untracked:  data/molQTLs/
    Untracked:  data/motifdistrupt/
    Untracked:  data/netseq/
    Untracked:  data/nonNorm_pheno/
    Untracked:  data/nuc_10up/
    Untracked:  data/nuc_10upclean/
    Untracked:  data/oldPASfiles/
    Untracked:  data/overlapeQTL_try2/
    Untracked:  data/overlapeQTLs/
    Untracked:  data/pQTLoverlap/
    Untracked:  data/pacbio/
    Untracked:  data/peakCoverage/
    Untracked:  data/peaks_5perc/
    Untracked:  data/phenotype/
    Untracked:  data/phenotype_5perc/
    Untracked:  data/pttQTL/
    Untracked:  data/pttQTLplots/
    Untracked:  data/sigDiffGenes.txt
    Untracked:  data/sort/
    Untracked:  data/sort_clean/
    Untracked:  data/sort_waspfilter/
    Untracked:  data/twoMech/
    Untracked:  data/verifyBAM/
    Untracked:  data/verifyBAM_full/
    Untracked:  docs/._.DS_Store
    Untracked:  docs/figure/._.DS_Store
    Untracked:  nohup.out
    Untracked:  output/._.DS_Store
    Untracked:  output/._meanCorrelationPhenotypes.svg
    Untracked:  output/dtPlots/
    Untracked:  output/fastqc/
    Untracked:  output/meanCorrelationPhenotypes.svg
    Untracked:  run_verifybam517N.err
    Untracked:  run_verifybam517N.out

Unstaged changes:
    Modified:   analysis/Readdistagainstfeatures.Rmd
    Modified:   analysis/overlapapaqtlsandeqtls.Rmd
    Modified:   analysis/version15bpfilter.Rmd
    Modified:   code/BothFracDTPlotGeneRegions.sh
    Modified:   code/Snakefile
    Modified:   code/SnakefilefiltPAS
    Modified:   code/apaQTLCorrectPvalMakeQQ.R
    Modified:   code/apaQTL_Nominal.sh
    Modified:   code/apaQTL_permuted.sh
    Modified:   code/apaQTLsnake.err
    Modified:   code/bam2bw.sh
    Modified:   code/bed2saf.py
    Modified:   code/cluster.json
    Modified:   code/clusterfiltPAS.json
    Modified:   code/config.yaml
    Modified:   code/environment.yaml
    Modified:   code/makePheno.py
    Modified:   code/mergeAllBam.sh
    Modified:   code/mergeByFracBam.sh
    Modified:   code/mergePeaks.sh
    Modified:   code/peakFC.sh
    Modified:   code/snakemake.batch
    Modified:   code/snakemakePAS.batch
    Modified:   code/snakemakefiltPAS.batch
    Modified:   code/submit-snakemake.sh
    Modified:   code/submit-snakemakePAS.sh
    Modified:   code/submit-snakemakefiltPAS.sh
    Deleted:    code/test.txt
    Modified:   data/MetaDataSequencing.txt
    Deleted:    docs/figure/DiffIsoAnalysis.Rmd/figure1Emain-1.pdf
    Deleted:    docs/figure/DiffIsoAnalysis.Rmd/figure1Esubset-1.pdf
    Deleted:    reads_graphs.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/15fbb3c19335788121e18ba26d659dc8f2d0b757/docs/DiffIsoAnalysis.html" target="_blank">15fbb3c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/aa1ad854d2d0e0947dc13175ad852117b0c30e47/analysis/DiffIsoAnalysis.Rmd" target="_blank">aa1ad85</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-31
</td>
<td>
add pdf for figrues
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/c5bf3fd840a86608c59e24fad98d4fdde02b7d16/docs/DiffIsoAnalysis.html" target="_blank">c5bf3fd</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/36d86c088e03cef75dec513361d07751f157c6ed/analysis/DiffIsoAnalysis.Rmd" target="_blank">36d86c0</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-02
</td>
<td>
post LM plot midifications
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/DiffIsoAnalysis.html" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/494ab8a3e91b36a1a24f9a88541314ac0aabcdc4/analysis/DiffIsoAnalysis.Rmd" target="_blank">494ab8a</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
<td>
add diff prop test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/499e504c5c6b608cf71e42d8396e29c35768c58e/docs/DiffIsoAnalysis.html" target="_blank">499e504</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/97e2ea836e419626a444e3d9d8fd65c0cafc93c3/analysis/DiffIsoAnalysis.Rmd" target="_blank">97e2ea8</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-22
</td>
<td>
add pie chart
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/6679c9507ae4f542b9d32e7f499d6a5e4f6da7f6/docs/DiffIsoAnalysis.html" target="_blank">6679c95</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/842be2582d36923230482d53ba372dfb9a46f08a/analysis/DiffIsoAnalysis.Rmd" target="_blank">842be25</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
<td>
fix fif
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/4f2326e310ffaf47c6a9a6afc54e0985dacf93aa/docs/DiffIsoAnalysis.html" target="_blank">4f2326e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/abd1a7371da754280383b89ff56f54244375c8cb/analysis/DiffIsoAnalysis.Rmd" target="_blank">abd1a73</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
<td>
fix figures
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/ae5c5a1a01a0f17c7e3e933df8ca97fb766903b5/docs/DiffIsoAnalysis.html" target="_blank">ae5c5a1</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/0d606c1c66742bf9bd29941523cd5e4b7cbc2d06/analysis/DiffIsoAnalysis.Rmd" target="_blank">0d606c1</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
<td>
fix figures
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/2d1a80c01ce5b56d1de2e91a768f77c5a154d9bb/docs/DiffIsoAnalysis.html" target="_blank">2d1a80c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/8944f90eb9e88724cf62a9e84ff9c45831bdfd20/analysis/DiffIsoAnalysis.Rmd" target="_blank">8944f90</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-16
</td>
<td>
fix effect size header
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/DiffIsoAnalysis.html" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/17955ab19295687f6dda0c7dc363054ec0865b5c/analysis/DiffIsoAnalysis.Rmd" target="_blank">17955ab</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
<td>
fix big bug
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/b6ed10c6f91f12894520b7190113460051f9f95d/docs/DiffIsoAnalysis.html" target="_blank">b6ed10c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/312d7d77fc2a30ad1bb85f5cf1f49fb08dce4e38/analysis/DiffIsoAnalysis.Rmd" target="_blank">312d7d7</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
<td>
add non facet plot
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/bf3a1e0ba9cbfc39facbabc357f4e690c7a05dda/docs/DiffIsoAnalysis.html" target="_blank">bf3a1e0</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/77ca26ac20a8a70f3ad011f8dbe1436c3568110b/analysis/DiffIsoAnalysis.Rmd" target="_blank">77ca26a</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-14
</td>
<td>
results by logef
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/760b297cc72009dd3a7bac9e7d15aed91a79f06c/docs/DiffIsoAnalysis.html" target="_blank">760b297</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/4c10e8f3402fa8b01e07005321ea18e0b28f6d87/analysis/DiffIsoAnalysis.Rmd" target="_blank">4c10e8f</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-14
</td>
<td>
add dist to PAS plot
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/d0aa6a390125f83f875919a531383e48eb6fe8d2/docs/DiffIsoAnalysis.html" target="_blank">d0aa6a3</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/f514b6e9efaf5153fb3d938c9a275b11aa5ffbc4/analysis/DiffIsoAnalysis.Rmd" target="_blank">f514b6e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-13
</td>
<td>
add combined plot
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/07c912574a6c40eb69bd055525665ba0ec9a3e10/docs/DiffIsoAnalysis.html" target="_blank">07c9125</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/981ac33005293ea9e74d3abcdfc28d0c0ccfeefb/analysis/DiffIsoAnalysis.Rmd" target="_blank">981ac33</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-13
</td>
<td>
add location of highly used
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/c561b14b9fe2e50426d2198114f12a3f21044c95/docs/DiffIsoAnalysis.html" target="_blank">c561b14</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/1d8a0a3e88aa036b1b965a6ca54efe058c2d2a1f/analysis/DiffIsoAnalysis.Rmd" target="_blank">1d8a0a3</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-06
</td>
<td>
add res
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/60093ce5e545fc47f274a3f3f76d2bc0c8c93cc1/docs/DiffIsoAnalysis.html" target="_blank">60093ce</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/24c2cebc7e4b061243ebb5b04abce358a1035334/analysis/DiffIsoAnalysis.Rmd" target="_blank">24c2ceb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-02
</td>
<td>
add diff iso
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(workflowr)</code></pre>
<pre><code>This is workflowr version 1.4.0
Run ?workflowr for help getting started</code></pre>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>── Attaching packages ────────────────────────────────────────────────────────────────────────────────────── tidyverse 1.2.1 ──</code></pre>
<pre><code>✔ ggplot2 3.1.1       ✔ purrr   0.3.2  
✔ tibble  2.1.1       ✔ dplyr   0.8.0.1
✔ tidyr   0.8.3       ✔ stringr 1.3.1  
✔ readr   1.3.1       ✔ forcats 0.3.0  </code></pre>
<pre><code>── Conflicts ───────────────────────────────────────────────────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(reshape2)</code></pre>
<pre><code>
Attaching package: &#39;reshape2&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:tidyr&#39;:

    smiths</code></pre>
<p>In this analysis I wil use leafcutter to call PAS with differential ussage between fractions.</p>
<div id="prepare-annotation" class="section level2">
<h2>Prepare annotation</h2>
<p>I first filter the annotated peak SAF file for peaks passing the 5% coverage in either fraction.</p>
<pre class="bash"><code>python makeSAFbothfrac5perc.py</code></pre>
</div>
<div id="peak-quantification" class="section level2">
<h2>Peak quantification</h2>
<pre class="bash"><code>mkdir bothFrac_FC</code></pre>
<p>Run feature counts with these peaks with both fractions:</p>
<pre class="bash"><code>sbatch bothFrac_FC.sh</code></pre>
<p>Fix the header:</p>
<pre class="bash"><code>python fixFChead_bothfrac.py ../data/bothFrac_FC/APApeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.5percCov.bothfrac.fc ../data/bothFrac_FC/APApeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.5percCov.bothfrac.fixed.fc</code></pre>
<p>Remove location demoniaiton:</p>
</div>
<div id="prepare-leafcutter-phenotype" class="section level1">
<h1>Prepare leafcutter phenotype</h1>
<pre class="bash"><code>mkdir ../data/DiffIso</code></pre>
<pre class="bash"><code>python fc2leafphen.py</code></pre>
<p>Fix pheno to remove location:</p>
<pre class="bash"><code>python removeloc_pheno.py ../data/DiffIso/APApeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.5percCov.bothfrac.fixed.forLC.fc ../data/DiffIso/APApeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.5percCov.bothfrac.fixed.forLC_noloc.fc</code></pre>
<pre class="bash"><code>python subset_diffisopheno.py 1
python subset_diffisopheno.py 2
python subset_diffisopheno.py 3
python subset_diffisopheno.py 4
python subset_diffisopheno.py 5
python subset_diffisopheno.py 6
python subset_diffisopheno.py 7
python subset_diffisopheno.py 8
python subset_diffisopheno.py 9
python subset_diffisopheno.py 10
python subset_diffisopheno.py 11
python subset_diffisopheno.py 12
python subset_diffisopheno.py 13
python subset_diffisopheno.py 14
python subset_diffisopheno.py 15
python subset_diffisopheno.py 16
python subset_diffisopheno.py 18
python subset_diffisopheno.py 19
python subset_diffisopheno.py 20
python subset_diffisopheno.py 21
python subset_diffisopheno.py 22</code></pre>
<p>Make the sample groups file:</p>
<pre class="bash"><code>python LC_samplegroups.py </code></pre>
<div id="run-leafcutter" class="section level2">
<h2>Run leafcutter</h2>
<p>The leafcutter environment is not in the three-prime-seq environment. Make sure leafcutter is installed and working.</p>
<pre class="bash"><code>sbatch run_leafcutterDiffIso.sh</code></pre>
<p>Rscript /project2/gilad/briana/davidaknowles-leafcutter-c3d9474/scripts/leafcutter_ds.R –num_threads 4 ../data/DiffIso/APApeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.5percCov.bothfrac.fixed.forLC.fc_22.txt ../data/bothFrac_FC/sample_groups.txt -o ../data/DiffIso/TN_diff_isoform_chr22.txt</p>
<p>Concatinate results:</p>
<pre class="bash"><code>awk &#39;{if(NR&gt;1)print}&#39; ../data/DiffIso/TN_diff_isoform_chr*.txt_effect_sizes.txt &gt; ../data/DiffIso/TN_diff_isoform_allChrom.txt_effect_sizes.txt


awk &#39;{if(NR&gt;1)print}&#39; ../data/DiffIso/TN_diff_isoform_chr*.txt_cluster_significance.txt &gt; ../data/DiffIso/TN_diff_isoform_AllChrom_cluster_significance.txt</code></pre>
</div>
<div id="evaluate-results" class="section level2">
<h2>Evaluate results</h2>
<div id="significant-clusters" class="section level3">
<h3>Significant clusters</h3>
<pre class="r"><code>sig=read.table(&quot;../data/DiffIso/TN_diff_isoform_AllChrom_cluster_significance.txt&quot;,sep=&quot;\t&quot; ,col.names = c(&#39;status&#39;,&#39;loglr&#39;,&#39;df&#39;,&#39;p&#39;,&#39;cluster&#39;,&#39;p.adjust&#39;),stringsAsFactors = F) %&gt;% filter(status==&quot;Success&quot;) 

sig$p.adjust=as.numeric(as.character(sig$p.adjust))</code></pre>
<pre class="r"><code>qqplot(-log10(runif(nrow(sig))), -log10(sig$p.adjust),ylab=&quot;-log10 Total Adjusted Leafcutter pvalue&quot;, xlab=&quot;-log 10 Uniform expectation&quot;, main=&quot;Leafcutter differencial isoform analysis between fractions&quot;)
abline(0,1)</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-1">
Past versions of unnamed-chunk-14-1.png
</button>
</p>
<div id="fig-unnamed-chunk-14-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-14-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/c561b14b9fe2e50426d2198114f12a3f21044c95/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-14-1.png" target="_blank">c561b14</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>tested_genes=nrow(sig)
tested_genes</code></pre>
<pre><code>[1] 9221</code></pre>
<pre class="r"><code>sig_genes=sig %&gt;% filter(p.adjust&lt;.05)
number_sig_genes=nrow(sig_genes)
number_sig_genes</code></pre>
<pre><code>[1] 6946</code></pre>
<pre class="r"><code>sig_genesonly=sig_genes %&gt;% separate(cluster,into=c(&quot;chrom&quot;, &quot;geneName&quot;), sep = &quot;:&quot;) %&gt;% dplyr::select(geneName)

write.table(sig_genesonly, file=&quot;../data/sigDiffGenes.txt&quot;, col.names = T, row.names = F, quote = F)</code></pre>
</div>
<div id="effect-sizes" class="section level3">
<h3>Effect sizes</h3>
<pre class="r"><code>effectsize=read.table(&quot;../data/DiffIso/TN_diff_isoform_allChrom.txt_effect_sizes.txt&quot;, stringsAsFactors = F, col.names=c(&#39;intron&#39;,  &#39;logef&#39; ,&#39;Nuclear&#39;, &#39;Total&#39;,&#39;deltaPAU&#39;)) %&gt;% filter(intron != &quot;intron&quot;)

write.table(effectsize,file=&quot;../data/DiffIso/EffectSizes.txt&quot;, quote = F, col.names = T, row.names = F)

effectsize$deltaPAU=as.numeric(as.character(effectsize$deltaPAU))
effectsize$logef=as.numeric(as.character(effectsize$logef))</code></pre>
<p>Plot delta PAU:</p>
<pre class="r"><code>plot(sort(effectsize$deltaPAU),main=&quot;Leafcutter delta PAU&quot;, ylab=&quot;Delta PAU&quot;, xlab=&quot;PAS Index&quot;)</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-18-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-18-1">
Past versions of unnamed-chunk-18-1.png
</button>
</p>
<div id="fig-unnamed-chunk-18-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-18-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/c561b14b9fe2e50426d2198114f12a3f21044c95/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-18-1.png" target="_blank">c561b14</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Filter PAU &gt; .2</p>
<pre class="r"><code>effectsize_deltaPAU= effectsize %&gt;% filter(abs(deltaPAU) &gt; .2) 
nrow(effectsize_deltaPAU)</code></pre>
<pre><code>[1] 1764</code></pre>
<pre class="r"><code>effectSize_highdiffGenes=effectsize_deltaPAU %&gt;% separate(intron, into=c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;GeneName&quot;), sep=&quot;:&quot;) %&gt;% dplyr::select(GeneName) %&gt;% unique()


write.table(effectSize_highdiffGenes, file=&quot;../data/highdiffsiggenes.txt&quot;, col.names = F, row.names = F, quote = F)</code></pre>
<p>Genes in this set:</p>
<pre class="r"><code>effectsize_deltaPAU_Genes= effectsize_deltaPAU %&gt;% separate(intron, into=c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;,&quot;gene&quot;),sep=&quot;:&quot;) %&gt;% group_by(gene) %&gt;% summarise(nperGene=n()) 

nrow(effectsize_deltaPAU_Genes)</code></pre>
<pre><code>[1] 1334</code></pre>
<p>Filter &gt;.2 in</p>
<pre class="r"><code>effectsize_deltaPAU_nuclear= effectsize_deltaPAU %&gt;% filter(deltaPAU &lt; -0.2)

#write out at bed
#need strand info
PAS=read.table(&quot;../data/PAS/APAPAS_GeneLocAnno.5perc.bed&quot;, stringsAsFactors = F,col.names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;peak&quot;, &quot;score&quot;, &quot;strand&quot;) )%&gt;% separate(peak, into=c(&quot;peaknum&quot;,&quot;peakID&quot;), sep=&quot;:&quot;) %&gt;% separate(peakID, into=c(&quot;gene&quot;, &quot;loc&quot;), sep=&quot;_&quot;) %&gt;% dplyr::select(gene, strand) %&gt;% unique()
effectsize_deltaPAU_nuclear_bed=effectsize_deltaPAU_nuclear %&gt;% separate(intron, into=c(&quot;chr&quot;, &quot;peakStart&quot;, &quot;peakEnd&quot;, &quot;gene&quot;), sep=&quot;:&quot;) %&gt;% inner_join(PAS, by=&quot;gene&quot;)  %&gt;% mutate(PASstart=ifelse(strand==&quot;+&quot;, as.integer(peakEnd)-1, as.integer(peakStart)+1)) %&gt;% mutate(PASend=ifelse(strand==&quot;+&quot;, as.integer(peakEnd), as.integer(peakStart))) %&gt;% mutate(score=&quot;.&quot;) %&gt;%  dplyr::select(chr, peakStart, peakEnd, gene, score, strand) 

write.table(effectsize_deltaPAU_nuclear_bed, file=&quot;../data/PAS/UsedMoreNuclearPAU2.bed&quot;, col.names = F, row.names = F, quote = F,sep = &quot;\t&quot;)</code></pre>
<p>Filter &gt;.2 in Total:</p>
<pre class="r"><code>effectsize_deltaPAU_total= effectsize_deltaPAU %&gt;% filter(deltaPAU &gt; 0.2)

effectsize_deltaPAU_total_bed=effectsize_deltaPAU_total %&gt;% separate(intron, into=c(&quot;chr&quot;, &quot;peakStart&quot;, &quot;peakEnd&quot;, &quot;gene&quot;), sep=&quot;:&quot;) %&gt;% inner_join(PAS, by=&quot;gene&quot;)  %&gt;% mutate(PASstart=ifelse(strand==&quot;+&quot;, as.integer(peakEnd)-1, as.integer(peakStart)+1)) %&gt;% mutate(PASend=ifelse(strand==&quot;+&quot;, as.integer(peakEnd), as.integer(peakStart))) %&gt;% mutate(score=&quot;.&quot;) %&gt;%  dplyr::select(chr, peakStart, peakEnd, gene, score, strand) 

write.table(effectsize_deltaPAU_total_bed, file=&quot;../data/PAS/UsedMoreTotalPAU2.bed&quot;, col.names = F, row.names = F, quote = F,sep=&quot;\t&quot;)</code></pre>
<p>Sort the files:</p>
<pre class="bash"><code>sort -k1,1 -k2,2n ../data/PAS/UsedMoreTotalPAU2.bed &gt; ../data/PAS/UsedMoreTotalPAU2.sort.bed
sort -k1,1 -k2,2n ../data/PAS/UsedMoreNuclearPAU2.bed &gt; ../data/PAS/UsedMoreNuclearPAU2.sort.bed</code></pre>
</div>
</div>
<div id="location-of-high-pau" class="section level2">
<h2>Location of high &gt;PAU</h2>
<div id="total" class="section level3">
<h3>Total:</h3>
<p>Pull in location information for each PAS:</p>
<pre class="r"><code>PAS=read.table(&quot;../data/peaks_5perc/APApeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.5percCov.bothfrac.SAF&quot;,stringsAsFactors = F,header = T) %&gt;% separate(GeneID, into=c(&quot;num&quot;, &quot;chr&quot;, &quot;start&quot;, &quot;end&quot;, &quot;strand&quot;, &quot;geneID&quot;), sep=&quot;:&quot;) %&gt;% separate(geneID, into=c(&quot;gene&quot;, &quot;loc&quot;),sep=&quot;_&quot;) %&gt;%  mutate(intron=paste(&quot;chr&quot;, Chr, &quot;:&quot;, Start, &quot;:&quot;, End, &quot;:&quot;, gene,sep=&quot;&quot;)) %&gt;% select(intron, loc)</code></pre>
<pre class="r"><code>effectsize_deltaPAU_total_loc=effectsize_deltaPAU_total %&gt;% inner_join(PAS, by=&quot;intron&quot;) 


ggplot(effectsize_deltaPAU_total_loc,aes(x=loc)) + geom_histogram(stat=&quot;count&quot;) + labs(title=&quot;Location of Total peaks &gt;.2 PAU&quot;) </code></pre>
<pre><code>Warning: Ignoring unknown parameters: binwidth, bins, pad</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-25-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-25-1">
Past versions of unnamed-chunk-25-1.png
</button>
</p>
<div id="fig-unnamed-chunk-25-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-25-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/07c912574a6c40eb69bd055525665ba0ec9a3e10/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-25-1.png" target="_blank">07c9125</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="nuclear" class="section level3">
<h3>Nuclear:</h3>
<pre class="r"><code>effectsize_deltaPAU_nuclear_loc=effectsize_deltaPAU_nuclear %&gt;% inner_join(PAS, by=&quot;intron&quot;) 


ggplot(effectsize_deltaPAU_nuclear_loc,aes(x=loc)) + geom_histogram(stat=&quot;count&quot;) + labs(title=&quot;Location of Nuclear peaks &gt;.2 PAU&quot;)</code></pre>
<pre><code>Warning: Ignoring unknown parameters: binwidth, bins, pad</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-26-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-26-1">
Past versions of unnamed-chunk-26-1.png
</button>
</p>
<div id="fig-unnamed-chunk-26-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-26-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>I will want to look at proportions. I need to know how many peaks are in each location:</p>
<pre class="r"><code>PAS_loc =PAS%&gt;% group_by(loc) %&gt;% summarise(nloc=n())</code></pre>
<pre class="r"><code>effectsize_deltaPAU_total_locProp=effectsize_deltaPAU_total_loc %&gt;% group_by(loc) %&gt;% summarise(nloctotal=n()) 
effectsize_deltaPAU_nuclear_locProp=effectsize_deltaPAU_nuclear_loc %&gt;% group_by(loc) %&gt;% summarise(nlocnuclear=n()) 

effectsize_deltaPAUProp_tot=effectsize_deltaPAU_total_locProp %&gt;% inner_join(PAS_loc, by=&quot;loc&quot;) %&gt;% mutate(Proportion_tot=nloctotal/nloc)

effectsize_deltaPAUProp_nuc=effectsize_deltaPAU_nuclear_locProp %&gt;% inner_join(PAS_loc, by=&quot;loc&quot;) %&gt;% mutate(Proportion_nuc=nlocnuclear/nloc)</code></pre>
<pre class="r"><code>ggplot(effectsize_deltaPAUProp_tot, aes(x=loc, y=Proportion_tot)) + geom_bar(stat=&quot;identity&quot;) + labs(y=&quot;Proportion of all called PAS&quot;, title=&quot;Location of high Total used PAS&quot;)</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-29-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-29-1">
Past versions of unnamed-chunk-29-1.png
</button>
</p>
<div id="fig-unnamed-chunk-29-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-29-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/07c912574a6c40eb69bd055525665ba0ec9a3e10/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-29-1.png" target="_blank">07c9125</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggplot(effectsize_deltaPAUProp_nuc, aes(x=loc, y=Proportion_nuc)) + geom_bar(stat=&quot;identity&quot;) + labs(y=&quot;Proportion of all called PAS&quot;, title=&quot;Location of high nuclear used PAS&quot;)</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-30-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-30-1">
Past versions of unnamed-chunk-30-1.png
</button>
</p>
<div id="fig-unnamed-chunk-30-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-30-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Merge to 1 figure:</p>
<pre class="r"><code>effectsize_deltaPAUProp_both= effectsize_deltaPAUProp_nuc %&gt;% inner_join(effectsize_deltaPAUProp_tot, by=c(&quot;loc&quot;,&quot;nloc&quot;)) %&gt;% dplyr::rename(Nuclear=Proportion_nuc, Total=Proportion_tot) %&gt;% select(loc, Nuclear, Total) 
effectsize_deltaPAUProp_both_melt= effectsize_deltaPAUProp_both %&gt;% melt(id.vars=&quot;loc&quot;, variable.name=&quot;Fraction&quot;, value.name = &quot;Proportion&quot;) 
effectsize_deltaPAUProp_both_melt$Fraction=as.character(effectsize_deltaPAUProp_both_melt$Fraction)</code></pre>
<pre class="r"><code>ggplot(effectsize_deltaPAUProp_both_melt, aes(x=loc, y=Proportion, by=Fraction, fill=Fraction)) + geom_bar(stat=&quot;identity&quot;, position=&quot;dodge&quot;) + scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;)) + labs(title=&quot;Proportion of PAS differential used by location&quot;,x=&quot;&quot;) +scale_x_discrete(labels = c(&#39;Coding&#39;,&#39;5kb downstream&#39;,&#39;Intronic&#39;,&quot;3&#39; UTR&quot;, &quot;5&#39; UTR&quot;)) +theme(axis.text.x = element_text(angle = 90, hjust = 1)) +  theme(legend.position = c(0.1,.9), legend.direction = &quot;horizontal&quot;) +  theme(panel.background = element_blank()) </code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-32-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-32-1">
Past versions of unnamed-chunk-32-1.png
</button>
</p>
<div id="fig-unnamed-chunk-32-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/6679c9507ae4f542b9d32e7f499d6a5e4f6da7f6/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-32-1.png" target="_blank">6679c95</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/4f2326e310ffaf47c6a9a6afc54e0985dacf93aa/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-32-1.png" target="_blank">4f2326e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ae5c5a1a01a0f17c7e3e933df8ca97fb766903b5/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-32-1.png" target="_blank">ae5c5a1</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-32-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>effectsize_deltaPAU_total_locProp</code></pre>
<pre><code># A tibble: 5 x 2
  loc    nloctotal
  &lt;chr&gt;      &lt;int&gt;
1 cds           30
2 end           63
3 intron       126
4 utr3         916
5 utr5          38</code></pre>
<pre class="r"><code>sum(effectsize_deltaPAU_total_locProp$nloctotal)</code></pre>
<pre><code>[1] 1173</code></pre>
<pre class="r"><code>effectsize_deltaPAU_nuclear_locProp</code></pre>
<pre><code># A tibble: 5 x 2
  loc    nlocnuclear
  &lt;chr&gt;        &lt;int&gt;
1 cds              9
2 end             83
3 intron         387
4 utr3           101
5 utr5            11</code></pre>
<pre class="r"><code>sum(effectsize_deltaPAU_nuclear_locProp$nlocnuclear)</code></pre>
<pre><code>[1] 591</code></pre>
<pre class="r"><code>effectsize_deltaPAUProp_both_melt_sm=effectsize_deltaPAUProp_both_melt %&gt;% filter(loc==&quot;intron&quot; | loc==&quot;utr3&quot;)


ggplot(effectsize_deltaPAUProp_both_melt_sm, aes(x=loc, y=Proportion, by=Fraction, fill=Fraction)) + geom_bar(stat=&quot;identity&quot;, position=&quot;dodge&quot;) + scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;)) + labs(title=&quot;Differencially Used PAS&quot;,x=&quot;&quot;, y=&quot;Proportion of All PAS at Location&quot;) +scale_x_discrete(labels = c(&#39;Intronic&#39;,&quot;3&#39; UTR&quot;)) +theme(axis.text.x = element_text(angle = 45, hjust = 1)) +  theme(legend.position =&quot;bottom&quot;, legend.direction = &quot;horizontal&quot;) +  theme(panel.background = element_blank(),text = element_text(size=16), plot.title = element_text(size = 20, face = &quot;bold&quot;),axis.text.x = element_text(size=14),axis.text.y = element_text(size=14))</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-34-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-34-1">
Past versions of unnamed-chunk-34-1.png
</button>
</p>
<div id="fig-unnamed-chunk-34-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/15fbb3c19335788121e18ba26d659dc8f2d0b757/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-34-1.png" target="_blank">15fbb3c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-31
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-34-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/760b297cc72009dd3a7bac9e7d15aed91a79f06c/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-34-1.png" target="_blank">760b297</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>effectsize_deltaPAUProp_both_melt_sm</code></pre>
<pre><code>     loc Fraction  Proportion
1 intron  Nuclear 0.030250918
2   utr3  Nuclear 0.004996537
3 intron    Total 0.009849136
4   utr3    Total 0.045315128</code></pre>
<pre class="r"><code>#intronic
prop.test(x=c(387,126), n=c(591,1173),alternative = &quot;greater&quot;)</code></pre>
<pre><code>
    2-sample test for equality of proportions with continuity
    correction

data:  c(387, 126) out of c(591, 1173)
X-squared = 568.34, df = 1, p-value &lt; 2.2e-16
alternative hypothesis: greater
95 percent confidence interval:
 0.5106946 1.0000000
sample estimates:
   prop 1    prop 2 
0.6548223 0.1074169 </code></pre>
<pre class="r"><code>#3&#39; utr
prop.test(x=c(101,916), n=c(591,1173),alternative = &quot;less&quot;)</code></pre>
<pre><code>
    2-sample test for equality of proportions with continuity
    correction

data:  c(101, 916) out of c(591, 1173)
X-squared = 596.48, df = 1, p-value &lt; 2.2e-16
alternative hypothesis: less
95 percent confidence interval:
 -1.0000000 -0.5764348
sample estimates:
   prop 1    prop 2 
0.1708968 0.7809037 </code></pre>
<p>More differentiall used in total. this makes sense because there are more used peaks in the nuclear which evens out the distribution of the ratios.</p>
</div>
<div id="stratify-by-different-delta-pau" class="section level3">
<h3>Stratify by different <span class="math inline">\(\Delta\)</span> PAU</h3>
<p>I want to create a data frame that has the location proportion distribution based on different <span class="math inline">\(\Delta\)</span> PAU. 0-.1 .1-.2 .2-.3 .3-.4 .4-.5 &gt;.5</p>
<p>First I will seperate the total and nuclear but the sign of the <span class="math inline">\(\Delta\)</span> PAU.</p>
<pre class="r"><code>colnames(effectsize)=c(&quot;intron&quot;, &quot;logef&quot;,&quot;Nuclear&quot;, &quot;Total&quot;, &quot;deltaPAU&quot;)
Total_dpau= effectsize %&gt;% filter(deltaPAU &gt; 0) %&gt;% inner_join(PAS, by=&quot;intron&quot;) %&gt;% select(-logef, -Nuclear,-Total) %&gt;%  mutate(fraction=&quot;Total&quot;, PAU_Cat=ifelse(deltaPAU &lt;.1, &quot;&lt;.1&quot;, ifelse(deltaPAU &gt;=.1 &amp; deltaPAU &lt;.2, &quot;&lt;.2&quot;, ifelse(deltaPAU &gt;=.2 &amp; deltaPAU &lt;.3, &quot;&lt;.3&quot;, ifelse(deltaPAU &gt;=.3 &amp; deltaPAU &lt;.4, &quot;&lt;.4&quot;, &quot;&lt;.5&quot;))))) 

Nuclear_dpau= effectsize %&gt;% filter(deltaPAU &lt;0) %&gt;% inner_join(PAS, by=&quot;intron&quot;) %&gt;% select(-logef,-Nuclear,-Total) %&gt;% mutate(fraction=&quot;Nuclear&quot;, PAU_Cat=ifelse(deltaPAU &gt;-.1, &quot;&lt;.1&quot;, ifelse(deltaPAU &lt;=-.1 &amp; deltaPAU &gt; -.2, &quot;&lt;.2&quot;, ifelse(deltaPAU &lt;=-.2 &amp; deltaPAU &gt;-.3, &quot;&lt;.3&quot;, ifelse(deltaPAU &lt;=-.3 &amp; deltaPAU &gt;-.4, &quot;&lt;.4&quot;, &quot;&lt;.5&quot;)))))</code></pre>
<p>Merge these together to start grouping:</p>
<pre class="r"><code>allPAU=as.data.frame(rbind(Total_dpau, Nuclear_dpau)) %&gt;% group_by(fraction, PAU_Cat, loc ) %&gt;% summarise(nperLoc=n()) %&gt;% full_join(PAS_loc, by =&quot;loc&quot;) %&gt;% mutate(Prop=nperLoc/nloc)</code></pre>
<p>Plot it:</p>
<pre class="r"><code>ggplot(allPAU, aes(x=loc,y=Prop, group=fraction, fill=fraction)) + geom_bar(stat=&quot;identity&quot;, position = &quot;dodge&quot;) + facet_wrap(~PAU_Cat)+ scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;)) + theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title=&quot;Proportion of PAS by location and delta PAU&quot;)</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-38-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-38-1">
Past versions of unnamed-chunk-38-1.png
</button>
</p>
<div id="fig-unnamed-chunk-38-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-38-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>allPAU_remove.1= allPAU %&gt;% filter(PAU_Cat != &quot;&lt;.1&quot;)

ggplot(allPAU_remove.1, aes(x=loc,y=Prop, group=fraction, fill=fraction)) + geom_bar(stat=&quot;identity&quot;, position = &quot;dodge&quot;) + facet_wrap(~PAU_Cat)+ scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;)) + theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title=&quot;Proportion of PAS by location and delta PAU&quot;)</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-39-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-39-1">
Past versions of unnamed-chunk-39-1.png
</button>
</p>
<div id="fig-unnamed-chunk-39-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-39-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/760b297cc72009dd3a7bac9e7d15aed91a79f06c/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-39-1.png" target="_blank">760b297</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Proportion within group:</p>
<pre class="r"><code>allPAU_ingroup= allPAU %&gt;% mutate(nCat=sum(nperLoc),proppercat=nperLoc/nCat)

ggplot(allPAU_ingroup, aes(x=loc,y=proppercat, group=fraction, fill=fraction)) + geom_bar(stat=&quot;identity&quot;, position = &quot;dodge&quot;) + facet_wrap(~PAU_Cat)+ scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;)) + theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title=&quot;Proportion of PAS by location and delta PAU&quot;)</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-40-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-40-1">
Past versions of unnamed-chunk-40-1.png
</button>
</p>
<div id="fig-unnamed-chunk-40-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-40-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-40-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="distance-to-tss" class="section level2">
<h2>Distance to TSS:</h2>
<p>I need to pull in the TSS information so I can look at the distance between the differentially used peaks and by distance .</p>
<pre class="r"><code>tss=read.table(&quot;../../genome_anotation_data/refseq.ProteinCoding.bed&quot;,col.names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;) ,stringsAsFactors = F) %&gt;% mutate(TSS= ifelse(strand==&quot;+&quot;, start, end)) %&gt;% select(gene, TSS, strand)</code></pre>
<p>Seperate effect size introns:</p>
<p>PAS base for + strand is end, PAS for neg stand in -</p>
<pre class="r"><code>effectsize_TSS= effectsize %&gt;% separate(intron, into=c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;),sep=&quot;:&quot;) %&gt;% mutate(fraction=ifelse(deltaPAU &lt; 0, &quot;nuclear&quot;, &quot;total&quot;)) %&gt;% inner_join(tss, by=&quot;gene&quot;) %&gt;% mutate(dist2PAS=ifelse(strand==&quot;+&quot;, as.numeric(end)-as.numeric(TSS), as.numeric(TSS)-as.numeric(start))) 

effectsize_TSS_tot= effectsize_TSS %&gt;% filter(fraction==&quot;total&quot;) %&gt;% mutate( PAU_Cat=ifelse(deltaPAU &lt;.1, &quot;&lt;.1&quot;, ifelse(deltaPAU &gt;=.1 &amp; deltaPAU &lt;.2, &quot;&lt;.2&quot;, ifelse(deltaPAU &gt;=.2 &amp; deltaPAU &lt;.3, &quot;&lt;.3&quot;, ifelse(deltaPAU &gt;=.3 &amp; deltaPAU &lt;.4, &quot;&lt;.4&quot;, &quot;&lt;.5&quot;))))) 


effectsize_TSS_nuc=effectsize_TSS %&gt;% filter(fraction==&quot;nuclear&quot;) %&gt;% mutate( PAU_Cat=ifelse(deltaPAU &gt;-.1, &quot;&lt;.1&quot;, ifelse(deltaPAU &lt;=-.1 &amp; deltaPAU &gt; -.2, &quot;&lt;.2&quot;, ifelse(deltaPAU &lt;=-.2 &amp; deltaPAU &gt;-.3, &quot;&lt;.3&quot;, ifelse(deltaPAU &lt;=-.3 &amp; deltaPAU &gt;-.4, &quot;&lt;.4&quot;, &quot;&lt;.5&quot;)))))


effectsize_TSS_cat=as.data.frame(rbind(effectsize_TSS_tot, effectsize_TSS_nuc)) %&gt;% filter(dist2PAS &gt;0)</code></pre>
<pre class="r"><code>ggplot(effectsize_TSS_cat, aes(x=log10(dist2PAS), by=fraction, fill=fraction))+ geom_density(alpha=.4) + facet_grid(~PAU_Cat) + labs(title=&quot;Distance to TSS for differentialy used PAS&quot;)+scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;)) </code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-43-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-43-1">
Past versions of unnamed-chunk-43-1.png
</button>
</p>
<div id="fig-unnamed-chunk-43-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-43-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-43-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="by-length-of-gene" class="section level3">
<h3>By length of gene</h3>
<pre class="r"><code>length=read.table(&quot;../../genome_anotation_data/refseq.ProteinCoding.bed&quot;,col.names = c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;) ,stringsAsFactors = F) %&gt;% mutate(length=abs(end-start)) %&gt;%  mutate(TSS= ifelse(strand==&quot;+&quot;, start, end)) %&gt;% select(gene, length,TSS, strand)</code></pre>
<pre class="r"><code>effectsize_length= effectsize %&gt;% separate(intron, into=c(&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;),sep=&quot;:&quot;) %&gt;% mutate(fraction=ifelse(deltaPAU &lt; 0, &quot;nuclear&quot;, &quot;total&quot;)) %&gt;% inner_join(length, by=&quot;gene&quot;) %&gt;% mutate(PercLength=ifelse(strand==&quot;+&quot;, ((as.numeric(end)-as.numeric(TSS))/as.numeric(length)), (1-(as.numeric(start)-as.numeric(TSS))/as.numeric(length)))) 

effectsize_length_tot= effectsize_length %&gt;% filter(fraction==&quot;total&quot;) %&gt;% mutate( PAU_Cat=ifelse(deltaPAU &lt;.1, &quot;&lt;.1&quot;, ifelse(deltaPAU &gt;=.1 &amp; deltaPAU &lt;.2, &quot;&lt;.2&quot;, ifelse(deltaPAU &gt;=.2 &amp; deltaPAU &lt;.3, &quot;&lt;.3&quot;, ifelse(deltaPAU &gt;=.3 &amp; deltaPAU &lt;.4, &quot;&lt;.4&quot;, &quot;&lt;.5&quot;))))) 


effectsize_length_nuc=effectsize_length %&gt;% filter(fraction==&quot;nuclear&quot;) %&gt;% mutate( PAU_Cat=ifelse(deltaPAU &gt;-.1, &quot;&lt;.1&quot;, ifelse(deltaPAU &lt;=-.1 &amp; deltaPAU &gt; -.2, &quot;&lt;.2&quot;, ifelse(deltaPAU &lt;=-.2 &amp; deltaPAU &gt;-.3, &quot;&lt;.3&quot;, ifelse(deltaPAU &lt;=-.3 &amp; deltaPAU &gt;-.4, &quot;&lt;.4&quot;, &quot;&lt;.5&quot;)))))


effectsize_length_cat=as.data.frame(rbind(effectsize_length_tot, effectsize_length_nuc)) %&gt;% filter(PercLength&lt;=1 &amp; PercLength &gt;0)

effectsize_length_catall=as.data.frame(rbind(effectsize_length_tot, effectsize_length_nuc)) </code></pre>
<pre class="r"><code>ggplot(effectsize_length_cat, aes(x=PercLength, by=fraction, fill=fraction))+ geom_histogram(alpha=.4,bins=10) + facet_grid(~PAU_Cat) + labs(title=&quot;Location of differentially used PAS within a gene body &quot;)+scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;)) </code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-46-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-46-1">
Past versions of unnamed-chunk-46-1.png
</button>
</p>
<div id="fig-unnamed-chunk-46-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-46-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/6679c9507ae4f542b9d32e7f499d6a5e4f6da7f6/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-46-1.png" target="_blank">6679c95</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/4f2326e310ffaf47c6a9a6afc54e0985dacf93aa/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-46-1.png" target="_blank">4f2326e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ae5c5a1a01a0f17c7e3e933df8ca97fb766903b5/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-46-1.png" target="_blank">ae5c5a1</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-21
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/9d0950ce5cbb2af3808c2fb5ee49911543466ca1/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-46-1.png" target="_blank">9d0950c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>summary(effectsize_length_catall$PercLength)</code></pre>
<pre><code>     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
-16763.99      0.89      1.04     22.93      1.91  86510.07 </code></pre>
<pre class="r"><code>summary(effectsize$logef)</code></pre>
<pre><code>     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
-2.241524 -0.319633 -0.006858  0.000000  0.329087  2.456682 </code></pre>
<pre class="r"><code>ggplot(effectsize_length_cat, aes(x=PercLength, by=fraction, fill=fraction))+ geom_histogram(,bins=50)  + labs(title=&quot;Location of differentially used PAS \nwithin a gene body&quot;, fill=&quot;Fraction&quot;, y=&quot;Number of PAS&quot;, x=&quot;Percent of Gene Length&quot;)+scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;),labels = c(&quot;Nuclear&quot;, &quot;Total&quot;))+ theme(legend.position = c(0.1,.9), legend.direction = &quot;horizontal&quot;)+  theme(panel.background = element_blank())</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-49-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-49-1">
Past versions of unnamed-chunk-49-1.png
</button>
</p>
<div id="fig-unnamed-chunk-49-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-49-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>densitylocdifuse=ggplot(effectsize_length_cat, aes(x=PercLength, by=fraction, fill=fraction))+ geom_density(alpha=.75)  + labs(title=&quot;Location of differentially used PAS \nwithin a gene body&quot;, fill=&quot;Fraction&quot;, x=&quot;Percent of Gene Length&quot;)+scale_fill_manual(values=c(&quot;deepskyblue3&quot;,&quot;darkviolet&quot;),labels = c(&quot;Nuclear&quot;, &quot;Total&quot;))+ theme(legend.position = &quot;bottom&quot;, legend.direction = &quot;horizontal&quot;)+  theme(panel.background = element_blank(),text = element_text(size=16), plot.title = element_text(size = 20, face = &quot;bold&quot;))

densitylocdifuse</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-50-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-50-1">
Past versions of unnamed-chunk-50-1.png
</button>
</p>
<div id="fig-unnamed-chunk-50-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/15fbb3c19335788121e18ba26d659dc8f2d0b757/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-50-1.png" target="_blank">15fbb3c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-31
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/c5bf3fd840a86608c59e24fad98d4fdde02b7d16/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-50-1.png" target="_blank">c5bf3fd</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-50-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Diff iso gene proportion:</p>
<pre class="r"><code>genes_sig=sig %&gt;% separate(cluster,into=c(&quot;chr&quot;, &quot;gene&quot;), sep=&quot;:&quot;) %&gt;% group_by(gene) %&gt;% summarise(n=n()) %&gt;% nrow
genes_detlapau= effectSize_highdiffGenes %&gt;% nrow()
testedgenes=read.table(&quot;../data/DiffIso/APApeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.5percCov.bothfrac.fixed.forLC.fc&quot;,header = T, stringsAsFactors = F) %&gt;% rownames_to_column(&quot;ID&quot;) %&gt;% select(ID)%&gt;% separate(ID, into=c(&quot;chr&quot;, &quot;start&quot;, &quot;end&quot;, &quot;geneID&quot;),sep=&quot;:&quot;) %&gt;% separate(geneID, into=c(&quot;gene&quot;, &quot;loc&quot;),sep=&quot;_&quot;)  %&gt;% group_by(gene) %&gt;% summarise(n=n()) %&gt;% nrow()
notsig=testedgenes-genes_sig
sighothighpau=genes_sig-genes_detlapau

cat=c(&quot;NotSig&quot;, &quot;SigNotHighPAU&quot;, &quot;SigandHighPAU&quot;)
values=c(unlist(notsig),unlist(sighothighpau),unlist(genes_detlapau))

difiso_df=as.data.frame(cbind(cat, values)) 
difiso_df$values=as.numeric(as.character(difiso_df$values))
difiso_df=difiso_df%&gt;% mutate(proportion=values/testedgenes)


ggplot(difiso_df, aes(x=&quot;&quot;,y=proportion, fill=cat)) + geom_bar(stat=&quot;identity&quot;)+geom_text(aes(label=values))</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-51-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-51-1">
Past versions of unnamed-chunk-51-1.png
</button>
</p>
<div id="fig-unnamed-chunk-51-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/c5bf3fd840a86608c59e24fad98d4fdde02b7d16/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-51-1.png" target="_blank">c5bf3fd</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/3e79995b518f62ca4f11cc3252aa728b1d01cd63/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-51-1.png" target="_blank">3e79995</a>
</td>
<td>
brimittleman
</td>
<td>
2019-06-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>slices &lt;- c(notsig, sighothighpau,genes_detlapau)
lbls &lt;- c(&quot;No Sig PAS&quot;, &quot;At least 1 \nSig PAS&quot;, &quot;At least 1 Sig PAS\n High Delta PAU&quot;)
pct &lt;- round(slices/sum(slices)*100)
lbls &lt;- paste(lbls, pct, sep=&quot;\n   &quot;) # add percents to labels 
lbls &lt;- paste(lbls,&quot;%&quot;,sep=&quot;&quot;) # ad % to labels 
pie(slices, labels = lbls,col=c(&quot;Azure2&quot;, &quot;Aquamarine1&quot;,&quot;Darkslateblue&quot;))</code></pre>
<p><img src="figure/DiffIsoAnalysis.Rmd/unnamed-chunk-52-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-52-1">
Past versions of unnamed-chunk-52-1.png
</button>
</p>
<div id="fig-unnamed-chunk-52-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/c5bf3fd840a86608c59e24fad98d4fdde02b7d16/docs/figure/DiffIsoAnalysis.Rmd/unnamed-chunk-52-1.png" target="_blank">c5bf3fd</a>
</td>
<td>
brimittleman
</td>
<td>
2019-07-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] reshape2_1.4.3  forcats_0.3.0   stringr_1.3.1   dplyr_0.8.0.1  
 [5] purrr_0.3.2     readr_1.3.1     tidyr_0.8.3     tibble_2.1.1   
 [9] ggplot2_3.1.1   tidyverse_1.2.1 workflowr_1.4.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.0       cellranger_1.1.0 plyr_1.8.4       compiler_3.5.1  
 [5] pillar_1.3.1     git2r_0.25.2     highr_0.7        tools_3.5.1     
 [9] digest_0.6.18    lubridate_1.7.4  jsonlite_1.6     evaluate_0.12   
[13] nlme_3.1-137     gtable_0.2.0     lattice_0.20-38  pkgconfig_2.0.2 
[17] rlang_0.4.0      cli_1.1.0        rstudioapi_0.10  yaml_2.2.0      
[21] haven_1.1.2      withr_2.1.2      xml2_1.2.0       httr_1.3.1      
[25] knitr_1.20       hms_0.4.2        generics_0.0.2   fs_1.3.1        
[29] rprojroot_1.3-2  grid_3.5.1       tidyselect_0.2.5 glue_1.3.0      
[33] R6_2.3.0         fansi_0.4.0      readxl_1.1.0     rmarkdown_1.10  
[37] modelr_0.1.2     magrittr_1.5     whisker_0.3-2    backports_1.1.2 
[41] scales_1.0.0     htmltools_0.3.6  rvest_0.3.2      assertthat_0.2.0
[45] colorspace_1.3-2 labeling_0.3     utf8_1.1.4       stringi_1.2.4   
[49] lazyeval_0.2.1   munsell_0.5.0    broom_0.5.1      crayon_1.3.4    </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
