<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />


<title>Nuclear fraction and nascent transcription</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">apaQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https/github.com/brimittleman/apaQTL">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Nuclear fraction and nascent transcription</h1>
<h4 class="author"><em>Briana Mittleman</em></h4>
<h4 class="date"><em>5/15/2019</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-05-17
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>apaQTL/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.3.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190411code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190411)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190411code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190411)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanapaQTLtree78b53a1cd0e8e370a6f7087ded2c3a495d3a1f41targetblank78b53a1a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/apaQTL/tree/78b53a1cd0e8e370a6f7087ded2c3a495d3a1f41" target="_blank">78b53a1</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanapaQTLtree78b53a1cd0e8e370a6f7087ded2c3a495d3a1f41targetblank78b53a1a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  .Rprofile
    Untracked:  ._.DS_Store
    Untracked:  .gitignore
    Untracked:  _workflowr.yml
    Untracked:  analysis/._PASdescriptiveplots.Rmd
    Untracked:  analysis/._cuttoffPercUsage.Rmd
    Untracked:  analysis/cuttoffPercUsage.Rmd
    Untracked:  apaQTL.Rproj
    Untracked:  code/._FC_UTR.sh
    Untracked:  code/._FC_newPeaks_olddata.sh
    Untracked:  code/._LC_samplegroups.py
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilefiltPAS
    Untracked:  code/._aAPAqtl_nominal39ind.sh
    Untracked:  code/._apaQTLCorrectPvalMakeQQ.R
    Untracked:  code/._apaQTL_Nominal.sh
    Untracked:  code/._apaQTL_permuted.sh
    Untracked:  code/._bed2saf.py
    Untracked:  code/._bothFrac_FC.sh
    Untracked:  code/._callPeaksYL.py
    Untracked:  code/._chooseAnno2SAF.py
    Untracked:  code/._chooseSignalSite
    Untracked:  code/._chooseSignalSite.py
    Untracked:  code/._cluster.json
    Untracked:  code/._clusterPAS.json
    Untracked:  code/._clusterfiltPAS.json
    Untracked:  code/._config.yaml
    Untracked:  code/._config2.yaml
    Untracked:  code/._configOLD.yaml
    Untracked:  code/._convertNumeric.py
    Untracked:  code/._dag.pdf
    Untracked:  code/._extractGenotypes.py
    Untracked:  code/._fc2leafphen.py
    Untracked:  code/._filter5perc.R
    Untracked:  code/._filter5percPheno.py
    Untracked:  code/._filterpeaks.py
    Untracked:  code/._finalPASbed2SAF.py
    Untracked:  code/._fix4su304corr.py
    Untracked:  code/._fix4su604corr.py
    Untracked:  code/._fix4sukalisto.py
    Untracked:  code/._fixFChead.py
    Untracked:  code/._fixFChead_bothfrac.py
    Untracked:  code/._fixH3k12ac.py
    Untracked:  code/._fixRNAhead4corr.py
    Untracked:  code/._fixRNAkalisto.py
    Untracked:  code/._fixhead_netseqfc.py
    Untracked:  code/._make5percPeakbed.py
    Untracked:  code/._makeFileID.py
    Untracked:  code/._makePheno.py
    Untracked:  code/._makeSAFbothfrac5perc.py
    Untracked:  code/._makegencondeTSSfile.py
    Untracked:  code/._mergeAllBam.sh
    Untracked:  code/._mergeByFracBam.sh
    Untracked:  code/._mergePeaks.sh
    Untracked:  code/._namePeaks.py
    Untracked:  code/._netseqFC.sh
    Untracked:  code/._peak2PAS.py
    Untracked:  code/._peakFC.sh
    Untracked:  code/._pheno2countonly.R
    Untracked:  code/._quantassign2parsedpeak.py
    Untracked:  code/._removeloc_pheno.py
    Untracked:  code/._run_leafcutterDiffIso.sh
    Untracked:  code/._selectNominalPvalues.py
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._submit-snakemakePAS.sh
    Untracked:  code/._submit-snakemakefiltPAS.sh
    Untracked:  code/._subset_diffisopheno.py
    Untracked:  code/._utrdms2saf.py
    Untracked:  code/.snakemake/
    Untracked:  code/APAqtl_nominal.err
    Untracked:  code/APAqtl_nominal.out
    Untracked:  code/APAqtl_nominal_39.err
    Untracked:  code/APAqtl_nominal_39.out
    Untracked:  code/APAqtl_permuted.err
    Untracked:  code/APAqtl_permuted.out
    Untracked:  code/BothFracDTPlotGeneRegions.err
    Untracked:  code/BothFracDTPlotGeneRegions.out
    Untracked:  code/DistPAS2Sig.py
    Untracked:  code/FC_UTR.err
    Untracked:  code/FC_UTR.out
    Untracked:  code/FC_UTR.sh
    Untracked:  code/FC_newPAS_olddata.err
    Untracked:  code/FC_newPAS_olddata.out
    Untracked:  code/FC_newPeaks_olddata.sh
    Untracked:  code/LC_samplegroups.py
    Untracked:  code/README.md
    Untracked:  code/Rplots.pdf
    Untracked:  code/Upstream100Bases_general.py
    Untracked:  code/aAPAqtl_nominal39ind.sh
    Untracked:  code/apaQTLCorrectPvalMakeQQ_4pc.R
    Untracked:  code/apaQTL_Nominal_4pc.sh
    Untracked:  code/apaQTL_permuted.4pc.sh
    Untracked:  code/bam2bw.err
    Untracked:  code/bam2bw.out
    Untracked:  code/bothFrac_FC.err
    Untracked:  code/bothFrac_FC.out
    Untracked:  code/bothFrac_FC.sh
    Untracked:  code/dag.pdf
    Untracked:  code/dagPAS.pdf
    Untracked:  code/dagfiltPAS.pdf
    Untracked:  code/extractGenotypes.py
    Untracked:  code/fc2leafphen.py
    Untracked:  code/finalPASbed2SAF.py
    Untracked:  code/findbuginpeaks.R
    Untracked:  code/fix4su304corr.py
    Untracked:  code/fix4su604corr.py
    Untracked:  code/fix4sukalisto.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixFChead_summary.py
    Untracked:  code/fixH3k12ac.py
    Untracked:  code/fixRNAhead4corr.py
    Untracked:  code/fixRNAkalisto.py
    Untracked:  code/fixhead_netseqfc.py
    Untracked:  code/get100upPAS.py
    Untracked:  code/getSeq100up.sh
    Untracked:  code/getseq100up.err
    Untracked:  code/getseq100up.out
    Untracked:  code/log/
    Untracked:  code/makeSAFbothfrac5perc.py
    Untracked:  code/makegencondeTSSfile.py
    Untracked:  code/netseqFC.err
    Untracked:  code/netseqFC.out
    Untracked:  code/netseqFC.sh
    Untracked:  code/removeloc_pheno.py
    Untracked:  code/run_DistPAS2Sig.err
    Untracked:  code/run_DistPAS2Sig.out
    Untracked:  code/run_distPAS2Sig.sh
    Untracked:  code/run_leafcutterDiffIso.sh
    Untracked:  code/run_leafcutter_ds.err
    Untracked:  code/run_leafcutter_ds.out
    Untracked:  code/selectNominalPvalues.py
    Untracked:  code/snakePASlog.out
    Untracked:  code/snakefiltPASlog.out
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/utrdms2saf.py
    Untracked:  data/CompareOldandNew/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffIso/
    Untracked:  data/PAS/
    Untracked:  data/QTLGenotypes/
    Untracked:  data/README.md
    Untracked:  data/RNAseq/
    Untracked:  data/Reads2UTR/
    Untracked:  data/SignalSiteFiles/
    Untracked:  data/ThirtyNineIndQtl_nominal/
    Untracked:  data/apaQTLNominal/
    Untracked:  data/apaQTLNominal_4pc/
    Untracked:  data/apaQTLPermuted/
    Untracked:  data/apaQTLPermuted_4pc/
    Untracked:  data/apaQTLs/
    Untracked:  data/assignedPeaks/
    Untracked:  data/bam/
    Untracked:  data/bam_clean/
    Untracked:  data/bam_waspfilt/
    Untracked:  data/bed_10up/
    Untracked:  data/bed_clean/
    Untracked:  data/bed_clean_sort/
    Untracked:  data/bed_waspfilter/
    Untracked:  data/bedsort_waspfilter/
    Untracked:  data/bothFrac_FC/
    Untracked:  data/exampleQTLs/
    Untracked:  data/fastq/
    Untracked:  data/filterPeaks/
    Untracked:  data/fourSU/
    Untracked:  data/h3k27ac/
    Untracked:  data/inclusivePeaks/
    Untracked:  data/inclusivePeaks_FC/
    Untracked:  data/mergedBG/
    Untracked:  data/mergedBW_byfrac/
    Untracked:  data/mergedBam/
    Untracked:  data/mergedbyFracBam/
    Untracked:  data/netseq/
    Untracked:  data/nuc_10up/
    Untracked:  data/nuc_10upclean/
    Untracked:  data/peakCoverage/
    Untracked:  data/peaks_5perc/
    Untracked:  data/phenotype/
    Untracked:  data/phenotype_5perc/
    Untracked:  data/sort/
    Untracked:  data/sort_clean/
    Untracked:  data/sort_waspfilter/
    Untracked:  nohup.out
    Untracked:  output/._.DS_Store
    Untracked:  output/._meanCorrelationPhenotypes.svg
    Untracked:  output/dtPlots/
    Untracked:  output/fastqc/
    Untracked:  output/meanCorrelationPhenotypes.svg

Unstaged changes:
    Modified:   analysis/PASusageQC.Rmd
    Modified:   analysis/choosePCs.Rmd
    Modified:   analysis/corrbetweenind.Rmd
    Modified:   analysis/nucintronicanalysis.Rmd
    Deleted:    code/Upstream10Bases_general.py
    Modified:   code/apaQTLCorrectPvalMakeQQ.R
    Modified:   code/apaQTL_permuted.sh
    Modified:   code/bed2saf.py
    Modified:   code/config.yaml
    Deleted:    code/test.txt

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/78b53a1cd0e8e370a6f7087ded2c3a495d3a1f41/analysis/nascenttranscription.Rmd" target="_blank">78b53a1</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-17
</td>
<td>
add full apa by loc
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/a295d27d552fc98286e043edd5c5f0fbb21f17d2/docs/nascenttranscription.html" target="_blank">a295d27</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/75f456708c9fb69499addbde5ed57b5f6370be83/analysis/nascenttranscription.Rmd" target="_blank">75f4567</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
<td>
add total intron/all
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/460e1fb1112f135cc0704c5a91d932f16a930450/docs/nascenttranscription.html" target="_blank">460e1fb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/1df3fe1053fa61acd4d2ca8f6a36287fd5ebf8f6/analysis/nascenttranscription.Rmd" target="_blank">1df3fe1</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
<td>
seperate fractions by locations
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/81a3e1646c908c34822af2ce80eac2e07ee602d6/docs/nascenttranscription.html" target="_blank">81a3e16</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/f484dcdb4493c72bb05e76cd64d14389de036457/analysis/nascenttranscription.Rmd" target="_blank">f484dcd</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-15
</td>
<td>
add nascent transcription plot
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(reshape2)
library(workflowr)</code></pre>
<pre><code>This is workflowr version 1.3.0
Run ?workflowr for help getting started</code></pre>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>── Attaching packages ──────────────────────────── tidyverse 1.2.1 ──</code></pre>
<pre><code>✔ ggplot2 3.1.1       ✔ purrr   0.3.2  
✔ tibble  2.1.1       ✔ dplyr   0.8.0.1
✔ tidyr   0.8.3       ✔ stringr 1.3.1  
✔ readr   1.3.1       ✔ forcats 0.3.0  </code></pre>
<pre><code>── Conflicts ─────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(viridis)</code></pre>
<pre><code>Loading required package: viridisLite</code></pre>
<p>Gene name switch file:</p>
<pre class="r"><code>geneNames=read.table(&quot;../../genome_anotation_data/ensemble_to_genename.txt&quot;, sep=&quot;\t&quot;, col.names = c(&#39;gene_id&#39;, &#39;GeneName&#39;, &#39;source&#39; ),stringsAsFactors = F)</code></pre>
<div id="create-transcription-phenotype" class="section level2">
<h2>Create transcription phenotype</h2>
<p>4su data</p>
<pre class="r"><code>FourSU=read.table(file = &quot;../data/fourSU//tr_decay_table_norm.txt&quot;, header=T, stringsAsFactors = F) %&gt;%  dplyr::select(gene_id,contains(&quot;4su_30&quot;))


FourSU_geneNames=FourSU %&gt;% inner_join(geneNames, by=&quot;gene_id&quot;) %&gt;% dplyr::select(GeneName, contains(&quot;4su_30&quot;))

FourgeneNames_long=melt(FourSU_geneNames,id.vars = &quot;GeneName&quot;, value.name = &quot;FourSU&quot;, variable.name = &quot;FourSU_ind&quot;) %&gt;% separate(FourSU_ind, into=c(&quot;type&quot;,&quot;time&quot;, &quot;1400&quot;, &quot;MAf&quot;, &quot;Individual&quot;), sep=&quot;_&quot;) %&gt;% dplyr::select(GeneName, Individual, FourSU) 

FourSU_geneMean=FourgeneNames_long %&gt;% group_by(GeneName) %&gt;%summarise(Mean_4su=mean(FourSU))</code></pre>
<p>rna seq</p>
<pre class="r"><code>RNA=read.table(file = &quot;../data/fourSU/tr_decay_table_norm.txt&quot;, header=T, stringsAsFactors = F) %&gt;%  dplyr::select(gene_id,contains(&quot;RNAseq_14000&quot;))


RNA_geneNames=RNA %&gt;% inner_join(geneNames, by=&quot;gene_id&quot;) %&gt;% dplyr::select(GeneName, contains(&quot;RNA&quot;))

RNAgeneNames_long=melt(RNA_geneNames,id.vars = &quot;GeneName&quot;, value.name = &quot;RNA&quot;, variable.name = &quot;RNA_ind&quot;) %&gt;%   separate(RNA_ind, into=c(&quot;type&quot;, &quot;1400&quot;, &quot;MAf&quot;, &quot;Individual&quot;), sep=&quot;_&quot;) %&gt;% dplyr::select(GeneName, Individual, RNA) 

RNA_geneMean=RNAgeneNames_long %&gt;% group_by(GeneName) %&gt;%summarise(Mean_RNA=mean(RNA))</code></pre>
<p>Make transcription phenotype</p>
<pre class="r"><code>Transcription=FourSU_geneMean %&gt;% inner_join(RNA_geneMean, by=&quot;GeneName&quot;) %&gt;% mutate(Transcription=Mean_4su/(Mean_4su + Mean_RNA)) %&gt;% dplyr::select(GeneName, Transcription) %&gt;% dplyr::rename(&quot;gene&quot;=GeneName)

Transcription2=FourSU_geneMean %&gt;% inner_join(RNA_geneMean, by=&quot;GeneName&quot;) %&gt;% mutate(Transcription=Mean_4su/Mean_RNA) %&gt;% dplyr::select(GeneName, Transcription) %&gt;% dplyr::rename(&quot;gene&quot;=GeneName)</code></pre>
</div>
<div id="apa-phenotype" class="section level2">
<h2>APA phenotype</h2>
<p>5 perc apa</p>
<pre class="r"><code>peaknumlist=read.table(&quot;../data/peaks_5perc/APApeak_Peaks_GeneLocAnno.5perc.bed&quot;, stringsAsFactors = F, header=F, col.names = c(&quot;chr&quot;, &quot;start&quot;,&quot;end&quot;, &quot;id&quot;, &quot;score&quot;, &quot;strand&quot;))  %&gt;% separate(id, into=c(&quot;peaknum&quot;, &quot;geneid&quot;), sep=&quot;:&quot;) %&gt;% mutate(peakid=paste(&quot;peak&quot;, peaknum,sep=&quot;&quot;))</code></pre>
<p>Nuclear apa</p>
<pre class="r"><code>NucAPA=read.table(&quot;../data/peakCoverage/APAPeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.Nuclear.Quant.Fixed.fc&quot;, stringsAsFactors = F, header = T) %&gt;% dplyr::select(-Chr, -Start, -End, -Strand, -Length) %&gt;% separate(Geneid, into=c(&quot;peakid&quot;,&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;strand&quot;, &quot;geneID&quot;),sep=&quot;:&quot;) %&gt;% semi_join(peaknumlist, by=&quot;peakid&quot;) %&gt;% separate(geneID, into=c(&quot;gene&quot;, &quot;loc&quot;), sep=&quot;_&quot;) %&gt;% dplyr::select(-chrom , -start, -end, -strand, -loc)

NucApaMelt=melt(NucAPA, id.vars =c( &quot;peakid&quot;, &quot;gene&quot;), value.name=&quot;count&quot;, variable.name=&quot;Ind&quot;) %&gt;% separate(Ind, into=c(&#39;Individual&#39;, &#39;fraction&#39;) ,sep=&quot;_&quot;)%&gt;% dplyr::select(peakid, gene, Individual, count)


NucAPA_bygene= NucApaMelt %&gt;% group_by(gene,Individual) %&gt;% summarise(NuclearSum=sum(count))</code></pre>
<p>total apa</p>
<pre class="r"><code>TotAPA=read.table(&quot;../data/peakCoverage/APAPeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.Total.Quant.Fixed.fc&quot;, stringsAsFactors = F, header = T) %&gt;% dplyr::select(-Chr, -Start, -End, -Strand, -Length) %&gt;% separate(Geneid, into=c(&quot;peakid&quot;,&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;strand&quot;, &quot;geneID&quot;),sep=&quot;:&quot;) %&gt;% semi_join(peaknumlist, by=&quot;peakid&quot;) %&gt;% separate(geneID, into=c(&quot;gene&quot;, &quot;loc&quot;), sep=&quot;_&quot;) %&gt;% dplyr::select(-chrom , -start, -end, -strand, -loc)

TotApaMelt=melt(TotAPA, id.vars =c( &quot;peakid&quot;, &quot;gene&quot;), value.name=&quot;count&quot;, variable.name=&quot;Ind&quot;) %&gt;% separate(Ind, into=c(&#39;Individual&#39;, &#39;fraction&#39;) ,sep=&quot;_&quot;)%&gt;% dplyr::select(peakid, gene, Individual, count)


TotAPA_bygene= TotApaMelt %&gt;% group_by(gene,Individual) %&gt;% summarise(TotalSum=sum(count))</code></pre>
<p>Sum together:</p>
<pre class="r"><code>ApaBothFrac=TotAPA_bygene %&gt;% inner_join(NucAPA_bygene, by=c(&quot;gene&quot;, &quot;Individual&quot;))

ApaBothFrac_melt=melt(ApaBothFrac, id.vars=c(&quot;gene&quot;, &quot;Individual&quot;),value.name=&quot;APA_val&quot; ) %&gt;% mutate(fraction=ifelse(variable==&quot;TotalSum&quot;, &quot;total&quot;, &quot;nuclear&quot;), line=paste(&quot;NA&quot;, substring(Individual, 2), sep=&quot;&quot;)) %&gt;% dplyr::select(gene, fraction, line, APA_val)</code></pre>
<p>Normalize with meta data info:</p>
<pre class="r"><code>metadata=read.table(&quot;../data/MetaDataSequencing.txt&quot;, header = T,stringsAsFactors = F) %&gt;% dplyr::select(line, fraction, Mapped_noMP)

metadata$line= as.character(metadata$line)

ApaBothFracStand=ApaBothFrac_melt %&gt;% full_join(metadata, by=c(&quot;line&quot;, &quot;fraction&quot;)) %&gt;% mutate(StandApa=APA_val/Mapped_noMP)

ApaBothFracStand_geneMean=ApaBothFracStand %&gt;% group_by(fraction, gene) %&gt;% summarise(meanAPA=mean(StandApa, na.rm=T))

ApaBothFracStand_geneMean_spread= spread(ApaBothFracStand_geneMean,fraction,meanAPA ) %&gt;% mutate(APAVal=nuclear/(total+ nuclear)) </code></pre>
</div>
<div id="join-data-and-plot" class="section level2">
<h2>Join data and plot</h2>
<p>Density function:</p>
<pre class="r"><code>get_density &lt;- function(x, y, ...) {
  dens &lt;- MASS::kde2d(x, y, ...)
  ix &lt;- findInterval(x, dens$x)
  iy &lt;- findInterval(y, dens$y)
  ii &lt;- cbind(ix, iy)
  return(dens$z[ii])
}

set.seed(1)
dat &lt;- data.frame(
  x = c(
    rnorm(1e4, mean = 0, sd = 0.1),
    rnorm(1e3, mean = 0, sd = 0.1)
  ),
  y = c(
    rnorm(1e4, mean = 0, sd = 0.1),
    rnorm(1e3, mean = 0.1, sd = 0.2)
  )
)</code></pre>
<p>Joing apa and transcription</p>
<pre class="r"><code>APAandTranscrption= Transcription %&gt;% inner_join(ApaBothFracStand_geneMean_spread, by=&quot;gene&quot;)
APAandTranscrption$density &lt;- get_density(APAandTranscrption$APAVal, APAandTranscrption$Transcription, n = 100)



summary(lm(data=APAandTranscrption, APAVal~Transcription))</code></pre>
<pre><code>
Call:
lm(formula = APAVal ~ Transcription, data = APAandTranscrption)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.37416 -0.09802  0.01235  0.10753  0.38048 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)   0.393873   0.007265   54.22   &lt;2e-16 ***
Transcription 0.285418   0.013551   21.06   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.1338 on 7885 degrees of freedom
Multiple R-squared:  0.05326,   Adjusted R-squared:  0.05314 
F-statistic: 443.6 on 1 and 7885 DF,  p-value: &lt; 2.2e-16</code></pre>
<p>Plot:</p>
<pre class="r"><code>ggplot(APAandTranscrption, aes(x=Transcription, y=APAVal))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;4su/4su+RNA&quot;, y=&quot;Nuclear/Nuclear+Total&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-13-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-1">
Past versions of unnamed-chunk-13-1.png
</button>
</p>
<div id="fig-unnamed-chunk-13-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/81a3e1646c908c34822af2ce80eac2e07ee602d6/docs/figure/nascenttranscription.Rmd/unnamed-chunk-13-1.png" target="_blank">81a3e16</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="split-nuclear-by-intronic" class="section level2">
<h2>Split Nuclear by intronic</h2>
<div id="nuclear-intronic" class="section level3">
<h3>Nuclear intronic:</h3>
<p>I will have to change the gene names for the 3’ info:</p>
<pre class="r"><code>NucAPAIntron=read.table(&quot;../data/peakCoverage/APAPeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.Nuclear.Quant.Fixed.fc&quot;, stringsAsFactors = F, header = T) %&gt;% dplyr::select(-Chr, -Start, -End, -Strand, -Length) %&gt;% separate(Geneid, into=c(&quot;peakid&quot;,&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;strand&quot;, &quot;geneID&quot;),sep=&quot;:&quot;) %&gt;% semi_join(peaknumlist, by=&quot;peakid&quot;) %&gt;% separate(geneID, into=c(&quot;gene&quot;, &quot;loc&quot;), sep=&quot;_&quot;) %&gt;% filter(loc==&quot;intron&quot;)%&gt;% dplyr::select(-chrom , -start, -end, -strand, -loc)

NucApaIntronMelt=melt(NucAPAIntron, id.vars =c( &quot;peakid&quot;, &quot;gene&quot;), value.name=&quot;count&quot;, variable.name=&quot;Ind&quot;) %&gt;% separate(Ind, into=c(&#39;Individual&#39;, &#39;fraction&#39;) ,sep=&quot;_&quot;)%&gt;% dplyr::select(peakid, gene, Individual, count)


NucAPAIntron_bygene= NucApaIntronMelt %&gt;% group_by(gene,Individual) %&gt;% summarise(NuclearIntronSum=sum(count))</code></pre>
</div>
<div id="total-utr" class="section level3">
<h3>Total UTR</h3>
<pre class="r"><code>TotUTRAPA=read.table(&quot;../data/peakCoverage/APAPeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.Total.Quant.Fixed.fc&quot;, stringsAsFactors = F, header = T) %&gt;% dplyr::select(-Chr, -Start, -End, -Strand, -Length) %&gt;% separate(Geneid, into=c(&quot;peakid&quot;,&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;strand&quot;, &quot;geneID&quot;),sep=&quot;:&quot;) %&gt;% semi_join(peaknumlist, by=&quot;peakid&quot;) %&gt;% separate(geneID, into=c(&quot;gene&quot;, &quot;loc&quot;), sep=&quot;_&quot;) %&gt;%filter(loc==&quot;utr3&quot;) %&gt;%  dplyr::select(-chrom , -start, -end, -strand, -loc)

TotApaUTRMelt=melt(TotUTRAPA, id.vars =c( &quot;peakid&quot;, &quot;gene&quot;), value.name=&quot;count&quot;, variable.name=&quot;Ind&quot;) %&gt;% separate(Ind, into=c(&#39;Individual&#39;, &#39;fraction&#39;) ,sep=&quot;_&quot;)%&gt;% dplyr::select(peakid, gene, Individual, count)


TotAPAUTR_bygene= TotApaUTRMelt %&gt;% group_by(gene,Individual) %&gt;% summarise(TotalUTRSum=sum(count))</code></pre>
<pre class="r"><code>ApaBothFracLoc=TotAPAUTR_bygene %&gt;% inner_join(NucAPAIntron_bygene, by=c(&quot;gene&quot;, &quot;Individual&quot;))

ApaBothFracLoc_melt=melt(ApaBothFracLoc, id.vars=c(&quot;gene&quot;, &quot;Individual&quot;),value.name=&quot;APA_val&quot; ) %&gt;% mutate(fraction=ifelse(variable==&quot;TotalUTRSum&quot;, &quot;total&quot;, &quot;nuclear&quot;), line=paste(&quot;NA&quot;, substring(Individual, 2), sep=&quot;&quot;)) %&gt;% dplyr::select(gene, fraction, line, APA_val)


ApaBothFracLocStand=ApaBothFracLoc_melt %&gt;% full_join(metadata, by=c(&quot;line&quot;, &quot;fraction&quot;)) %&gt;% mutate(StandApa=APA_val/Mapped_noMP)

ApaBothFracLocStand_geneMean=ApaBothFracLocStand %&gt;% group_by(fraction, gene) %&gt;% summarise(meanAPA=mean(StandApa, na.rm=T))

ApaBothFracLocStand_geneMean_spread= spread(ApaBothFracLocStand_geneMean,fraction,meanAPA ) %&gt;% mutate(APAValLoc=nuclear/(total+nuclear)) 

ApaBothFracLocStand_geneMean_spread2= spread(ApaBothFracLocStand_geneMean,fraction,meanAPA ) %&gt;% mutate(APAValLoc=nuclear/total) </code></pre>
<p>Join this with the transcription info:</p>
<pre class="r"><code>APAlocationandTranscrption= Transcription %&gt;% inner_join(ApaBothFracLocStand_geneMean_spread, by=&quot;gene&quot;)
APAlocationandTranscrption$density &lt;- get_density(APAlocationandTranscrption$APAValLoc, APAlocationandTranscrption$Transcription, n = 100)</code></pre>
<pre class="r"><code>ggplot(APAlocationandTranscrption, aes(x=Transcription, y=APAValLoc))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;4su/4su+RNA&quot;, y=&quot;NuclearIntron/TotalUTR + IntronNuclear&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-18-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-18-1">
Past versions of unnamed-chunk-18-1.png
</button>
</p>
<div id="fig-unnamed-chunk-18-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/460e1fb1112f135cc0704c5a91d932f16a930450/docs/figure/nascenttranscription.Rmd/unnamed-chunk-18-1.png" target="_blank">460e1fb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>summary(lm(data=APAlocationandTranscrption, APAValLoc~Transcription))</code></pre>
<pre><code>
Call:
lm(formula = APAValLoc ~ Transcription, data = APAlocationandTranscrption)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.40940 -0.19655 -0.07216  0.15348  0.80483 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)    0.02328    0.01768   1.317    0.188    
Transcription  0.47752    0.03232  14.777   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.2455 on 5491 degrees of freedom
Multiple R-squared:  0.03825,   Adjusted R-squared:  0.03807 
F-statistic: 218.4 on 1 and 5491 DF,  p-value: &lt; 2.2e-16</code></pre>
<p>Just the ratio:</p>
<pre class="r"><code>APAlocationandTranscrption2= Transcription2 %&gt;% inner_join(ApaBothFracLocStand_geneMean_spread2, by=&quot;gene&quot;)
APAlocationandTranscrption2$density &lt;- get_density(APAlocationandTranscrption2$APAValLoc, APAlocationandTranscrption2$Transcription, n = 100)
summary(lm(data=APAlocationandTranscrption2, log10(APAValLoc)~log10(Transcription)))</code></pre>
<pre><code>
Call:
lm(formula = log10(APAValLoc) ~ log10(Transcription), data = APAlocationandTranscrption2)

Residuals:
    Min      1Q  Median      3Q     Max 
-3.2920 -0.4924  0.0373  0.5100  3.3917 

Coefficients:
                     Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)          -0.69768    0.01097  -63.59   &lt;2e-16 ***
log10(Transcription)  0.90749    0.05474   16.58   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.7643 on 5491 degrees of freedom
Multiple R-squared:  0.04766,   Adjusted R-squared:  0.04749 
F-statistic: 274.8 on 1 and 5491 DF,  p-value: &lt; 2.2e-16</code></pre>
<pre class="r"><code>ggplot(APAlocationandTranscrption2, aes(x=log10(Transcription), y=log10(APAValLoc)))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;log10(4su/RNA)&quot;, y=&quot;log10(NuclearIntron/TotalUTR)&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-20-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-20-1">
Past versions of unnamed-chunk-20-1.png
</button>
</p>
<div id="fig-unnamed-chunk-20-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/460e1fb1112f135cc0704c5a91d932f16a930450/docs/figure/nascenttranscription.Rmd/unnamed-chunk-20-1.png" target="_blank">460e1fb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="compare-nuclear-and-total-utr" class="section level2">
<h2>Compare nuclear and total UTR</h2>
<pre class="r"><code>NucAPAUTR=read.table(&quot;../data/peakCoverage/APAPeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.Nuclear.Quant.Fixed.fc&quot;, stringsAsFactors = F, header = T) %&gt;% dplyr::select(-Chr, -Start, -End, -Strand, -Length) %&gt;% separate(Geneid, into=c(&quot;peakid&quot;,&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;strand&quot;, &quot;geneID&quot;),sep=&quot;:&quot;) %&gt;% semi_join(peaknumlist, by=&quot;peakid&quot;) %&gt;% separate(geneID, into=c(&quot;gene&quot;, &quot;loc&quot;), sep=&quot;_&quot;) %&gt;% filter(loc==&quot;utr3&quot;)%&gt;% dplyr::select(-chrom , -start, -end, -strand, -loc)

NucAPAUTRMelt=melt(NucAPAUTR, id.vars =c( &quot;peakid&quot;, &quot;gene&quot;), value.name=&quot;count&quot;, variable.name=&quot;Ind&quot;) %&gt;% separate(Ind, into=c(&#39;Individual&#39;, &#39;fraction&#39;) ,sep=&quot;_&quot;)%&gt;% dplyr::select(peakid, gene, Individual, count)


NucAPAUTR_bygene= NucAPAUTRMelt %&gt;% group_by(gene,Individual) %&gt;% summarise(NuclearUTRSum=sum(count))</code></pre>
<pre class="r"><code>ApaBothFracUTR=TotAPAUTR_bygene %&gt;% inner_join(NucAPAUTR_bygene, by=c(&quot;gene&quot;, &quot;Individual&quot;))

ApaBothFracUTR_melt=melt(ApaBothFracUTR, id.vars=c(&quot;gene&quot;, &quot;Individual&quot;),value.name=&quot;APA_val&quot; ) %&gt;% mutate(fraction=ifelse(variable==&quot;TotalUTRSum&quot;, &quot;total&quot;, &quot;nuclear&quot;), line=paste(&quot;NA&quot;, substring(Individual, 2), sep=&quot;&quot;)) %&gt;% dplyr::select(gene, fraction, line, APA_val)


ApaBothFracUTRStand=ApaBothFracUTR_melt %&gt;% full_join(metadata, by=c(&quot;line&quot;, &quot;fraction&quot;)) %&gt;% mutate(StandApa=APA_val/Mapped_noMP)

ApaBothFracUTRStand_geneMean=ApaBothFracUTRStand %&gt;% group_by(fraction, gene) %&gt;% summarise(meanAPA=mean(StandApa, na.rm=T))

ApaBothFracUTRStand_geneMean_spread= spread(ApaBothFracUTRStand_geneMean,fraction,meanAPA ) %&gt;% mutate(APAValLoc=nuclear/total)
ApaBothFracUTRStand_geneMean_spread2= spread(ApaBothFracUTRStand_geneMean,fraction,meanAPA ) %&gt;% mutate(APAValLoc=nuclear/(total+nuclear))</code></pre>
<p>THis is nuclear vs total only looking at teh UTR:</p>
<pre class="r"><code>APAUTRandTranscrption= Transcription %&gt;% inner_join(ApaBothFracUTRStand_geneMean_spread, by=&quot;gene&quot;)
APAUTRandTranscrption$density &lt;- get_density(APAUTRandTranscrption$APAValLoc, APAUTRandTranscrption$Transcription, n = 100)


summary(lm(data=APAUTRandTranscrption, log10(APAValLoc)~log10(Transcription)))</code></pre>
<pre><code>
Call:
lm(formula = log10(APAValLoc) ~ log10(Transcription), data = APAUTRandTranscrption)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.69221 -0.18902 -0.00244  0.19660  0.90638 

Coefficients:
                     Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)          0.155656   0.008892   17.50   &lt;2e-16 ***
log10(Transcription) 0.485080   0.028924   16.77   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.252 on 7753 degrees of freedom
Multiple R-squared:  0.03501,   Adjusted R-squared:  0.03488 
F-statistic: 281.3 on 1 and 7753 DF,  p-value: &lt; 2.2e-16</code></pre>
<pre class="r"><code>ggplot(APAUTRandTranscrption, aes(x=Transcription, y=APAValLoc))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;4su/RNA&quot;, y=&quot;NuclearUTR/TotalUTR&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-23-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-23-1">
Past versions of unnamed-chunk-23-1.png
</button>
</p>
<div id="fig-unnamed-chunk-23-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/460e1fb1112f135cc0704c5a91d932f16a930450/docs/figure/nascenttranscription.Rmd/unnamed-chunk-23-1.png" target="_blank">460e1fb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>APAUTRandTranscrption2= Transcription %&gt;% inner_join(ApaBothFracUTRStand_geneMean_spread2, by=&quot;gene&quot;)
APAUTRandTranscrption2$density &lt;- get_density(APAUTRandTranscrption2$APAValLoc, APAUTRandTranscrption2$Transcription, n = 100)


summary(lm(data=APAUTRandTranscrption2, log10(APAValLoc)~log10(Transcription)))</code></pre>
<pre><code>
Call:
lm(formula = log10(APAValLoc) ~ log10(Transcription), data = APAUTRandTranscrption2)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.45055 -0.08567  0.01585  0.10118  0.37283 

Coefficients:
                     Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)          -0.23370    0.00447  -52.29   &lt;2e-16 ***
log10(Transcription)  0.26974    0.01454   18.55   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.1267 on 7753 degrees of freedom
Multiple R-squared:  0.04251,   Adjusted R-squared:  0.04239 
F-statistic: 344.2 on 1 and 7753 DF,  p-value: &lt; 2.2e-16</code></pre>
<pre class="r"><code>ggplot(APAUTRandTranscrption2, aes(x=Transcription, y=APAValLoc))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;4su/RNA+4su&quot;, y=&quot;NuclearUTR/TotalUTR+NuclearUTR&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-24-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-24-1">
Past versions of unnamed-chunk-24-1.png
</button>
</p>
<div id="fig-unnamed-chunk-24-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/460e1fb1112f135cc0704c5a91d932f16a930450/docs/figure/nascenttranscription.Rmd/unnamed-chunk-24-1.png" target="_blank">460e1fb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="intron-nuclear-over-all-nuclear" class="section level2">
<h2>Intron Nuclear over all nuclear</h2>
<p>Nuclear intron= NucAPAIntron_bygene</p>
<p>all nuclear =NucAPA_bygene</p>
<p>Create this pheno:</p>
<pre class="r"><code>ApaNuclear_byloc=NucAPAIntron_bygene %&gt;% inner_join(NucAPA_bygene, by=c(&quot;gene&quot;, &quot;Individual&quot;)) %&gt;% mutate(IntronOverAll=NuclearIntronSum/NuclearSum) %&gt;% mutate(fraction=&quot;nuclear&quot;,line=paste(&quot;NA&quot;, substring(Individual, 2), sep=&quot;&quot;)) %&gt;% dplyr::select(gene, fraction, line, IntronOverAll) %&gt;% group_by(gene) %&gt;% filter(IntronOverAll!=0) %&gt;%  summarise(MeanIntronoverAll=mean(IntronOverAll)) %&gt;% dplyr::rename(&quot;GeneName&quot;=gene)</code></pre>
<p>Join with RNA</p>
<pre class="r"><code>nuclearandRNA=ApaNuclear_byloc %&gt;% inner_join(RNA_geneMean, by=&quot;GeneName&quot;)

nuclearandRNA$density &lt;- get_density(nuclearandRNA$MeanIntronoverAll, nuclearandRNA$Mean_RNA, n = 100)</code></pre>
<p>Plot:</p>
<pre class="r"><code>ggplot(nuclearandRNA, aes(x=log10(Mean_RNA), y=MeanIntronoverAll))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;log10(RNA)&quot;, y=&quot;NuclearIntron/NuclearAll&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-27-1">
Past versions of unnamed-chunk-27-1.png
</button>
</p>
<div id="fig-unnamed-chunk-27-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/460e1fb1112f135cc0704c5a91d932f16a930450/docs/figure/nascenttranscription.Rmd/unnamed-chunk-27-1.png" target="_blank">460e1fb</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>summary(lm(data=nuclearandRNA, MeanIntronoverAll~log10(Mean_RNA)))</code></pre>
<pre><code>
Call:
lm(formula = MeanIntronoverAll ~ log10(Mean_RNA), data = nuclearandRNA)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.39640 -0.15292 -0.04646  0.11444  0.84926 

Coefficients:
                 Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)     -0.408362   0.024437  -16.71   &lt;2e-16 ***
log10(Mean_RNA) -0.159782   0.005852  -27.30   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.2018 on 5560 degrees of freedom
Multiple R-squared:  0.1182,    Adjusted R-squared:  0.1181 
F-statistic: 745.5 on 1 and 5560 DF,  p-value: &lt; 2.2e-16</code></pre>
<div id="same-plot-with-transcription-phenotype-on-bottom" class="section level3">
<h3>Same plot with transcription phenotype on bottom:</h3>
<pre class="r"><code>ApaNuclear_byloc_rename=ApaNuclear_byloc %&gt;% dplyr::rename(&quot;gene&quot;=GeneName)
nuclearandtranscription=ApaNuclear_byloc_rename %&gt;% inner_join(Transcription, by=&quot;gene&quot;)

nuclearandtranscription$density &lt;- get_density(nuclearandtranscription$MeanIntronoverAll, nuclearandtranscription$Transcription, n = 100)

ggplot(nuclearandtranscription, aes(x=Transcription, y=MeanIntronoverAll))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;4su/4su+RNA&quot;, y=&quot;NuclearIntron/NuclearAll&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-29-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-29-1">
Past versions of unnamed-chunk-29-1.png
</button>
</p>
<div id="fig-unnamed-chunk-29-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/a295d27d552fc98286e043edd5c5f0fbb21f17d2/docs/figure/nascenttranscription.Rmd/unnamed-chunk-29-1.png" target="_blank">a295d27</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>summary(lm(data=nuclearandtranscription, MeanIntronoverAll~Transcription))</code></pre>
<pre><code>
Call:
lm(formula = MeanIntronoverAll ~ Transcription, data = nuclearandtranscription)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.29935 -0.17119 -0.05628  0.12082  0.79573 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)    0.14461    0.01530   9.452  &lt; 2e-16 ***
Transcription  0.20500    0.02797   7.329 2.66e-13 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.2138 on 5560 degrees of freedom
Multiple R-squared:  0.009567,  Adjusted R-squared:  0.009389 
F-statistic: 53.71 on 1 and 5560 DF,  p-value: 2.658e-13</code></pre>
</div>
</div>
<div id="intron-total-over-all-total" class="section level2">
<h2>Intron Total over all Total</h2>
<p>First I need to get the total intronic:</p>
<pre class="r"><code>TotAPAIntron=read.table(&quot;../data/peakCoverage/APAPeaks.ALLChrom.Filtered.Named.GeneLocAnnoPARSED.Total.Quant.Fixed.fc&quot;, stringsAsFactors = F, header = T) %&gt;% dplyr::select(-Chr, -Start, -End, -Strand, -Length) %&gt;% separate(Geneid, into=c(&quot;peakid&quot;,&quot;chrom&quot;, &quot;start&quot;, &quot;end&quot;, &quot;strand&quot;, &quot;geneID&quot;),sep=&quot;:&quot;) %&gt;% semi_join(peaknumlist, by=&quot;peakid&quot;) %&gt;% separate(geneID, into=c(&quot;gene&quot;, &quot;loc&quot;), sep=&quot;_&quot;) %&gt;%filter(loc==&quot;intron&quot;) %&gt;%  dplyr::select(-chrom , -start, -end, -strand, -loc)

TotAPAIntronMelt=melt(TotAPAIntron, id.vars =c( &quot;peakid&quot;, &quot;gene&quot;), value.name=&quot;count&quot;, variable.name=&quot;Ind&quot;) %&gt;% separate(Ind, into=c(&#39;Individual&#39;, &#39;fraction&#39;) ,sep=&quot;_&quot;)%&gt;% dplyr::select(peakid, gene, Individual, count)


TotAPAIntron_bygene= TotAPAIntronMelt %&gt;% group_by(gene,Individual) %&gt;% summarise(TotalIntronSum=sum(count))</code></pre>
<pre class="r"><code>ApaTotal_byloc=TotAPAIntron_bygene %&gt;% inner_join(TotAPA_bygene, by=c(&quot;gene&quot;, &quot;Individual&quot;)) %&gt;% mutate(IntronOverAll=TotalIntronSum/TotalSum) %&gt;% mutate(fraction=&quot;total&quot;,line=paste(&quot;NA&quot;, substring(Individual, 2), sep=&quot;&quot;)) %&gt;% dplyr::select(gene, fraction, line, IntronOverAll) %&gt;% group_by(gene) %&gt;% filter(IntronOverAll!=0) %&gt;%  summarise(MeanIntronoverAll=mean(IntronOverAll)) %&gt;% dplyr::rename(&quot;GeneName&quot;=gene)</code></pre>
<p>Join with RNA</p>
<pre class="r"><code>totalandRNA=ApaTotal_byloc %&gt;% inner_join(RNA_geneMean, by=&quot;GeneName&quot;)

totalandRNA$density &lt;- get_density(totalandRNA$MeanIntronoverAll, totalandRNA$Mean_RNA, n = 100)</code></pre>
<p>Plot:</p>
<pre class="r"><code>ggplot(totalandRNA, aes(x=log10(Mean_RNA), y=MeanIntronoverAll))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;log10(RNA)&quot;, y=&quot;TotalIntron/TotalAll&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-33-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-33-1">
Past versions of unnamed-chunk-33-1.png
</button>
</p>
<div id="fig-unnamed-chunk-33-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/a295d27d552fc98286e043edd5c5f0fbb21f17d2/docs/figure/nascenttranscription.Rmd/unnamed-chunk-33-1.png" target="_blank">a295d27</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>summary(lm(data=totalandRNA, MeanIntronoverAll~log10(Mean_RNA)))</code></pre>
<pre><code>
Call:
lm(formula = MeanIntronoverAll ~ log10(Mean_RNA), data = totalandRNA)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.30265 -0.10190 -0.03972  0.05381  0.92485 

Coefficients:
                 Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)     -0.445003   0.019435  -22.90   &lt;2e-16 ***
log10(Mean_RNA) -0.145939   0.004654  -31.36   &lt;2e-16 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.1605 on 5560 degrees of freedom
Multiple R-squared:  0.1503,    Adjusted R-squared:  0.1501 
F-statistic: 983.2 on 1 and 5560 DF,  p-value: &lt; 2.2e-16</code></pre>
<div id="same-plot-with-transcription-phenotype-on-bottom-1" class="section level3">
<h3>Same plot with transcription phenotype on bottom:</h3>
<pre class="r"><code>ApaTotal_byloc_rename=ApaTotal_byloc %&gt;% dplyr::rename(&quot;gene&quot;=GeneName)
totalandtranscription=ApaTotal_byloc_rename %&gt;% inner_join(Transcription, by=&quot;gene&quot;)

totalandtranscription$density &lt;- get_density(totalandtranscription$MeanIntronoverAll, totalandtranscription$Transcription, n = 100)

ggplot(totalandtranscription, aes(x=Transcription, y=MeanIntronoverAll))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;4su/4su+RNA&quot;, y=&quot;TotalIntron/TotalAll&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-35-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-35-1">
Past versions of unnamed-chunk-35-1.png
</button>
</p>
<div id="fig-unnamed-chunk-35-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/a295d27d552fc98286e043edd5c5f0fbb21f17d2/docs/figure/nascenttranscription.Rmd/unnamed-chunk-35-1.png" target="_blank">a295d27</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-16
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>summary(lm(data=totalandtranscription, MeanIntronoverAll~Transcription))</code></pre>
<pre><code>
Call:
lm(formula = MeanIntronoverAll ~ Transcription, data = totalandtranscription)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.19578 -0.11811 -0.05860  0.05286  0.87881 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)    0.07454    0.01240   6.012 1.95e-09 ***
Transcription  0.16028    0.02267   7.070 1.74e-12 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.1733 on 5560 degrees of freedom
Multiple R-squared:  0.00891,   Adjusted R-squared:  0.008732 
F-statistic: 49.99 on 1 and 5560 DF,  p-value: 1.739e-12</code></pre>
</div>
</div>
<div id="put-it-all-together" class="section level2">
<h2>Put it all together:</h2>
<p>(Nuclear intronic/nuclear all)/(total intronic/total all) vs 4su/(4su+RNA)</p>
<p>Nucintron v nuc all:</p>
<p>ApaNuclear_byloc_rename</p>
<p>ApaTotal_byloc_rename</p>
<p>Transcription</p>
<pre class="r"><code>fullapa=ApaNuclear_byloc_rename %&gt;% dplyr::rename(&quot;NuclearIntronoverall&quot;=MeanIntronoverAll)%&gt;% inner_join(ApaTotal_byloc_rename, by=&quot;gene&quot;) %&gt;% mutate(fullAPA=NuclearIntronoverall/MeanIntronoverAll) %&gt;% dplyr::select(gene,fullAPA)


#join with transcription 


BothlocPhenoandtranscription=fullapa %&gt;% inner_join(Transcription, by=&quot;gene&quot;)

BothlocPhenoandtranscription$density &lt;- get_density(BothlocPhenoandtranscription$fullAPA, BothlocPhenoandtranscription$Transcription, n = 100)

ggplot(BothlocPhenoandtranscription, aes(x=Transcription, y=fullAPA))+ geom_point(aes(color=density)) + geom_smooth(method = &quot;lm&quot;) + labs(x=&quot;4su/4su+RNA&quot;, y=&quot;(NuclearIntron/Nuclearall)/(TotalIntron/TotalAll)&quot;, title=&quot;Relationship between APA fraction and transcription&quot;) + scale_color_viridis()</code></pre>
<p><img src="figure/nascenttranscription.Rmd/unnamed-chunk-36-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre class="r"><code>summary(lm(data=totalandtranscription, MeanIntronoverAll~Transcription))</code></pre>
<pre><code>
Call:
lm(formula = MeanIntronoverAll ~ Transcription, data = totalandtranscription)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.19578 -0.11811 -0.05860  0.05286  0.87881 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)    0.07454    0.01240   6.012 1.95e-09 ***
Transcription  0.16028    0.02267   7.070 1.74e-12 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.1733 on 5560 degrees of freedom
Multiple R-squared:  0.00891,   Adjusted R-squared:  0.008732 
F-statistic: 49.99 on 1 and 5560 DF,  p-value: 1.739e-12</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] viridis_0.5.1     viridisLite_0.3.0 forcats_0.3.0    
 [4] stringr_1.3.1     dplyr_0.8.0.1     purrr_0.3.2      
 [7] readr_1.3.1       tidyr_0.8.3       tibble_2.1.1     
[10] ggplot2_3.1.1     tidyverse_1.2.1   workflowr_1.3.0  
[13] reshape2_1.4.3   

loaded via a namespace (and not attached):
 [1] tidyselect_0.2.5 haven_1.1.2      lattice_0.20-38  colorspace_1.3-2
 [5] generics_0.0.2   htmltools_0.3.6  yaml_2.2.0       rlang_0.3.1     
 [9] pillar_1.3.1     glue_1.3.0       withr_2.1.2      modelr_0.1.2    
[13] readxl_1.1.0     plyr_1.8.4       munsell_0.5.0    gtable_0.2.0    
[17] cellranger_1.1.0 rvest_0.3.2      evaluate_0.12    labeling_0.3    
[21] knitr_1.20       broom_0.5.1      Rcpp_1.0.0       scales_1.0.0    
[25] backports_1.1.2  jsonlite_1.6     fs_1.2.6         gridExtra_2.3   
[29] hms_0.4.2        digest_0.6.18    stringi_1.2.4    grid_3.5.1      
[33] rprojroot_1.3-2  cli_1.0.1        tools_3.5.1      magrittr_1.5    
[37] lazyeval_0.2.1   crayon_1.3.4     whisker_0.3-2    pkgconfig_2.0.2 
[41] MASS_7.3-51.1    xml2_1.2.0       lubridate_1.7.4  assertthat_0.2.0
[45] rmarkdown_1.10   httr_1.3.1       rstudioapi_0.10  R6_2.3.0        
[49] nlme_3.1-137     git2r_0.23.0     compiler_3.5.1  </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
