<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />


<title>New QTL heatmap</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">apaQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https/github.com/brimittleman/apaQTL">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">New QTL heatmap</h1>
<h4 class="author"><em>Briana Mittleman</em></h4>
<h4 class="date"><em>4/29/2019</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-04-30
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>apaQTL/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.3.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190411code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190411)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190411code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190411)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanapaQTLtreecf0998569593f3f277ac95338b7037ba5e1736d0targetblankcf09985a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/apaQTL/tree/cf0998569593f3f277ac95338b7037ba5e1736d0" target="_blank">cf09985</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanapaQTLtreecf0998569593f3f277ac95338b7037ba5e1736d0targetblankcf09985a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  .Rprofile
    Untracked:  ._.DS_Store
    Untracked:  .gitignore
    Untracked:  _workflowr.yml
    Untracked:  analysis/._PASdescriptiveplots.Rmd
    Untracked:  analysis/._cuttoffPercUsage.Rmd
    Untracked:  analysis/cuttoffPercUsage.Rmd
    Untracked:  apaQTL.Rproj
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilefiltPAS
    Untracked:  code/._aAPAqtl_nominal39ind.sh
    Untracked:  code/._apaQTLCorrectPvalMakeQQ.R
    Untracked:  code/._apaQTL_Nominal.sh
    Untracked:  code/._apaQTL_permuted.sh
    Untracked:  code/._bed2saf.py
    Untracked:  code/._callPeaksYL.py
    Untracked:  code/._chooseAnno2SAF.py
    Untracked:  code/._chooseSignalSite
    Untracked:  code/._chooseSignalSite.py
    Untracked:  code/._cluster.json
    Untracked:  code/._clusterPAS.json
    Untracked:  code/._clusterfiltPAS.json
    Untracked:  code/._config.yaml
    Untracked:  code/._config2.yaml
    Untracked:  code/._configOLD.yaml
    Untracked:  code/._convertNumeric.py
    Untracked:  code/._dag.pdf
    Untracked:  code/._extractGenotypes.py
    Untracked:  code/._filter5perc.R
    Untracked:  code/._filter5percPheno.py
    Untracked:  code/._filterpeaks.py
    Untracked:  code/._fixFChead.py
    Untracked:  code/._make5percPeakbed.py
    Untracked:  code/._makeFileID.py
    Untracked:  code/._makePheno.py
    Untracked:  code/._mergeAllBam.sh
    Untracked:  code/._mergeByFracBam.sh
    Untracked:  code/._mergePeaks.sh
    Untracked:  code/._namePeaks.py
    Untracked:  code/._peak2PAS.py
    Untracked:  code/._peakFC.sh
    Untracked:  code/._pheno2countonly.R
    Untracked:  code/._quantassign2parsedpeak.py
    Untracked:  code/._selectNominalPvalues.py
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._submit-snakemakePAS.sh
    Untracked:  code/._submit-snakemakefiltPAS.sh
    Untracked:  code/.snakemake/
    Untracked:  code/APAqtl_nominal.err
    Untracked:  code/APAqtl_nominal.out
    Untracked:  code/APAqtl_nominal_39.err
    Untracked:  code/APAqtl_nominal_39.out
    Untracked:  code/APAqtl_permuted.err
    Untracked:  code/APAqtl_permuted.out
    Untracked:  code/BothFracDTPlotGeneRegions.err
    Untracked:  code/BothFracDTPlotGeneRegions.out
    Untracked:  code/DistPAS2Sig.py
    Untracked:  code/README.md
    Untracked:  code/Rplots.pdf
    Untracked:  code/Upstream100Bases_general.py
    Untracked:  code/aAPAqtl_nominal39ind.sh
    Untracked:  code/bam2bw.err
    Untracked:  code/bam2bw.out
    Untracked:  code/dag.pdf
    Untracked:  code/dagPAS.pdf
    Untracked:  code/dagfiltPAS.pdf
    Untracked:  code/extractGenotypes.py
    Untracked:  code/findbuginpeaks.R
    Untracked:  code/get100upPAS.py
    Untracked:  code/getSeq100up.sh
    Untracked:  code/getseq100up.err
    Untracked:  code/getseq100up.out
    Untracked:  code/log/
    Untracked:  code/run_DistPAS2Sig.err
    Untracked:  code/run_DistPAS2Sig.out
    Untracked:  code/run_distPAS2Sig.sh
    Untracked:  code/selectNominalPvalues.py
    Untracked:  code/snakePASlog.out
    Untracked:  code/snakefiltPASlog.out
    Untracked:  data/DTmatrix/
    Untracked:  data/PAS/
    Untracked:  data/QTLGenotypes/
    Untracked:  data/README.md
    Untracked:  data/SignalSiteFiles/
    Untracked:  data/ThirtyNineIndQtl_nominal/
    Untracked:  data/apaQTLNominal/
    Untracked:  data/apaQTLPermuted/
    Untracked:  data/apaQTLs/
    Untracked:  data/assignedPeaks/
    Untracked:  data/bam/
    Untracked:  data/bam_clean/
    Untracked:  data/bam_waspfilt/
    Untracked:  data/bed_10up/
    Untracked:  data/bed_clean/
    Untracked:  data/bed_clean_sort/
    Untracked:  data/bed_waspfilter/
    Untracked:  data/bedsort_waspfilter/
    Untracked:  data/fastq/
    Untracked:  data/filterPeaks/
    Untracked:  data/inclusivePeaks/
    Untracked:  data/inclusivePeaks_FC/
    Untracked:  data/mergedBG/
    Untracked:  data/mergedBW_byfrac/
    Untracked:  data/mergedBam/
    Untracked:  data/mergedbyFracBam/
    Untracked:  data/nuc_10up/
    Untracked:  data/nuc_10upclean/
    Untracked:  data/peakCoverage/
    Untracked:  data/peaks_5perc/
    Untracked:  data/phenotype/
    Untracked:  data/phenotype_5perc/
    Untracked:  data/sort/
    Untracked:  data/sort_clean/
    Untracked:  data/sort_waspfilter/
    Untracked:  nohup.out
    Untracked:  output/._.DS_Store
    Untracked:  output/dtPlots/
    Untracked:  output/fastqc/

Unstaged changes:
    Modified:   analysis/PASusageQC.Rmd
    Modified:   analysis/corrbetweenind.Rmd
    Deleted:    code/Upstream10Bases_general.py
    Modified:   code/apaQTLCorrectPvalMakeQQ.R
    Modified:   code/apaQTL_permuted.sh
    Modified:   code/bed2saf.py
    Deleted:    code/test.txt

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/cf0998569593f3f277ac95338b7037ba5e1736d0/analysis/newQTLheatmap.Rmd" target="_blank">cf09985</a>
</td>
<td>
brimittleman
</td>
<td>
2019-04-30
</td>
<td>
add beta corr plots
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/e3bdc3ada15495b74f56c2b12038d4364c179354/docs/newQTLheatmap.html" target="_blank">e3bdc3a</a>
</td>
<td>
brimittleman
</td>
<td>
2019-04-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/39a65720a87b13e232ed3ca1fb874dd9bd8e17e7/analysis/newQTLheatmap.Rmd" target="_blank">39a6572</a>
</td>
<td>
brimittleman
</td>
<td>
2019-04-29
</td>
<td>
add correlation genotype heatmap
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(gdata)</code></pre>
<pre><code>gdata: read.xls support for &#39;XLS&#39; (Excel 97-2004) files ENABLED.</code></pre>
<pre><code></code></pre>
<pre><code>gdata: Unable to load perl libaries needed by read.xls()
gdata: to support &#39;XLSX&#39; (Excel 2007+) files.</code></pre>
<pre><code></code></pre>
<pre><code>gdata: Run the function &#39;installXLSXsupport()&#39;
gdata: to automatically download and install the perl
gdata: libaries needed to support Excel XLS and XLSX formats.</code></pre>
<pre><code>
Attaching package: &#39;gdata&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    nobs</code></pre>
<pre><code>The following object is masked from &#39;package:utils&#39;:

    object.size</code></pre>
<pre><code>The following object is masked from &#39;package:base&#39;:

    startsWith</code></pre>
<pre class="r"><code>library(workflowr)</code></pre>
<pre><code>This is workflowr version 1.3.0
Run ?workflowr for help getting started</code></pre>
<pre class="r"><code>library(gplots)</code></pre>
<pre><code>
Attaching package: &#39;gplots&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:stats&#39;:

    lowess</code></pre>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>── Attaching packages ──────────────────────────────── tidyverse 1.2.1 ──</code></pre>
<pre><code>✔ ggplot2 3.1.0       ✔ purrr   0.3.2  
✔ tibble  2.1.1       ✔ dplyr   0.8.0.1
✔ tidyr   0.8.3       ✔ stringr 1.3.1  
✔ readr   1.3.1       ✔ forcats 0.3.0  </code></pre>
<pre><code>── Conflicts ─────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::combine() masks gdata::combine()
✖ dplyr::filter()  masks stats::filter()
✖ dplyr::first()   masks gdata::first()
✖ purrr::keep()    masks gdata::keep()
✖ dplyr::lag()     masks stats::lag()
✖ dplyr::last()    masks gdata::last()</code></pre>
<pre class="r"><code>library(cowplot)</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggplot2&#39;:

    ggsave</code></pre>
<div id="compare-qtls-to-those-found-with-previous-batch-data" class="section level2">
<h2>Compare QTLs to those found with previous batch data</h2>
<p>I have about double the QTLs hear compared to before resequencing batch 4. I will look at the new QTL to see if there is evidence for them being false positives. I am going to see if there is structure in the genotypes for these QTLs.</p>
<p>The old QTLs are from the threeprimeseq repository.</p>
<div id="total" class="section level3">
<h3>Total</h3>
<p>Import old QTLs</p>
<pre class="r"><code>oldtot=read.table(&quot;../../threeprimeseq/data/perm_APAqtl_GeneLocAnno_noMP_5percUs/filtered_APApeaks_merged_allchrom_refseqGenes.GeneLocAnno.NoMP_sm_quant.Total.fixed.pheno_5perc_permResBH.txt&quot;, header=T,stringsAsFactors = F) %&gt;% separate(pid, into=c(&quot;Chr&quot;, &quot;Start&quot;, &quot;End&quot;, &quot;PeakID&quot;), sep=&quot;:&quot;) %&gt;% separate(PeakID, into=c(&quot;Gene&quot;, &quot;Strand&quot;,&quot;Peak&quot;), sep=&quot;_&quot;)</code></pre>
<pre><code>Warning: Expected 3 pieces. Additional pieces discarded in 3 rows [886,
887, 888].</code></pre>
<pre class="r"><code>OldTotQTLs= oldtot %&gt;% filter(-log10(bh)&gt;=1)
nrow(OldTotQTLs)</code></pre>
<pre><code>[1] 291</code></pre>
<p>Import new QTLs:</p>
<pre class="r"><code>newTotQTLs=read.table(&quot;../data/apaQTLs/Total_apaQTLs_5fdr.txt&quot;, stringsAsFactors = F, header = T)
nrow(newTotQTLs)</code></pre>
<pre><code>[1] 502</code></pre>
<p>Filter out those matching from the old:</p>
<pre class="r"><code>UniqueNewTot=newTotQTLs %&gt;% semi_join(OldTotQTLs, by=&quot;sid&quot;)</code></pre>
<p>There are only 105 new snps This makes sense because 1 sno associates with multiple peaks.</p>
<p>Write these out to fetch the genotypes:</p>
<pre class="r"><code>write.table(UniqueNewTot, file=&quot;../data/apaQTLs/Total_apaQTLs_5fdr_NewUnique.txt&quot;, quote = F, col.names = F, row.names = F)</code></pre>
</div>
<div id="nuclear" class="section level3">
<h3>Nuclear</h3>
<pre class="r"><code>oldnuc=read.table(&quot;../../threeprimeseq/data/perm_APAqtl_GeneLocAnno_noMP_5percUs/filtered_APApeaks_merged_allchrom_refseqGenes.GeneLocAnno.NoMP_sm_quant.Nuclear.fixed.pheno_5perc_permResBH.txt&quot;, header=T,stringsAsFactors = F) %&gt;% separate(pid, into=c(&quot;Chr&quot;, &quot;Start&quot;, &quot;End&quot;, &quot;PeakID&quot;), sep=&quot;:&quot;) %&gt;% separate(PeakID, into=c(&quot;Gene&quot;, &quot;Strand&quot;,&quot;Peak&quot;), sep=&quot;_&quot;)</code></pre>
<pre><code>Warning: Expected 3 pieces. Additional pieces discarded in 3 rows [1056,
1057, 1058].</code></pre>
<pre class="r"><code>OldNucQTLs= oldnuc %&gt;% filter(-log10(bh)&gt;=1)
nrow(OldNucQTLs)</code></pre>
<pre><code>[1] 615</code></pre>
<p>Import new QTLs:</p>
<pre class="r"><code>newNucQTLs=read.table(&quot;../data/apaQTLs/Nuclear_apaQTLs_5fdr.txt&quot;, stringsAsFactors = F, header = T)
nrow(newNucQTLs)</code></pre>
<pre><code>[1] 1070</code></pre>
<p>Filter out those matching from the old:</p>
<pre class="r"><code>UniqueNewNuc=newNucQTLs %&gt;% semi_join(OldNucQTLs, by=&quot;sid&quot;)</code></pre>
<p>There are 200 new snps in this set.</p>
<pre class="r"><code>write.table(UniqueNewNuc, file=&quot;../data/apaQTLs/Nuclear_apaQTLs_5fdr_NewUnique.txt&quot;, quote = F, col.names = F, row.names = F)</code></pre>
</div>
</div>
<div id="extract-genotypes" class="section level2">
<h2>Extract genotypes:</h2>
<p>I wrote a script to pull the doses from the vcf file. Run it with:</p>
<pre class="bash"><code> python extractGenotypes.py ../data/apaQTLs/Nuclear_apaQTLs_5fdr_NewUnique.txt ../data/QTLGenotypes/Genotypes_NuclearapaQTLS_newunique.txt
 
  python extractGenotypes.py ../data/apaQTLs/Total_apaQTLs_5fdr_NewUnique.txt ../data/QTLGenotypes/Genotypes_TotalapaQTLS_newunique.txt</code></pre>
<p>I also need the header from the VCF to have the individuals:</p>
<pre class="bash"><code>head -n14 /project2/gilad/briana/YRI_geno_hg19/allChrom.dose.filt.vcf | tail -n1  &gt; ../data/QTLGenotypes/vcfheader.txt

#manually remove # and unneaded columns, keep snp and ind. </code></pre>
<pre class="r"><code>vcfhead=read.table(&quot;../data/QTLGenotypes/vcfheader.txt&quot;, header = T)</code></pre>
<p>input sample list:</p>
<pre class="r"><code>samples=read.table(&quot;../data/phenotype/SAMPLE.txt&quot;)
samplist=as.vector(samples$V1)</code></pre>
<div id="total-1" class="section level3">
<h3>Total:</h3>
<pre class="r"><code>totgeno=read.table(&quot;../data/QTLGenotypes/Genotypes_TotalapaQTLS_newunique.txt&quot;, col.names = colnames(vcfhead)) %&gt;% select(samplist) %&gt;% t()</code></pre>
<p>Correlation:</p>
<pre class="r"><code>totgeneCorr=round(cor(totgeno),2)

heatmap.2(as.matrix(totgeneCorr),trace=&quot;none&quot;, dendrogram =c(&quot;none&quot;), main=&quot;Genotype correlation\n for new Total QTL snps&quot;)</code></pre>
<p><img src="figure/newQTLheatmap.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-15-1">
Past versions of unnamed-chunk-15-1.png
</button>
</p>
<div id="fig-unnamed-chunk-15-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/e3bdc3ada15495b74f56c2b12038d4364c179354/docs/figure/newQTLheatmap.Rmd/unnamed-chunk-15-1.png" target="_blank">e3bdc3a</a>
</td>
<td>
brimittleman
</td>
<td>
2019-04-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="nuclear-1" class="section level3">
<h3>Nuclear</h3>
<pre class="r"><code>nucgeno=read.table(&quot;../data/QTLGenotypes/Genotypes_NuclearapaQTLS_newunique.txt&quot;, col.names = colnames(vcfhead)) %&gt;% select(samplist) %&gt;% t()</code></pre>
<p>Correlation:</p>
<pre class="r"><code>nucgeneCorr=round(cor(nucgeno),2)

heatmap.2(as.matrix(nucgeneCorr),trace=&quot;none&quot;, dendrogram =c(&quot;none&quot;),main=&quot;Genotype correlation \n for new Nuclear QTL snps&quot;)</code></pre>
<p><img src="figure/newQTLheatmap.Rmd/unnamed-chunk-17-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-17-1">
Past versions of unnamed-chunk-17-1.png
</button>
</p>
<div id="fig-unnamed-chunk-17-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/e3bdc3ada15495b74f56c2b12038d4364c179354/docs/figure/newQTLheatmap.Rmd/unnamed-chunk-17-1.png" target="_blank">e3bdc3a</a>
</td>
<td>
brimittleman
</td>
<td>
2019-04-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="compare-beta-values-in-55-vs-39" class="section level2">
<h2>Compare beta values in 55 vs 39</h2>
<p>I want to make a scatter plot comaparring the new QTL associations in the 55 vs 39 individauls. If the qtls are real we expect a high correlation.</p>
<p>To do this I can recall the qtls with a smaller sample list excluding the 15 new individuals.</p>
<p>I need to make a list of the individuals not in the 4th batch.</p>
<pre class="r"><code>batch1.2.3=read.table(&quot;../data/MetaDataSequencing.txt&quot;, header=T,stringsAsFactors = F)%&gt;% filter(fraction==&quot;total&quot;) %&gt;%  select(line, batch) %&gt;% filter(batch != 4)</code></pre>
<pre class="r"><code>samplelist=read.table(&quot;../data/phenotype/SAMPLE.txt&quot;, col.names = c(&quot;line&quot;),stringsAsFactors = F)</code></pre>
<p>Make a new directory for the 39ind qtls:</p>
<pre class="bash"><code>mkdir ../data/ThirtyNineIndQtl_nominal</code></pre>
<p>Filter the sample list</p>
<pre class="r"><code>samplelist_39= samplelist %&gt;% semi_join(batch1.2.3, by=&quot;line&quot;)</code></pre>
<pre class="r"><code>write.table(samplelist_39, file=&quot;../data/ThirtyNineIndQtl_nominal/samplelist39.txt&quot;, col.names = F, row.names = F, quote = F)</code></pre>
<p>Run the QTL code with this sample list</p>
<pre class="bash"><code>sbatch aAPAqtl_nominal39ind.sh
</code></pre>
<p>Concatinate results:</p>
<pre class="bash"><code>cat APApeak_Phenotype_GeneLocAnno.Total.5perc.fc.gz.qqnorm_chr* &gt; APApeak_Phenotype_GeneLocAnno.Total.5perc.fc.gz.qqnorm_AllChr.txt

cat APApeak_Phenotype_GeneLocAnno.Nuclear.5perc.fc.gz.qqnorm_chr* &gt; APApeak_Phenotype_GeneLocAnno.Nuclear.5perc.fc.gz.qqnorm_AllChr.txt
</code></pre>
<p>I want to filter the results for the new snps in the uniquenewtot. These results are in data/apaQTLs</p>
<p>I need to write a script that makes a dictionary with each of the new QTLs in the format above. Then I can run throguh the nominal values and keep only the values in the dictionary.</p>
<p>I can run this on the 55 and 39 nominal files then combine the files to create the scatterplot.</p>
<div id="total-2" class="section level3">
<h3>total</h3>
<pre class="bash"><code>python selectNominalPvalues.py ../data/apaQTLs/Total_apaQTLs_5fdr_NewUnique.txt ../data/ThirtyNineIndQtl_nominal/APApeak_Phenotype_GeneLocAnno.Total.5perc.fc.gz.qqnorm_AllChr.txt ../data/ThirtyNineIndQtl_nominal/Total_apaQTLs_NewUniqNom_37ing.txt

python selectNominalPvalues.py ../data/apaQTLs/Total_apaQTLs_5fdr_NewUnique.txt ../data/apaQTLNominal/APApeak_Phenotype_GeneLocAnno.Total.5perc.fc.gz.qqnorm_AllChr.txt ../data/ThirtyNineIndQtl_nominal/Total_apaQTLs_NewUniqNom_55ind.txt
</code></pre>
<p>Import files:</p>
<pre class="r"><code>newin37_tot=read.table(&quot;../data/ThirtyNineIndQtl_nominal/Total_apaQTLs_NewUniqNom_37ing.txt&quot;,col.names=c(&quot;peakID&quot;, &quot;snp&quot;, &quot;dist&quot;, &quot;Nompval39&quot;,&quot;Beta39&quot;), stringsAsFactors = F)%&gt;% select(peakID, snp, Beta39)

newin54_tot=read.table(&quot;../data/ThirtyNineIndQtl_nominal/Total_apaQTLs_NewUniqNom_55ind.txt&quot;,col.names=c(&quot;peakID&quot;, &quot;snp&quot;, &quot;dist&quot;, &quot;Nompval54&quot;,&quot;Beta54&quot;), stringsAsFactors = F) %&gt;% select(peakID, snp, Beta54)</code></pre>
<p>Join these:</p>
<pre class="r"><code>newinboth=newin54_tot %&gt;% full_join(newin37_tot, by=c(&quot;peakID&quot;, &quot;snp&quot;))</code></pre>
<pre class="r"><code>total_qtlind=ggplot(newinboth,aes(x=Beta54, y=Beta39)) + geom_point()  + labs(title=&quot;New Total apaQTLs \nin different ind. sets&quot;, ylab=&quot;Beta 39 ind&quot;, xlab=&quot;Beta 55 ind&quot;)</code></pre>
</div>
<div id="nuclear-2" class="section level3">
<h3>nuclear</h3>
<pre class="bash"><code>python selectNominalPvalues.py ../data/apaQTLs/Nuclear_apaQTLs_5fdr_NewUnique.txt ../data/ThirtyNineIndQtl_nominal/APApeak_Phenotype_GeneLocAnno.Nuclear.5perc.fc.gz.qqnorm_AllChr.txt ../data/ThirtyNineIndQtl_nominal/Nuclear_apaQTLs_NewUniqNom_37ing.txt

python selectNominalPvalues.py ../data/apaQTLs/Nuclear_apaQTLs_5fdr_NewUnique.txt ../data/apaQTLNominal/APApeak_Phenotype_GeneLocAnno.Nuclear.5perc.fc.gz.qqnorm_AllChr.txt ../data/ThirtyNineIndQtl_nominal/Nuclear_apaQTLs_NewUniqNom_55ind.txt</code></pre>
<pre class="r"><code>newin37_nuc=read.table(&quot;../data/ThirtyNineIndQtl_nominal/Nuclear_apaQTLs_NewUniqNom_37ing.txt&quot;,col.names=c(&quot;peakID&quot;, &quot;snp&quot;, &quot;dist&quot;, &quot;Nompval39&quot;,&quot;Beta39&quot;), stringsAsFactors = F)%&gt;% select(peakID, snp, Beta39)

newin54_nuc=read.table(&quot;../data/ThirtyNineIndQtl_nominal/Nuclear_apaQTLs_NewUniqNom_55ind.txt&quot;,col.names=c(&quot;peakID&quot;, &quot;snp&quot;, &quot;dist&quot;, &quot;Nompval54&quot;,&quot;Beta54&quot;), stringsAsFactors = F) %&gt;% select(peakID, snp, Beta54)</code></pre>
<p>Join these:</p>
<pre class="r"><code>newinboth_nuc=newin54_nuc %&gt;% full_join(newin37_nuc, by=c(&quot;peakID&quot;, &quot;snp&quot;))</code></pre>
<pre class="r"><code>nuclear_qtlind=ggplot(newinboth_nuc,aes(x=Beta54, y=Beta39)) + geom_point()  + labs(title=&quot;New Nuclear apaQTLs\n in different ind. sets&quot;, ylab=&quot;Beta 39 ind&quot;, xlab=&quot;Beta 55 ind&quot;)</code></pre>
<p>plot both:</p>
<pre class="r"><code>plot_grid(total_qtlind, nuclear_qtlind)</code></pre>
<p><img src="figure/newQTLheatmap.Rmd/unnamed-chunk-33-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] cowplot_0.9.4   forcats_0.3.0   stringr_1.3.1   dplyr_0.8.0.1  
 [5] purrr_0.3.2     readr_1.3.1     tidyr_0.8.3     tibble_2.1.1   
 [9] ggplot2_3.1.0   tidyverse_1.2.1 gplots_3.0.1    workflowr_1.3.0
[13] gdata_2.18.0   

loaded via a namespace (and not attached):
 [1] gtools_3.8.1       tidyselect_0.2.5   haven_1.1.2       
 [4] lattice_0.20-38    colorspace_1.3-2   generics_0.0.2    
 [7] htmltools_0.3.6    yaml_2.2.0         rlang_0.3.1       
[10] pillar_1.3.1       glue_1.3.0         withr_2.1.2       
[13] modelr_0.1.2       readxl_1.1.0       plyr_1.8.4        
[16] munsell_0.5.0      gtable_0.2.0       cellranger_1.1.0  
[19] rvest_0.3.2        caTools_1.17.1.1   evaluate_0.12     
[22] labeling_0.3       knitr_1.20         broom_0.5.1       
[25] Rcpp_1.0.0         KernSmooth_2.23-15 scales_1.0.0      
[28] backports_1.1.2    jsonlite_1.6       fs_1.2.6          
[31] hms_0.4.2          digest_0.6.18      stringi_1.2.4     
[34] grid_3.5.1         rprojroot_1.3-2    cli_1.0.1         
[37] tools_3.5.1        bitops_1.0-6       magrittr_1.5      
[40] lazyeval_0.2.1     crayon_1.3.4       whisker_0.3-2     
[43] pkgconfig_2.0.2    xml2_1.2.0         lubridate_1.7.4   
[46] assertthat_0.2.0   rmarkdown_1.10     httr_1.3.1        
[49] rstudioapi_0.10    R6_2.3.0           nlme_3.1-137      
[52] git2r_0.23.0       compiler_3.5.1    </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
