<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Briana Mittleman" />


<title>Location of Nuclear Intronic PAS</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/journal.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">apaQTL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https/github.com/brimittleman/apaQTL">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Location of Nuclear Intronic PAS</h1>
<h4 class="author"><em>Briana Mittleman</em></h4>
<h4 class="date"><em>5/15/2019</em></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-05-29
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>apaQTL/analysis/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.3.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20190411code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20190411)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20190411code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20190411)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanapaQTLtree974cad6dc70d4dd71a922a449d39d1f1d0fe44f4targetblank974cad6a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/brimittleman/apaQTL/tree/974cad6dc70d4dd71a922a449d39d1f1d0fe44f4" target="_blank">974cad6</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcombrimittlemanapaQTLtree974cad6dc70d4dd71a922a449d39d1f1d0fe44f4targetblank974cad6a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  .Rprofile
    Untracked:  ._.DS_Store
    Untracked:  .gitignore
    Untracked:  _workflowr.yml
    Untracked:  analysis/._PASdescriptiveplots.Rmd
    Untracked:  analysis/._cuttoffPercUsage.Rmd
    Untracked:  analysis/cuttoffPercUsage.Rmd
    Untracked:  analysis/nascentRNA.Rmd
    Untracked:  apaQTL.Rproj
    Untracked:  code/._ApaQTL_nominalNonnorm.sh
    Untracked:  code/._BothFracDTPlotGeneRegions_normalized.sh
    Untracked:  code/._FC_UTR.sh
    Untracked:  code/._FC_newPeaks_olddata.sh
    Untracked:  code/._LC_samplegroups.py
    Untracked:  code/._NascentRNAdtPlot.sh
    Untracked:  code/._NetSeq_fourthintronDT.sh
    Untracked:  code/._SnakefilePAS
    Untracked:  code/._SnakefilefiltPAS
    Untracked:  code/._TESplots100bp.sh
    Untracked:  code/._TESplots150bp.sh
    Untracked:  code/._TESplots200bp.sh
    Untracked:  code/._Untitled
    Untracked:  code/._ZipandTabPheno.sh
    Untracked:  code/._aAPAqtl_nominal39ind.sh
    Untracked:  code/._apaQTLCorrectPvalMakeQQ.R
    Untracked:  code/._apaQTL_Nominal.sh
    Untracked:  code/._apaQTL_permuted.sh
    Untracked:  code/._assignNucIntonpeak2intronlocs.sh
    Untracked:  code/._assignTotIntronpeak2intronlocs.sh
    Untracked:  code/._bam2BW_5primemost.sh
    Untracked:  code/._bed2saf.py
    Untracked:  code/._bothFracDTplot1stintron.sh
    Untracked:  code/._bothFracDTplot4thintron.sh
    Untracked:  code/._bothFrac_FC.sh
    Untracked:  code/._callPeaksYL.py
    Untracked:  code/._chooseAnno2SAF.py
    Untracked:  code/._chooseSignalSite
    Untracked:  code/._chooseSignalSite.py
    Untracked:  code/._cluster.json
    Untracked:  code/._clusterPAS.json
    Untracked:  code/._clusterfiltPAS.json
    Untracked:  code/._codingdms2bed.py
    Untracked:  code/._config.yaml
    Untracked:  code/._config2.yaml
    Untracked:  code/._configOLD.yaml
    Untracked:  code/._convertNumeric.py
    Untracked:  code/._dag.pdf
    Untracked:  code/._encodeRNADTplots.sh
    Untracked:  code/._extractGenotypes.py
    Untracked:  code/._fc2leafphen.py
    Untracked:  code/._filter5perc.R
    Untracked:  code/._filter5percPheno.py
    Untracked:  code/._filterpeaks.py
    Untracked:  code/._finalPASbed2SAF.py
    Untracked:  code/._fix4su304corr.py
    Untracked:  code/._fix4su604corr.py
    Untracked:  code/._fix4sukalisto.py
    Untracked:  code/._fixFChead.py
    Untracked:  code/._fixFChead_bothfrac.py
    Untracked:  code/._fixH3k12ac.py
    Untracked:  code/._fixRNAhead4corr.py
    Untracked:  code/._fixRNAkalisto.py
    Untracked:  code/._fixgroupedtranscript.py
    Untracked:  code/._fixhead_netseqfc.py
    Untracked:  code/._grouptranscripts.py
    Untracked:  code/._make5percPeakbed.py
    Untracked:  code/._makeFileID.py
    Untracked:  code/._makePheno.py
    Untracked:  code/._makeSAFbothfrac5perc.py
    Untracked:  code/._makegencondeTSSfile.py
    Untracked:  code/._mergeAllBam.sh
    Untracked:  code/._mergeBW_norm.sh
    Untracked:  code/._mergeBamNascent.sh
    Untracked:  code/._mergeByFracBam.sh
    Untracked:  code/._mergePeaks.sh
    Untracked:  code/._mnase1stintron.sh
    Untracked:  code/._mnaseDT_fourthintron.sh
    Untracked:  code/._namePeaks.py
    Untracked:  code/._netseqDTplot1stIntron.sh
    Untracked:  code/._netseqFC.sh
    Untracked:  code/._peak2PAS.py
    Untracked:  code/._peakFC.sh
    Untracked:  code/._pheno2countonly.R
    Untracked:  code/._qtlsPvalOppFrac.py
    Untracked:  code/._quantassign2parsedpeak.py
    Untracked:  code/._removeloc_pheno.py
    Untracked:  code/._run_leafcutterDiffIso.sh
    Untracked:  code/._run_sepUsagephen.sh
    Untracked:  code/._selectNominalPvalues.py
    Untracked:  code/._sepUsagePhen.py
    Untracked:  code/._snakemakePAS.batch
    Untracked:  code/._snakemakefiltPAS.batch
    Untracked:  code/._submit-snakemakePAS.sh
    Untracked:  code/._submit-snakemakefiltPAS.sh
    Untracked:  code/._subset_diffisopheno.py
    Untracked:  code/._subtrachfiveprimeUTR.sh
    Untracked:  code/._subtractExons.sh
    Untracked:  code/._subtractfiveprimeUTR.sh
    Untracked:  code/._utrdms2saf.py
    Untracked:  code/.snakemake/
    Untracked:  code/APAqtl_nominal.err
    Untracked:  code/APAqtl_nominal.out
    Untracked:  code/APAqtl_nominal_39.err
    Untracked:  code/APAqtl_nominal_39.out
    Untracked:  code/APAqtl_nominal_nonNorm.err
    Untracked:  code/APAqtl_nominal_nonNorm.out
    Untracked:  code/APAqtl_permuted.err
    Untracked:  code/APAqtl_permuted.out
    Untracked:  code/ApaQTL_nominalNonnorm.sh
    Untracked:  code/BothFracDTPlot1stintron.err
    Untracked:  code/BothFracDTPlot1stintron.out
    Untracked:  code/BothFracDTPlot4stintron.err
    Untracked:  code/BothFracDTPlot4stintron.out
    Untracked:  code/BothFracDTPlotGeneRegions.err
    Untracked:  code/BothFracDTPlotGeneRegions.out
    Untracked:  code/BothFracDTPlotGeneRegions_norm.err
    Untracked:  code/BothFracDTPlotGeneRegions_norm.out
    Untracked:  code/BothFracDTPlotGeneRegions_normalized.sh
    Untracked:  code/DistPAS2Sig.py
    Untracked:  code/EncodeRNADTPlotGeneRegions.err
    Untracked:  code/EncodeRNADTPlotGeneRegions.out
    Untracked:  code/FC_UTR.err
    Untracked:  code/FC_UTR.out
    Untracked:  code/FC_UTR.sh
    Untracked:  code/FC_newPAS_olddata.err
    Untracked:  code/FC_newPAS_olddata.out
    Untracked:  code/FC_newPeaks_olddata.sh
    Untracked:  code/LC_samplegroups.py
    Untracked:  code/NascentRNAdtPlot.sh
    Untracked:  code/NetSeq_fourthintronDT.sh
    Untracked:  code/README.md
    Untracked:  code/Rplots.pdf
    Untracked:  code/TESplots100bp.err
    Untracked:  code/TESplots100bp.out
    Untracked:  code/TESplots100bp.sh
    Untracked:  code/TESplots150bp.err
    Untracked:  code/TESplots150bp.out
    Untracked:  code/TESplots150bp.sh
    Untracked:  code/TESplots200bp.err
    Untracked:  code/TESplots200bp.out
    Untracked:  code/TESplots200bp.sh
    Untracked:  code/Untitled
    Untracked:  code/Upstream100Bases_general.py
    Untracked:  code/ZipandTabPheno.sh
    Untracked:  code/aAPAqtl_nominal39ind.sh
    Untracked:  code/apaQTLCorrectPvalMakeQQ_4pc.R
    Untracked:  code/apaQTL_Nominal_4pc.sh
    Untracked:  code/apaQTL_permuted.4pc.sh
    Untracked:  code/apaqtlfacetboxplots.R
    Untracked:  code/assignNucIntonpeak2intronlocs.sh
    Untracked:  code/assignPeak2Intronicregion.err
    Untracked:  code/assignPeak2Intronicregion.out
    Untracked:  code/assignTotIntronpeak2intronlocs.sh
    Untracked:  code/assigntotPeak2Intronicregion.err
    Untracked:  code/assigntotPeak2Intronicregion.out
    Untracked:  code/bam2BW_5primemost.sh
    Untracked:  code/bam2bw.err
    Untracked:  code/bam2bw.out
    Untracked:  code/bam2bw_5primemost.err
    Untracked:  code/bam2bw_5primemost.out
    Untracked:  code/bothFracDTplot1stintron.sh
    Untracked:  code/bothFracDTplot4thintron.sh
    Untracked:  code/bothFrac_FC.err
    Untracked:  code/bothFrac_FC.out
    Untracked:  code/bothFrac_FC.sh
    Untracked:  code/codingdms2bed.py
    Untracked:  code/dag.pdf
    Untracked:  code/dagPAS.pdf
    Untracked:  code/dagfiltPAS.pdf
    Untracked:  code/encodeRNADTplots.sh
    Untracked:  code/extractGenotypes.py
    Untracked:  code/fc2leafphen.py
    Untracked:  code/finalPASbed2SAF.py
    Untracked:  code/findbuginpeaks.R
    Untracked:  code/fix4su304corr.py
    Untracked:  code/fix4su604corr.py
    Untracked:  code/fix4sukalisto.py
    Untracked:  code/fixFChead_bothfrac.py
    Untracked:  code/fixFChead_summary.py
    Untracked:  code/fixH3k12ac.py
    Untracked:  code/fixRNAhead4corr.py
    Untracked:  code/fixRNAkalisto.py
    Untracked:  code/fixgroupedtranscript.py
    Untracked:  code/fixhead_netseqfc.py
    Untracked:  code/get100upPAS.py
    Untracked:  code/getSeq100up.sh
    Untracked:  code/getseq100up.err
    Untracked:  code/getseq100up.out
    Untracked:  code/grouptranscripts.err
    Untracked:  code/grouptranscripts.out
    Untracked:  code/grouptranscripts.py
    Untracked:  code/log/
    Untracked:  code/makeSAFbothfrac5perc.py
    Untracked:  code/makegencondeTSSfile.py
    Untracked:  code/mergeBW_norm.sh
    Untracked:  code/mergeBWnorm.err
    Untracked:  code/mergeBWnorm.out
    Untracked:  code/mergeBamNacent.err
    Untracked:  code/mergeBamNacent.out
    Untracked:  code/mergeBamNascent.sh
    Untracked:  code/mnase1stintron.sh
    Untracked:  code/mnaseDTPlot1stintron.err
    Untracked:  code/mnaseDTPlot1stintron.out
    Untracked:  code/mnaseDTPlot4thintron.err
    Untracked:  code/mnaseDTPlot4thintron.out
    Untracked:  code/mnaseDT_fourthintron.sh
    Untracked:  code/netDTPlot4thintron.out
    Untracked:  code/netseqDTplot1stIntron.sh
    Untracked:  code/netseqFC.err
    Untracked:  code/netseqFC.out
    Untracked:  code/netseqFC.sh
    Untracked:  code/neyDTPlot4thintron.err
    Untracked:  code/qtlFacetBoxplots.err
    Untracked:  code/qtlFacetBoxplots.out
    Untracked:  code/qtlsPvalOppFrac.py
    Untracked:  code/removeloc_pheno.py
    Untracked:  code/run_DistPAS2Sig.err
    Untracked:  code/run_DistPAS2Sig.out
    Untracked:  code/run_distPAS2Sig.sh
    Untracked:  code/run_leafcutterDiffIso.sh
    Untracked:  code/run_leafcutter_ds.err
    Untracked:  code/run_leafcutter_ds.out
    Untracked:  code/run_qtlFacetBoxplots.sh
    Untracked:  code/run_sepUsagephen.sh
    Untracked:  code/run_sepusage.err
    Untracked:  code/run_sepusage.out
    Untracked:  code/selectNominalPvalues.py
    Untracked:  code/sepUsagePhen.py
    Untracked:  code/snakePASlog.out
    Untracked:  code/snakefiltPASlog.out
    Untracked:  code/subset_diffisopheno.py
    Untracked:  code/subtract5UTR.err
    Untracked:  code/subtract5UTR.out
    Untracked:  code/subtractExons.err
    Untracked:  code/subtractExons.out
    Untracked:  code/subtractExons.sh
    Untracked:  code/subtractfiveprimeUTR.sh
    Untracked:  code/transcriptdm2bed.py
    Untracked:  code/utrdms2saf.py
    Untracked:  code/zipandtabPhen.err
    Untracked:  code/zipandtabPhen.out
    Untracked:  data/CompareOldandNew/
    Untracked:  data/DTmatrix/
    Untracked:  data/DiffIso/
    Untracked:  data/EncodeRNA/
    Untracked:  data/ExampleQTLPlots/
    Untracked:  data/GeuvadisRNA/
    Untracked:  data/NascentRNA/
    Untracked:  data/PAS/
    Untracked:  data/QTLGenotypes/
    Untracked:  data/QTLoverlap/
    Untracked:  data/QTLoverlap_nonNorm/
    Untracked:  data/README.md
    Untracked:  data/RNAseq/
    Untracked:  data/Reads2UTR/
    Untracked:  data/SignalSiteFiles/
    Untracked:  data/ThirtyNineIndQtl_nominal/
    Untracked:  data/apaQTLNominal/
    Untracked:  data/apaQTLNominal_4pc/
    Untracked:  data/apaQTLPermuted/
    Untracked:  data/apaQTLPermuted_4pc/
    Untracked:  data/apaQTLs/
    Untracked:  data/assignedPeaks/
    Untracked:  data/bam/
    Untracked:  data/bam_clean/
    Untracked:  data/bam_waspfilt/
    Untracked:  data/bed_10up/
    Untracked:  data/bed_clean/
    Untracked:  data/bed_clean_sort/
    Untracked:  data/bed_waspfilter/
    Untracked:  data/bedsort_waspfilter/
    Untracked:  data/bothFrac_FC/
    Untracked:  data/bw_norm/
    Untracked:  data/exampleQTLs/
    Untracked:  data/fastq/
    Untracked:  data/filterPeaks/
    Untracked:  data/fourSU/
    Untracked:  data/h3k27ac/
    Untracked:  data/highdiffsiggenes.txt
    Untracked:  data/inclusivePeaks/
    Untracked:  data/inclusivePeaks_FC/
    Untracked:  data/intron_analysis/
    Untracked:  data/mergedBG/
    Untracked:  data/mergedBW_byfrac/
    Untracked:  data/mergedBW_norm/
    Untracked:  data/mergedBam/
    Untracked:  data/mergedbyFracBam/
    Untracked:  data/netseq/
    Untracked:  data/nonNorm_pheno/
    Untracked:  data/nuc_10up/
    Untracked:  data/nuc_10upclean/
    Untracked:  data/peakCoverage/
    Untracked:  data/peaks_5perc/
    Untracked:  data/phenotype/
    Untracked:  data/phenotype_5perc/
    Untracked:  data/sigDiffGenes.txt
    Untracked:  data/sort/
    Untracked:  data/sort_clean/
    Untracked:  data/sort_waspfilter/
    Untracked:  nohup.out
    Untracked:  output/._.DS_Store
    Untracked:  output/._meanCorrelationPhenotypes.svg
    Untracked:  output/dtPlots/
    Untracked:  output/fastqc/
    Untracked:  output/meanCorrelationPhenotypes.svg

Unstaged changes:
    Modified:   analysis/PASusageQC.Rmd
    Modified:   analysis/Readdistagainstfeatures.Rmd
    Modified:   analysis/choosePCs.Rmd
    Modified:   analysis/corrbetweenind.Rmd
    Modified:   analysis/nascenttranscription.Rmd
    Modified:   analysis/rerunQTL_changePC.Rmd
    Modified:   analysis/rna_netseq_h3k12ac.Rmd
    Modified:   code/BothFracDTPlotGeneRegions.sh
    Modified:   code/Snakefile
    Deleted:    code/Upstream10Bases_general.py
    Modified:   code/apaQTLCorrectPvalMakeQQ.R
    Modified:   code/apaQTL_permuted.sh
    Modified:   code/apaQTLsnake.err
    Modified:   code/bed2saf.py
    Modified:   code/cluster.json
    Modified:   code/config.yaml
    Deleted:    code/test.txt

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/974cad6dc70d4dd71a922a449d39d1f1d0fe44f4/docs/nucintronicanalysis.html" target="_blank">974cad6</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-29
</td>
<td>
strat by exp
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/491253b623c5af7b8081b328b447fa31c5153c0a/docs/nucintronicanalysis.html" target="_blank">491253b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/f82d5a4af310a92d87494b6809e6cb61323cc178/analysis/nucintronicanalysis.Rmd" target="_blank">f82d5a4</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-24
</td>
<td>
add pull 1st and 4th
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/ef247581d915a099c0adc0acce2f1d2998a24961/docs/nucintronicanalysis.html" target="_blank">ef24758</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/2632f111d82c09c085ff251404e734f0bb862dba/analysis/nucintronicanalysis.Rmd" target="_blank">2632f11</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
<td>
add tot enriched
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/b454fc6e4926213fe62688447a066aa5990c4dee/docs/nucintronicanalysis.html" target="_blank">b454fc6</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/fc5727e38d6bb9317a92fa3f9ae8762090c19069/analysis/nucintronicanalysis.Rmd" target="_blank">fc5727e</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
<td>
add boxplots by pau
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/ab2692689afb940c49ce91e6d28d41719d30075d/docs/nucintronicanalysis.html" target="_blank">ab26926</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/6bfa078733ba29f9d36b40aeb0806a4106dd293b/analysis/nucintronicanalysis.Rmd" target="_blank">6bfa078</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
<td>
by delta pau
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/ace11ccd5c571d9720e66baa89949d6e9439a69d/docs/nucintronicanalysis.html" target="_blank">ace11cc</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/nucintronicanalysis.html" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/a259a17e4ca3bfe70a6fcf372a01bd63ad4ca1bd/analysis/nucintronicanalysis.Rmd" target="_blank">a259a17</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
<td>
fix bug with utrs
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/365e817d4e8a26a474e308bfb04ac638f3b62e92/docs/nucintronicanalysis.html" target="_blank">365e817</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/d859f02f7079e5fdac54304f492e7e658bf3f0b6/docs/nucintronicanalysis.html" target="_blank">d859f02</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/82fdc65cc1f44b0d2ec541d4d992c803c6067730/analysis/nucintronicanalysis.Rmd" target="_blank">82fdc65</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-21
</td>
<td>
add by length
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/801ca1b09492172c6076aa44cb901b40524e0164/docs/nucintronicanalysis.html" target="_blank">801ca1b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/a45570155fbe1b9b4bec8f57969257f733790afe/analysis/nucintronicanalysis.Rmd" target="_blank">a455701</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-20
</td>
<td>
analysis plot
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/brimittleman/apaQTL/d89772dae76346f78daaef40d61bd313034c4966/docs/nucintronicanalysis.html" target="_blank">d89772d</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ee9296437fb85f9bc1eee9ababefeebcb346be55/analysis/nucintronicanalysis.Rmd" target="_blank">ee92964</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-15
</td>
<td>
start ideas for inton analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>I am interested in understanding where in the introns the nuclear peaks are. Are they closer to the three prime or five prime edge of the intron. This may help us understand if NMD is contributing to the loss of transcripts between the nuclear and total fraction.</p>
<p>I need to create an annotation with introns that do not overlap. For this I will use line up all of the exons for a gene then take the open spaces as introns.</p>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>── Attaching packages ───────────────────────────────────────────────── tidyverse 1.2.1 ──</code></pre>
<pre><code>✔ ggplot2 3.1.1       ✔ purrr   0.3.2  
✔ tibble  2.1.1       ✔ dplyr   0.8.0.1
✔ tidyr   0.8.3       ✔ stringr 1.3.1  
✔ readr   1.3.1       ✔ forcats 0.3.0  </code></pre>
<pre><code>── Conflicts ──────────────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()</code></pre>
<pre class="r"><code>library(workflowr)</code></pre>
<pre><code>This is workflowr version 1.3.0
Run ?workflowr for help getting started</code></pre>
<pre class="r"><code>library(cowplot)</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;</code></pre>
<pre><code>The following object is masked from &#39;package:ggplot2&#39;:

    ggsave</code></pre>
<pre class="r"><code>nucIntronicPeaks=read.table(&quot;../data/peaks_5perc/APApeak_Peaks_GeneLocAnno.Nuclear.5perc.fc&quot;, stringsAsFactors = F, header = F,col.names = c(&quot;chr&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;, &quot;loc&quot;, &quot;strand&quot;, &quot;peak&quot;, &quot;avgUsage&quot;)) %&gt;% filter(loc==&quot;intron&quot;)

nucIntronicPeaksBed=nucIntronicPeaks %&gt;% mutate(ID=paste(peak,gene,loc, sep=&quot;:&quot;)) %&gt;% dplyr::select(chr, start, end, ID, avgUsage, strand)

write.table(nucIntronicPeaksBed, &quot;../data/intron_analysis/NuclearIntronicPeaks.bed&quot;, col.names = F, row.names = F, quote = F,sep=&quot;\t&quot;)</code></pre>
<p>I will need to assign each of these to an intron in the new annotation.</p>
<p>The genome annotation file, Transcript2GeneName.dms has the information i need. I will need to parse this file. I need all exons for a gene (longest transcript) The file has the exon starts and ends for each transcript.</p>
<p>I will remove the exon locations for full transcripts using bedtools subtract.</p>
<p>Create transcript file.I will select all of the transcripts in the dms file and merge by gene name. Then I can subtract the exons</p>
<pre class="bash"><code>python transcriptdm2bed.py</code></pre>
<p>Sort the output, group by transcript and fix order of columns.</p>
<pre class="bash"><code>sort -k1,1 -k2,2n /project2/gilad/briana/genome_anotation_data/RefSeq_annotations/AllTranscriptsbyName.bed &gt; /project2/gilad/briana/genome_anotation_data/RefSeq_annotations/AllTranscriptsbyName.sort.bed

sbatch grouptranscripts.py

python fixgroupedtranscript.py</code></pre>
<p>I want to subract any exon or UTR seqeunce. I have an annotation bed file I will use:</p>
<pre class="r"><code>exonandUTRs=read.table(&quot;../../genome_anotation_data/RefSeq_annotations/ncbiRefSeq_FormatedallAnnotation.sort.bed&quot;, col.names = c(&quot;CHR&quot;, &quot;start&quot;, &quot;end&quot;, &quot;ID&quot;, &quot;score&quot;, &quot;strand&quot;),stringsAsFactors = F)%&gt;% separate(ID, into=c(&quot;loc&quot;, &quot;gene&quot;),sep=&quot;:&quot;) %&gt;% filter(loc!=&quot;intron&quot;) %&gt;% dplyr::select(-loc) %&gt;% mutate(CHR=paste(&quot;chr&quot;, CHR, sep=&quot;&quot;))


write.table(exonandUTRs, file=&quot;../data/intron_analysis/ExonandUTRloc.bed&quot;, quote=F, col.names = F, row.names = F, sep=&quot;\t&quot;)</code></pre>
<pre class="bash"><code>sort -k1,1 -k2,2n ../data/intron_analysis/ExonandUTRloc.bed &gt; ../data/intron_analysis/ExonandUTRloc.sort.bed</code></pre>
<pre class="bash"><code>sbatch subtractExons.sh</code></pre>
<p>sort:</p>
<pre class="bash"><code>sort -k1,1 -k2,2n /project2/gilad/briana/apaQTL/data/intron_analysis/transcriptsMinusExons.bed &gt; /project2/gilad/briana/apaQTL/data/intron_analysis/transcriptsMinusExons.sort.bed</code></pre>
<p>Next I will map the intronic peaks on these positions.</p>
<pre class="bash"><code>sbatch assignNucIntonpeak2intronlocs.sh</code></pre>
<p>Plot percentage of intron where PAS is.</p>
<pre class="r"><code>pas2intron=read.table(&quot;../data/intron_analysis/IntronPeaksontoIntrons.bed&quot;,col.names = c(&quot;intronCHR&quot;, &quot;intronStart&quot;, &quot;intronEnd&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;, &quot;peakCHR&quot;, &quot;peakStart&quot;, &quot;peakEnd&quot;, &quot;PeakID&quot;, &quot;meanUsage&quot;, &quot;peakStrand&quot;)) %&gt;% mutate(PASloc=ifelse(strand==&quot;+&quot;, peakEnd, peakStart)) %&gt;% dplyr::select(intronStart, intronEnd, gene, strand, PeakID, PASloc ,meanUsage) %&gt;% mutate(intronLength=intronEnd-intronStart , distance2PAS= ifelse(strand==&quot;+&quot;, PASloc-intronStart, intronEnd-PASloc), propIntron=distance2PAS/intronLength)</code></pre>
<pre class="r"><code>nuclearplot=ggplot(pas2intron, aes(x=propIntron)) + geom_histogram(bins=50, aes(y=..count../33345)) + labs(title=&quot;PAS position within intron \n for Nuclear PAS&quot;, y=&quot;Proportion of Intronic PAS&quot;, x=&quot;Proportion of Intronic Length&quot;)</code></pre>
<p>Facet by usage 0-25, 25-50, 50-75, 75-1</p>
<pre class="r"><code>pas2intron_usage=pas2intron %&gt;% mutate(UsageCat=ifelse(meanUsage&lt;=.5, &quot;low&quot;, &quot;high&quot;))

ggplot(pas2intron_usage, aes(x=propIntron, fill=UsageCat)) + geom_histogram(bins=50, aes(y=..count../33345)) + labs(title=&quot;PAS position within intron&quot;, y=&quot;Proportion of Intronic PAS&quot;, x=&quot;Proportion of Intronic Length&quot;) + facet_grid(~UsageCat)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-12-1">
Past versions of unnamed-chunk-12-1.png
</button>
</p>
<div id="fig-unnamed-chunk-12-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-12-1.png" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/d859f02f7079e5fdac54304f492e7e658bf3f0b6/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-12-1.png" target="_blank">d859f02</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look at different intron lengths:</p>
<p>First i want to look at the distribution of intorn lengths:</p>
<pre class="r"><code>ggplot(pas2intron_usage, aes(x=log10(intronLength))) + geom_density()</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-13-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-1">
Past versions of unnamed-chunk-13-1.png
</button>
</p>
<div id="fig-unnamed-chunk-13-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-13-1.png" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/d859f02f7079e5fdac54304f492e7e658bf3f0b6/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-13-1.png" target="_blank">d859f02</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-21
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>I will look at above and below the mean intron length:</p>
<pre class="r"><code>meanIntronlength=mean(pas2intron_usage$intronLength)

pas2intron_length=pas2intron %&gt;% mutate(LengthCat=ifelse(intronLength&lt;=meanIntronlength, &quot;bottom&quot;, &quot;top&quot;))


ggplot(pas2intron_length, aes(x=propIntron, fill=LengthCat)) + geom_histogram(bins=50, aes(y=..count../33345)) + labs(title=&quot;PAS position within intron&quot;, y=&quot;Proportion of Intronic PAS&quot;, x=&quot;Proportion of Intronic Length&quot;) + facet_grid(~LengthCat)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-14-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-1">
Past versions of unnamed-chunk-14-1.png
</button>
</p>
<div id="fig-unnamed-chunk-14-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-14-1.png" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggplot(pas2intron_length, aes(x=distance2PAS, fill=LengthCat)) + geom_histogram(bins=50, aes(y=..count../33345)) + labs(title=&quot;PAS position within intron&quot;, y=&quot;Proportion of Intronic PAS&quot;, x=&quot;Proportion of Intronic Length&quot;) + facet_grid(~LengthCat)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-14-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-2">
Past versions of unnamed-chunk-14-2.png
</button>
</p>
<div id="fig-unnamed-chunk-14-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-14-2.png" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look at quartiles:</p>
<pre class="r"><code>summary(pas2intron_usage$intronLength)</code></pre>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
    106    3929    9220   22248   24094 1102540 </code></pre>
<pre class="r"><code>pas2intron_length2=pas2intron %&gt;% mutate(LengthCat=ifelse(intronLength&lt;=3929, &quot;first&quot;, ifelse(intronLength&gt;3929 &amp;intronLength&lt;=9220, &quot;second&quot;, ifelse(intronLength&gt;9220 &amp;intronLength&lt;=24094, &quot;third&quot;, &quot;fourth&quot;))))

pas2intron_length2$LengthCat &lt;- factor(pas2intron_length2$LengthCat, levels=c(&quot;first&quot;, &quot;second&quot;, &quot;third&quot;, &quot;fourth&quot;))
                                         
ggplot(pas2intron_length2, aes(x=propIntron, fill=LengthCat)) + geom_histogram(bins=50, aes(y=..count../33345)) + labs(title=&quot;PAS position within intron \n Nuclear intronic PAS&quot;, y=&quot;Proportion of Intronic PAS&quot;, x=&quot;Proportion of Intronic Length&quot;) + facet_grid(~LengthCat)+theme(axis.text.x = element_text(angle = 90, hjust = 1))</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-15-1">
Past versions of unnamed-chunk-15-1.png
</button>
</p>
<div id="fig-unnamed-chunk-15-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ace11ccd5c571d9720e66baa89949d6e9439a69d/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-15-1.png" target="_blank">ace11cc</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-15-1.png" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggplot(pas2intron_length2, aes(x=log(distance2PAS)), by=LengthCat, col=LengthCat) + stat_ecdf(aes(col=LengthCat)) </code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-15-2.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-15-2">
Past versions of unnamed-chunk-15-2.png
</button>
</p>
<div id="fig-unnamed-chunk-15-2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ace11ccd5c571d9720e66baa89949d6e9439a69d/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-15-2.png" target="_blank">ace11cc</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-15-2.png" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look at distribution in total fraction:</p>
<pre class="r"><code>totIntronicPeaks=read.table(&quot;../data/peaks_5perc/APApeak_Peaks_GeneLocAnno.Total.5perc.fc&quot;, stringsAsFactors = F, header = F,col.names = c(&quot;chr&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;, &quot;loc&quot;, &quot;strand&quot;, &quot;peak&quot;, &quot;avgUsage&quot;)) %&gt;% filter(loc==&quot;intron&quot;)

totIntronicPeaksBed=totIntronicPeaks %&gt;% mutate(ID=paste(peak,gene,loc, sep=&quot;:&quot;)) %&gt;% dplyr::select(chr, start, end, ID, avgUsage, strand)

write.table(totIntronicPeaksBed, &quot;../data/intron_analysis/TotalIntronicPeaks.bed&quot;, col.names = F, row.names = F, quote = F,sep=&quot;\t&quot;)</code></pre>
<p>map these to the intron file</p>
<pre class="bash"><code>sbatch assignTotIntronpeak2intronlocs.sh</code></pre>
<pre class="r"><code>pas2intronTot=read.table(&quot;../data/intron_analysis/TotalIntronPeaksontoIntrons.bed&quot;,col.names = c(&quot;intronCHR&quot;, &quot;intronStart&quot;, &quot;intronEnd&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;, &quot;peakCHR&quot;, &quot;peakStart&quot;, &quot;peakEnd&quot;, &quot;PeakID&quot;, &quot;meanUsage&quot;, &quot;peakStrand&quot;)) %&gt;% mutate(PASloc=ifelse(strand==&quot;+&quot;, peakEnd, peakStart)) %&gt;% dplyr::select(intronStart, intronEnd, gene, strand, PeakID, PASloc ,meanUsage) %&gt;% mutate(intronLength=intronEnd-intronStart , distance2PAS= ifelse(strand==&quot;+&quot;, PASloc-intronStart, intronEnd-PASloc), propIntron=distance2PAS/intronLength)

nrow(pas2intronTot)</code></pre>
<pre><code>[1] 31954</code></pre>
<pre class="r"><code>totalplot=ggplot(pas2intronTot, aes(x=propIntron)) + geom_histogram(bins=50, aes(y=..count../31954)) + labs(title=&quot;PAS position within intron \nfor total PAS&quot;, y=&quot;Proportion of Intronic PAS&quot;, x=&quot;Proportion of Intronic Length&quot;)</code></pre>
<pre class="r"><code>plot_grid(totalplot, nuclearplot)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-20-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-20-1">
Past versions of unnamed-chunk-20-1.png
</button>
</p>
<div id="fig-unnamed-chunk-20-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-20-1.png" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>summary(pas2intronTot$intronLength)</code></pre>
<pre><code>   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
    106    3785    8872   21032   22928 1102540 </code></pre>
<pre class="r"><code>pas2intron_totlength2=pas2intronTot %&gt;% mutate(LengthCat=ifelse(intronLength&lt;=3785, &quot;first&quot;, ifelse(intronLength&gt;3785 &amp;intronLength&lt;=8872, &quot;second&quot;, ifelse(intronLength&gt;8872 &amp;intronLength&lt;=22928, &quot;third&quot;, &quot;fourth&quot;))))

pas2intron_totlength2$LengthCat &lt;- factor(pas2intron_totlength2$LengthCat, levels=c(&quot;first&quot;, &quot;second&quot;, &quot;third&quot;, &quot;fourth&quot;))

                                         
ggplot(pas2intron_totlength2, aes(x=propIntron, fill=LengthCat)) + geom_histogram(bins=50, aes(y=..count../31954)) + labs(title=&quot;PAS position within intron \n Total intronic PAS&quot;, y=&quot;Proportion of Intronic PAS&quot;, x=&quot;Proportion of Intronic Length&quot;) + facet_grid(~LengthCat) + theme(axis.text.x = element_text(angle = 90, hjust = 1))</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-21-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-21-1">
Past versions of unnamed-chunk-21-1.png
</button>
</p>
<div id="fig-unnamed-chunk-21-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ace11ccd5c571d9720e66baa89949d6e9439a69d/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-21-1.png" target="_blank">ace11cc</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/57d8a8c3302213fce92c63c5cb645ab7ede038d2/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-21-1.png" target="_blank">57d8a8c</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Look by differences in <span class="math inline">\(\Delta\)</span> PAU.</p>
<pre class="r"><code>effectsize=read.table(&quot;../data/DiffIso/TN_diff_isoform_AllChrom_effect_sizes.txt&quot;, stringsAsFactors = F, col.names=c(&#39;PAS&#39;,  &#39;logef&#39; ,&#39;Nuclear&#39;, &#39;Total&#39;,&#39;deltaPAU&#39;)) %&gt;% filter(PAS != &quot;intron&quot;) %&gt;% dplyr::select(PAS, deltaPAU)</code></pre>
<pre class="r"><code>pas2intronNuc=read.table(&quot;../data/intron_analysis/IntronPeaksontoIntrons.bed&quot;,col.names = c(&quot;intronCHR&quot;, &quot;intronStart&quot;, &quot;intronEnd&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;, &quot;peakCHR&quot;, &quot;peakStart&quot;, &quot;peakEnd&quot;, &quot;PeakID&quot;, &quot;meanUsage&quot;, &quot;peakStrand&quot;)) %&gt;% mutate(PAS=paste(peakCHR, peakStart, peakEnd, gene, sep=&quot;:&quot;)) %&gt;% inner_join(effectsize, by=&quot;PAS&quot;) %&gt;% mutate(PASloc=ifelse(strand==&quot;+&quot;, peakEnd, peakStart)) %&gt;% dplyr::select(intronStart, intronEnd, gene, strand, PeakID, PASloc ,meanUsage, deltaPAU) %&gt;% mutate(intronLength=intronEnd-intronStart , distance2PAS= ifelse(strand==&quot;+&quot;, PASloc-intronStart, intronEnd-PASloc), propIntron=distance2PAS/intronLength)

pas2intronNuc$deltaPAU= as.numeric(pas2intronNuc$deltaPAU)</code></pre>
<p>Plot relationship between delta PAU and distance to proporiton of intron</p>
<pre class="r"><code>ggplot(pas2intronNuc, aes(x=propIntron, y=deltaPAU)) + geom_point() + geom_density2d(col=&quot;red&quot;)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-24-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-24-1">
Past versions of unnamed-chunk-24-1.png
</button>
</p>
<div id="fig-unnamed-chunk-24-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ab2692689afb940c49ce91e6d28d41719d30075d/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-24-1.png" target="_blank">ab26926</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pas2intronNuc_cat=pas2intronNuc %&gt;% mutate(Fraction=ifelse(deltaPAU &lt;= -0.2, &quot;NuclearSpecific&quot;, ifelse(deltaPAU&gt;-.2 &amp;deltaPAU &lt; .2, &quot;NotDiff&quot;, &quot;TotalEnriched&quot;))) %&gt;% mutate(LengthCat=ifelse(intronLength&lt;=3929, &quot;first&quot;, ifelse(intronLength&gt;3929 &amp;intronLength&lt;=9220, &quot;second&quot;, ifelse(intronLength&gt;9220 &amp;intronLength&lt;=24094, &quot;third&quot;, &quot;fourth&quot;))))

pas2intronNuc_cat$LengthCat &lt;- factor(pas2intronNuc_cat$LengthCat, levels=c(&quot;first&quot;, &quot;second&quot;, &quot;third&quot;, &quot;fourth&quot;))</code></pre>
<p>Plot this:</p>
<pre class="r"><code>ggplot(pas2intronNuc_cat, aes(y=propIntron, x=Fraction, fill=Fraction)) + geom_boxplot(alpha=.5) + facet_grid(~LengthCat)  + theme(axis.text.x = element_text(angle = 90, hjust = 1)) </code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-26-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-26-1">
Past versions of unnamed-chunk-26-1.png
</button>
</p>
<div id="fig-unnamed-chunk-26-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ef247581d915a099c0adc0acce2f1d2998a24961/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-26-1.png" target="_blank">ef24758</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/b454fc6e4926213fe62688447a066aa5990c4dee/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-26-1.png" target="_blank">b454fc6</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ab2692689afb940c49ce91e6d28d41719d30075d/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-26-1.png" target="_blank">ab26926</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggplot(pas2intronNuc_cat, aes(x=propIntron, by=Fraction, fill=Fraction)) + geom_density(alpha=.5) + facet_grid(~LengthCat)  + theme(axis.text.x = element_text(angle = 90, hjust = 1))</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-27-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-27-1">
Past versions of unnamed-chunk-27-1.png
</button>
</p>
<div id="fig-unnamed-chunk-27-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ef247581d915a099c0adc0acce2f1d2998a24961/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-27-1.png" target="_blank">ef24758</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/b454fc6e4926213fe62688447a066aa5990c4dee/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-27-1.png" target="_blank">b454fc6</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>ggplot(pas2intronNuc_cat, aes(y=propIntron, x=Fraction, fill=Fraction)) + geom_boxplot(alpha=.5)  + theme(axis.text.x = element_text(angle = 90, hjust = 1)) </code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-28-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-28-1">
Past versions of unnamed-chunk-28-1.png
</button>
</p>
<div id="fig-unnamed-chunk-28-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ef247581d915a099c0adc0acce2f1d2998a24961/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-28-1.png" target="_blank">ef24758</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pas2intronTotal=read.table(&quot;../data/intron_analysis/TotalIntronPeaksontoIntrons.bed&quot;,col.names = c(&quot;intronCHR&quot;, &quot;intronStart&quot;, &quot;intronEnd&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;, &quot;peakCHR&quot;, &quot;peakStart&quot;, &quot;peakEnd&quot;, &quot;PeakID&quot;, &quot;meanUsage&quot;, &quot;peakStrand&quot;)) %&gt;% mutate(PAS=paste(peakCHR, peakStart, peakEnd, gene, sep=&quot;:&quot;)) %&gt;% inner_join(effectsize, by=&quot;PAS&quot;) %&gt;% mutate(PASloc=ifelse(strand==&quot;+&quot;, peakEnd, peakStart)) %&gt;% dplyr::select(intronStart, intronEnd, gene, strand, PeakID, PASloc ,meanUsage, deltaPAU) %&gt;% mutate(intronLength=intronEnd-intronStart , distance2PAS= ifelse(strand==&quot;+&quot;, PASloc-intronStart, intronEnd-PASloc), propIntron=distance2PAS/intronLength)

pas2intronTotal$deltaPAU= as.numeric(pas2intronTotal$deltaPAU)


pas2intronTotal_cat=pas2intronTotal %&gt;% mutate(Fraction=ifelse(deltaPAU &lt;= -0.1, &quot;NuclearEnriched&quot;, ifelse(deltaPAU&gt;-.1 &amp;deltaPAU &lt; .1, &quot;NotDiff&quot;, &quot;TotalEnriched&quot;))) %&gt;% mutate(LengthCat=ifelse(intronLength&lt;=3785, &quot;first&quot;, ifelse(intronLength&gt;3785 &amp;intronLength&lt;=8872, &quot;second&quot;, ifelse(intronLength&gt;8872 &amp;intronLength&lt;=22928, &quot;third&quot;, &quot;fourth&quot;))))%&gt;% group_by(gene) %&gt;% mutate(Intornid=ifelse(strand==&quot;+&quot;,  1:n(),n():1)) %&gt;% ungroup()



pas2intronTotal_cat$LengthCat &lt;- factor(pas2intronTotal_cat$LengthCat, levels=c(&quot;first&quot;, &quot;second&quot;, &quot;third&quot;, &quot;fourth&quot;))

ggplot(pas2intronTotal_cat, aes(x=propIntron, by=Fraction, fill=Fraction)) + geom_density(alpha=.5) + facet_grid(~Intornid)  + theme(axis.text.x = element_text(angle = 90, hjust = 1)) </code></pre>
<pre><code>Warning: Groups with fewer than two data points have been dropped.

Warning: Groups with fewer than two data points have been dropped.

Warning: Groups with fewer than two data points have been dropped.

Warning: Groups with fewer than two data points have been dropped.</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-29-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-29-1">
Past versions of unnamed-chunk-29-1.png
</button>
</p>
<div id="fig-unnamed-chunk-29-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/491253b623c5af7b8081b328b447fa31c5153c0a/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-29-1.png" target="_blank">491253b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-24
</td>
</tr>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/ef247581d915a099c0adc0acce2f1d2998a24961/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-29-1.png" target="_blank">ef24758</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-23
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Fascet analysis by which number intron in a gene. I need to reverse the ordering for the negative strand.</p>
<p>First group by gene:</p>
<pre class="r"><code>pas2intronNuc_exonofgene= pas2intronNuc_cat %&gt;% group_by(gene) %&gt;% mutate(Intornid=ifelse(strand==&quot;+&quot;,  1:n(),n():1))</code></pre>
<pre class="r"><code>ggplot(pas2intronNuc_exonofgene, aes(x=propIntron,fill=LengthCat)) + geom_histogram(bins=50)  + facet_grid(LengthCat~Intornid) + theme(axis.text.x = element_text(angle = 90, hjust = 1))</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-31-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-31-1">
Past versions of unnamed-chunk-31-1.png
</button>
</p>
<div id="fig-unnamed-chunk-31-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/491253b623c5af7b8081b328b447fa31c5153c0a/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-31-1.png" target="_blank">491253b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>I’d like to look at just the first exon and the top 50% intron length</p>
<pre class="r"><code>pas2intronTotal_cat_first=pas2intronTotal_cat %&gt;% filter(Intornid==1) %&gt;% mutate(LengthHalf=ifelse(intronLength&lt;=8872, &quot;Short&quot;, &quot;Long&quot;))

ggplot(pas2intronTotal_cat_first, aes(x=propIntron, by=Fraction, fill=Fraction)) + geom_density(alpha=.5) + facet_grid(~LengthHalf)  + theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title=&quot;First intron proportion of the intron\n by intron length and delta PAU&quot;)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-32-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-32-1">
Past versions of unnamed-chunk-32-1.png
</button>
</p>
<div id="fig-unnamed-chunk-32-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/491253b623c5af7b8081b328b447fa31c5153c0a/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-32-1.png" target="_blank">491253b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Took at distance to TSS.</p>
<p>I have the merged transcript gene file and I will remove 5’ UTRs.<br />
I can pull in the merged transcript gene.</p>
<pre class="r"><code>fiveprimeUTRs=read.table(&quot;../../genome_anotation_data/RefSeq_annotations/ncbiRefSeq_FormatedallAnnotation.sort.bed&quot;, col.names = c(&quot;CHR&quot;, &quot;start&quot;, &quot;end&quot;, &quot;ID&quot;, &quot;score&quot;, &quot;strand&quot;),stringsAsFactors = F)%&gt;% separate(ID, into=c(&quot;loc&quot;, &quot;gene&quot;),sep=&quot;:&quot;) %&gt;% filter(loc==&quot;utr5&quot;) %&gt;% dplyr::select(-loc) %&gt;% mutate(CHR=paste(&quot;chr&quot;, CHR, sep=&quot;&quot;))


write.table(fiveprimeUTRs, file=&quot;../data/intron_analysis/fiveprimeloc.bed&quot;, quote=F, col.names = F, row.names = F, sep=&quot;\t&quot;)</code></pre>
<pre class="bash"><code>sort -k1,1 -k2,2n ../data/intron_analysis/fiveprimeloc.bed &gt; ../data/intron_analysis/fiveprimeloc.sort.bed</code></pre>
<pre class="bash"><code>sbatch subtractfiveprimeUTR.sh</code></pre>
<pre class="r"><code>tss=read.table(&quot;../data/intron_analysis/transcriptsMinus5UTR.bed&quot;,col.names = c(&quot;chr&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;), stringsAsFactors = F) %&gt;% mutate(TSS=ifelse(strand==&quot;+&quot;, start, end)) %&gt;% dplyr::select(TSS, gene)</code></pre>
<pre class="r"><code>pas2intronTotal_cat$gene= as.character(pas2intronTotal_cat$gene)
pas2intronTotal_catTSS= pas2intronTotal_cat %&gt;% inner_join(tss, by=&quot;gene&quot;) %&gt;% mutate(dist2TSS= ifelse(strand==&quot;+&quot;, abs(PASloc-TSS), abs(TSS-PASloc))) %&gt;% filter(dist2TSS&lt;10000)


ggplot(pas2intronTotal_catTSS, aes(x=dist2TSS, by=Fraction, fill=Fraction)) + geom_density(alpha=.5) + labs(title=&quot;Distance to TSS by delta PAU first 10KB&quot;, x=&quot;Distance to TSS&quot;)+  theme(axis.text.x = element_text(angle = 90, hjust = 1)) </code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-37-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-37-1">
Past versions of unnamed-chunk-37-1.png
</button>
</p>
<div id="fig-unnamed-chunk-37-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/491253b623c5af7b8081b328b447fa31c5153c0a/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-37-1.png" target="_blank">491253b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pas2intronTotal_catTSS %&gt;% group_by(Fraction) %&gt;% summarise(n())</code></pre>
<pre><code># A tibble: 3 x 2
  Fraction        `n()`
  &lt;chr&gt;           &lt;int&gt;
1 NotDiff         32321
2 NuclearEnriched  1803
3 TotalEnriched     607</code></pre>
<pre class="r"><code>ggplot(pas2intronTotal_catTSS, aes(x=dist2TSS, by=Fraction, fill=Fraction)) + geom_density(alpha=.5) + labs(title=&quot;Distance to TSS by delta PAU&quot;, x=&quot;Distance to TSS&quot;)+  theme(axis.text.x = element_text(angle = 90, hjust = 1)) + scale_x_log10() + facet_grid(~Intornid)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-39-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-39-1">
Past versions of unnamed-chunk-39-1.png
</button>
</p>
<div id="fig-unnamed-chunk-39-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/491253b623c5af7b8081b328b447fa31c5153c0a/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-39-1.png" target="_blank">491253b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pas2intronTotal_catTSS_first=pas2intronTotal_catTSS %&gt;% filter(Intornid==1, dist2TSS&lt;10000)

ggplot(pas2intronTotal_catTSS_first, aes(x=dist2TSS, by=Fraction, fill=Fraction)) + geom_density(alpha=.5) + labs(title=&quot;Distance to TSS for first exon first 10KB&quot;,x=&quot;Distance to TSS&quot;) + theme(axis.text.x = element_text(angle = 90, hjust = 1)) </code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-40-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-40-1">
Past versions of unnamed-chunk-40-1.png
</button>
</p>
<div id="fig-unnamed-chunk-40-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/brimittleman/apaQTL/blob/491253b623c5af7b8081b328b447fa31c5153c0a/docs/figure/nucintronicanalysis.Rmd/unnamed-chunk-40-1.png" target="_blank">491253b</a>
</td>
<td>
brimittleman
</td>
<td>
2019-05-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<pre class="r"><code>pas2intronTotal_catTSS_first %&gt;% group_by(Fraction) %&gt;% summarise(n())</code></pre>
<pre><code># A tibble: 3 x 2
  Fraction        `n()`
  &lt;chr&gt;           &lt;int&gt;
1 NotDiff         12119
2 NuclearEnriched  1107
3 TotalEnriched     300</code></pre>
<div id="deep-tools-first-intron" class="section level2">
<h2>deep tools first intron</h2>
<pre class="r"><code>FirstIntron= read.table(&quot;../data/intron_analysis/transcriptsMinusExons.sort.bed&quot;, stringsAsFactors = F, col.names=c(&quot;chr&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;)) %&gt;% group_by(gene) %&gt;% mutate(Intornid=ifelse(strand==&quot;+&quot;,  1:n(),n():1)) %&gt;% mutate(Chr=str_sub(chr, 4, str_length(chr))) %&gt;% filter(Intornid==1) %&gt;% select(Chr, start, end ,gene ,score ,strand) 

write.table(FirstIntron,file=&quot;../data/intron_analysis/FirstIntronOnly.bed&quot;, col.names = F, row.names = F, quote=F, sep=&quot;\t&quot;)</code></pre>
<p>Sort this:</p>
<pre class="bash"><code>sort -k1,1 -k2,2n ../data/intron_analysis/FirstIntronOnly.bed &gt; ../data/intron_analysis/FirstIntronOnly_Sort.bed</code></pre>
<div id="compare-to-4th" class="section level3">
<h3>compare to 4th</h3>
<pre class="r"><code>FourthIntron= read.table(&quot;../data/intron_analysis/transcriptsMinusExons.sort.bed&quot;, stringsAsFactors = F, col.names=c(&quot;chr&quot;, &quot;start&quot;, &quot;end&quot;, &quot;gene&quot;, &quot;score&quot;, &quot;strand&quot;)) %&gt;% group_by(gene) %&gt;% mutate(Intornid=ifelse(strand==&quot;+&quot;,  1:n(),n():1)) %&gt;% mutate(Chr=str_sub(chr, 4, str_length(chr))) %&gt;% filter(Intornid==4) %&gt;% select(Chr, start, end ,gene ,score ,strand) 

write.table(FourthIntron,file=&quot;../data/intron_analysis/FourthIntronOnly.bed&quot;, col.names = F, row.names = F, quote=F, sep=&quot;\t&quot;)</code></pre>
<p>Sort this:</p>
<pre class="bash"><code>sort -k1,1 -k2,2n ../data/intron_analysis/FourthIntronOnly.bed &gt; ../data/intron_analysis/FourthIntronOnly_Sort.bed</code></pre>
<p>Seperate plots by expression:</p>
<pre class="r"><code>geneNames=read.table(&quot;../../genome_anotation_data/ensemble_to_genename.txt&quot;, sep=&quot;\t&quot;, col.names = c(&#39;gene_id&#39;, &#39;GeneName&#39;, &#39;source&#39; ),stringsAsFactors = F) %&gt;% select(-source)
RNA_TPM=read.table(&#39;../data/RNAseq/kallisto_RNAseq.txt&#39;, stringsAsFactors = F,header = T) %&gt;% dplyr::rename(&quot;gene_id&quot;=gene) %&gt;% inner_join(geneNames,by=&quot;gene_id&quot; )
RNA_TPMmeans=rowMeans(RNA_TPM[,2:21])
RNA_TPMmeanName=as.data.frame(cbind(gene=RNA_TPM$GeneName, RPKM=RNA_TPMmeans)) 
RNA_TPMmeanName$RPKM=as.numeric(as.character(RNA_TPMmeanName$RPKM))
summary(RNA_TPMmeanName$RPKM)</code></pre>
<pre><code>     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
    0.000     0.214     0.523    17.301     2.379 16253.029 </code></pre>
<pre class="r"><code>RNA_TPMmeanName_cat=RNA_TPMmeanName %&gt;% mutate(RPKM_quart=ifelse(RPKM&lt;=.214, &quot;first&quot;, ifelse(RPKM &gt; .214 &amp; RPKM&lt;= .523, &quot;second&quot;, ifelse(RPKM &gt;.523 &amp; RPKM &lt;=  2.379,&quot;third&quot;, &quot;fourth&quot;))))
                                                                 
RNA_TPMmeanName_cat$RPKM_quart &lt;- factor(RNA_TPMmeanName_cat$RPKM_quart, levels=c(&quot;first&quot;, &quot;second&quot;, &quot;third&quot;, &quot;fourth&quot;))</code></pre>
<pre class="r"><code>pas2intronNuc_exonofgene$gene=as.character(pas2intronNuc_exonofgene$gene)
RNA_TPMmeanName_cat$gene=as.character(RNA_TPMmeanName_cat$gene)
pas2intronNuc_exonofgene_rpkm= pas2intronNuc_exonofgene %&gt;% inner_join(RNA_TPMmeanName_cat, by=&quot;gene&quot;)
ggplot(pas2intronNuc_exonofgene_rpkm, aes(x=propIntron,fill=RPKM_quart)) + geom_histogram(bins=50)  + facet_grid(RPKM_quart~Intornid) + theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title=&quot;Location of Intronic Peaks by intron and expression&quot;)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-47-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>Filter just to the fourth quartile and divide this into 4 categories.</p>
<pre class="r"><code>pas2intronNuc_exonofgene_rpkm_topexp=pas2intronNuc_exonofgene_rpkm %&gt;% filter(RPKM_quart==&quot;fourth&quot;)
summary(pas2intronNuc_exonofgene_rpkm_topexp$RPKM)</code></pre>
<pre><code>     Min.   1st Qu.    Median      Mean   3rd Qu.      Max. 
    2.384    10.329    20.795    50.089    43.011 11185.798 </code></pre>
<pre class="r"><code>pas2intronNuc_exonofgene_rpkm_topexp= pas2intronNuc_exonofgene_rpkm_topexp %&gt;%mutate(RPKM_quartTop=ifelse(RPKM&lt;=10.329, &quot;first&quot;, ifelse(RPKM &gt; 10.329 &amp; RPKM&lt;= 20.795, &quot;second&quot;, ifelse(RPKM &gt;20.795 &amp; RPKM &lt;=  43.011,&quot;third&quot;, &quot;fourth&quot;))))
pas2intronNuc_exonofgene_rpkm_topexp$RPKM_quartTop &lt;- factor(pas2intronNuc_exonofgene_rpkm_topexp$RPKM_quartTop, levels=c(&quot;first&quot;, &quot;second&quot;, &quot;third&quot;, &quot;fourth&quot;))


ggplot(pas2intronNuc_exonofgene_rpkm_topexp, aes(x=propIntron,fill=RPKM_quartTop)) + geom_histogram(bins=50)  + facet_grid(RPKM_quartTop~Intornid) + theme(axis.text.x = element_text(angle = 90, hjust = 1)) + labs(title=&quot;Location of Intronic Peaks by intron and expression&quot;)</code></pre>
<p><img src="figure/nucintronicanalysis.Rmd/unnamed-chunk-48-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Scientific Linux 7.4 (Nitrogen)

Matrix products: default
BLAS/LAPACK: /software/openblas-0.2.19-el7-x86_64/lib/libopenblas_haswellp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] cowplot_0.9.4   workflowr_1.3.0 forcats_0.3.0   stringr_1.3.1  
 [5] dplyr_0.8.0.1   purrr_0.3.2     readr_1.3.1     tidyr_0.8.3    
 [9] tibble_2.1.1    ggplot2_3.1.1   tidyverse_1.2.1

loaded via a namespace (and not attached):
 [1] tidyselect_0.2.5 reshape2_1.4.3   haven_1.1.2      lattice_0.20-38 
 [5] colorspace_1.3-2 generics_0.0.2   htmltools_0.3.6  yaml_2.2.0      
 [9] utf8_1.1.4       rlang_0.3.1      pillar_1.3.1     glue_1.3.0      
[13] withr_2.1.2      modelr_0.1.2     readxl_1.1.0     plyr_1.8.4      
[17] munsell_0.5.0    gtable_0.2.0     cellranger_1.1.0 rvest_0.3.2     
[21] evaluate_0.12    labeling_0.3     knitr_1.20       fansi_0.4.0     
[25] broom_0.5.1      Rcpp_1.0.0       scales_1.0.0     backports_1.1.2 
[29] jsonlite_1.6     fs_1.2.6         hms_0.4.2        digest_0.6.18   
[33] stringi_1.2.4    grid_3.5.1       rprojroot_1.3-2  cli_1.0.1       
[37] tools_3.5.1      magrittr_1.5     lazyeval_0.2.1   crayon_1.3.4    
[41] whisker_0.3-2    pkgconfig_2.0.2  MASS_7.3-51.1    xml2_1.2.0      
[45] lubridate_1.7.4  assertthat_0.2.0 rmarkdown_1.10   httr_1.3.1      
[49] rstudioapi_0.10  R6_2.3.0         nlme_3.1-137     git2r_0.23.0    
[53] compiler_3.5.1  </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
